<!DOCTYPE html>
<html lang="en-US" prefix="og: https://ogp.me/ns# fb: https://ogp.me/ns/fb#">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
	<link rel="profile" href="http://gmpg.org/xfn/11">
	<title>Edoardo M Airoldi</title>
<meta name='robots' content='noindex, nofollow' />
	<style>img:is([sizes="auto" i], [sizes^="auto," i]) { contain-intrinsic-size: 3000px 1500px }</style>
	<link rel="alternate" type="application/rss+xml" title="Edoardo M Airoldi &raquo; Feed" href="https://sites.harvard.edu/airoldi/feed/" />
<script type="text/javascript">
/* <![CDATA[ */
window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/15.0.3\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/15.0.3\/svg\/","svgExt":".svg","source":{"concatemoji":"https:\/\/sites.harvard.edu\/airoldi\/wp-includes\/js\/wp-emoji-release.min.js?ver=6.7.2"}};
/*! This file is auto-generated */
!function(i,n){var o,s,e;function c(e){try{var t={supportTests:e,timestamp:(new Date).valueOf()};sessionStorage.setItem(o,JSON.stringify(t))}catch(e){}}function p(e,t,n){e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(t,0,0);var t=new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data),r=(e.clearRect(0,0,e.canvas.width,e.canvas.height),e.fillText(n,0,0),new Uint32Array(e.getImageData(0,0,e.canvas.width,e.canvas.height).data));return t.every(function(e,t){return e===r[t]})}function u(e,t,n){switch(t){case"flag":return n(e,"\ud83c\udff3\ufe0f\u200d\u26a7\ufe0f","\ud83c\udff3\ufe0f\u200b\u26a7\ufe0f")?!1:!n(e,"\ud83c\uddfa\ud83c\uddf3","\ud83c\uddfa\u200b\ud83c\uddf3")&&!n(e,"\ud83c\udff4\udb40\udc67\udb40\udc62\udb40\udc65\udb40\udc6e\udb40\udc67\udb40\udc7f","\ud83c\udff4\u200b\udb40\udc67\u200b\udb40\udc62\u200b\udb40\udc65\u200b\udb40\udc6e\u200b\udb40\udc67\u200b\udb40\udc7f");case"emoji":return!n(e,"\ud83d\udc26\u200d\u2b1b","\ud83d\udc26\u200b\u2b1b")}return!1}function f(e,t,n){var r="undefined"!=typeof WorkerGlobalScope&&self instanceof WorkerGlobalScope?new OffscreenCanvas(300,150):i.createElement("canvas"),a=r.getContext("2d",{willReadFrequently:!0}),o=(a.textBaseline="top",a.font="600 32px Arial",{});return e.forEach(function(e){o[e]=t(a,e,n)}),o}function t(e){var t=i.createElement("script");t.src=e,t.defer=!0,i.head.appendChild(t)}"undefined"!=typeof Promise&&(o="wpEmojiSettingsSupports",s=["flag","emoji"],n.supports={everything:!0,everythingExceptFlag:!0},e=new Promise(function(e){i.addEventListener("DOMContentLoaded",e,{once:!0})}),new Promise(function(t){var n=function(){try{var e=JSON.parse(sessionStorage.getItem(o));if("object"==typeof e&&"number"==typeof e.timestamp&&(new Date).valueOf()<e.timestamp+604800&&"object"==typeof e.supportTests)return e.supportTests}catch(e){}return null}();if(!n){if("undefined"!=typeof Worker&&"undefined"!=typeof OffscreenCanvas&&"undefined"!=typeof URL&&URL.createObjectURL&&"undefined"!=typeof Blob)try{var e="postMessage("+f.toString()+"("+[JSON.stringify(s),u.toString(),p.toString()].join(",")+"));",r=new Blob([e],{type:"text/javascript"}),a=new Worker(URL.createObjectURL(r),{name:"wpTestEmojiSupports"});return void(a.onmessage=function(e){c(n=e.data),a.terminate(),t(n)})}catch(e){}c(n=f(s,u,p))}t(n)}).then(function(e){for(var t in e)n.supports[t]=e[t],n.supports.everything=n.supports.everything&&n.supports[t],"flag"!==t&&(n.supports.everythingExceptFlag=n.supports.everythingExceptFlag&&n.supports[t]);n.supports.everythingExceptFlag=n.supports.everythingExceptFlag&&!n.supports.flag,n.DOMReady=!1,n.readyCallback=function(){n.DOMReady=!0}}).then(function(){return e}).then(function(){var e;n.supports.everything||(n.readyCallback(),(e=n.source||{}).concatemoji?t(e.concatemoji):e.wpemoji&&e.twemoji&&(t(e.twemoji),t(e.wpemoji)))}))}((window,document),window._wpemojiSettings);
/* ]]> */
</script>
<style id='wp-emoji-styles-inline-css' type='text/css'>

	img.wp-smiley, img.emoji {
		display: inline !important;
		border: none !important;
		box-shadow: none !important;
		height: 1em !important;
		width: 1em !important;
		margin: 0 0.07em !important;
		vertical-align: -0.1em !important;
		background: none !important;
		padding: 0 !important;
	}
</style>
<link rel='stylesheet' id='wp-block-library-css' href='https://sites.harvard.edu/airoldi/wp-includes/css/dist/block-library/style.min.css?ver=6.7.2' type='text/css' media='all' />
<style id='safe-svg-svg-icon-style-inline-css' type='text/css'>
.safe-svg-cover{text-align:center}.safe-svg-cover .safe-svg-inside{display:inline-block;max-width:100%}.safe-svg-cover svg{height:100%;max-height:100%;max-width:100%;width:100%}

</style>
<link rel='stylesheet' id='coblocks-frontend-css' href='https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/style-coblocks-1.css?ver=3.1.15' type='text/css' media='all' />
<link rel='stylesheet' id='coblocks-extensions-css' href='https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/style-coblocks-extensions.css?ver=3.1.15' type='text/css' media='all' />
<link rel='stylesheet' id='coblocks-animation-css' href='https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/style-coblocks-animation.css?ver=2677611078ee87eb3b1c' type='text/css' media='all' />
<style id='classic-theme-styles-inline-css' type='text/css'>
/*! This file is auto-generated */
.wp-block-button__link{color:#fff;background-color:#32373c;border-radius:9999px;box-shadow:none;text-decoration:none;padding:calc(.667em + 2px) calc(1.333em + 2px);font-size:1.125em}.wp-block-file__button{background:#32373c;color:#fff;text-decoration:none}
</style>
<style id='global-styles-inline-css' type='text/css'>
:root{--wp--preset--aspect-ratio--square: 1;--wp--preset--aspect-ratio--4-3: 4/3;--wp--preset--aspect-ratio--3-4: 3/4;--wp--preset--aspect-ratio--3-2: 3/2;--wp--preset--aspect-ratio--2-3: 2/3;--wp--preset--aspect-ratio--16-9: 16/9;--wp--preset--aspect-ratio--9-16: 9/16;--wp--preset--color--black: #000;--wp--preset--color--cyan-bluish-gray: #abb8c3;--wp--preset--color--white: white;--wp--preset--color--pale-pink: #f78da7;--wp--preset--color--vivid-red: #cf2e2e;--wp--preset--color--luminous-vivid-orange: #ff6900;--wp--preset--color--luminous-vivid-amber: #fcb900;--wp--preset--color--light-green-cyan: #7bdcb5;--wp--preset--color--vivid-green-cyan: #00d084;--wp--preset--color--pale-cyan-blue: #8ed1fc;--wp--preset--color--vivid-cyan-blue: #0693e3;--wp--preset--color--vivid-purple: #9b51e0;--wp--preset--color--color-bg-alt: #eeeeec;--wp--preset--color--hero-main-bg-color: #b41111;--wp--preset--color--color-bg: #ffffff;--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple: linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%);--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan: linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%);--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange: linear-gradient(135deg,rgba(252,185,0,1) 0%,rgba(255,105,0,1) 100%);--wp--preset--gradient--luminous-vivid-orange-to-vivid-red: linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%);--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray: linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%);--wp--preset--gradient--cool-to-warm-spectrum: linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%);--wp--preset--gradient--blush-light-purple: linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%);--wp--preset--gradient--blush-bordeaux: linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%);--wp--preset--gradient--luminous-dusk: linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%);--wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%);--wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%);--wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%);--wp--preset--font-size--small: 13px;--wp--preset--font-size--medium: 20px;--wp--preset--font-size--large: 36px;--wp--preset--font-size--x-large: 42px;--wp--preset--font-size--larger: 48px;--wp--preset--spacing--20: 0.44rem;--wp--preset--spacing--30: 0.67rem;--wp--preset--spacing--40: 1rem;--wp--preset--spacing--50: 1.5rem;--wp--preset--spacing--60: 2.25rem;--wp--preset--spacing--70: 3.38rem;--wp--preset--spacing--80: 5.06rem;--wp--preset--shadow--natural: 6px 6px 9px rgba(0, 0, 0, 0.2);--wp--preset--shadow--deep: 12px 12px 50px rgba(0, 0, 0, 0.4);--wp--preset--shadow--sharp: 6px 6px 0px rgba(0, 0, 0, 0.2);--wp--preset--shadow--outlined: 6px 6px 0px -3px rgba(255, 255, 255, 1), 6px 6px rgba(0, 0, 0, 1);--wp--preset--shadow--crisp: 6px 6px 0px rgba(0, 0, 0, 1);}:where(.is-layout-flex){gap: 0.5em;}:where(.is-layout-grid){gap: 0.5em;}body .is-layout-flex{display: flex;}.is-layout-flex{flex-wrap: wrap;align-items: center;}.is-layout-flex > :is(*, div){margin: 0;}body .is-layout-grid{display: grid;}.is-layout-grid > :is(*, div){margin: 0;}:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}:where(.wp-block-post-template.is-layout-flex){gap: 1.25em;}:where(.wp-block-post-template.is-layout-grid){gap: 1.25em;}.has-black-color{color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-color{color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-color{color: var(--wp--preset--color--white) !important;}.has-pale-pink-color{color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-color{color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-color{color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-color{color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-color{color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-color{color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-color{color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-color{color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-color{color: var(--wp--preset--color--vivid-purple) !important;}.has-black-background-color{background-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-background-color{background-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-background-color{background-color: var(--wp--preset--color--white) !important;}.has-pale-pink-background-color{background-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-background-color{background-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-background-color{background-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-background-color{background-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-background-color{background-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-background-color{background-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-background-color{background-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-background-color{background-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-background-color{background-color: var(--wp--preset--color--vivid-purple) !important;}.has-black-border-color{border-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-border-color{border-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-border-color{border-color: var(--wp--preset--color--white) !important;}.has-pale-pink-border-color{border-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-border-color{border-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-border-color{border-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-border-color{border-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-border-color{border-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-border-color{border-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-border-color{border-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-border-color{border-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-border-color{border-color: var(--wp--preset--color--vivid-purple) !important;}.has-vivid-cyan-blue-to-vivid-purple-gradient-background{background: var(--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple) !important;}.has-light-green-cyan-to-vivid-green-cyan-gradient-background{background: var(--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan) !important;}.has-luminous-vivid-amber-to-luminous-vivid-orange-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange) !important;}.has-luminous-vivid-orange-to-vivid-red-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-orange-to-vivid-red) !important;}.has-very-light-gray-to-cyan-bluish-gray-gradient-background{background: var(--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray) !important;}.has-cool-to-warm-spectrum-gradient-background{background: var(--wp--preset--gradient--cool-to-warm-spectrum) !important;}.has-blush-light-purple-gradient-background{background: var(--wp--preset--gradient--blush-light-purple) !important;}.has-blush-bordeaux-gradient-background{background: var(--wp--preset--gradient--blush-bordeaux) !important;}.has-luminous-dusk-gradient-background{background: var(--wp--preset--gradient--luminous-dusk) !important;}.has-pale-ocean-gradient-background{background: var(--wp--preset--gradient--pale-ocean) !important;}.has-electric-grass-gradient-background{background: var(--wp--preset--gradient--electric-grass) !important;}.has-midnight-gradient-background{background: var(--wp--preset--gradient--midnight) !important;}.has-small-font-size{font-size: var(--wp--preset--font-size--small) !important;}.has-medium-font-size{font-size: var(--wp--preset--font-size--medium) !important;}.has-large-font-size{font-size: var(--wp--preset--font-size--large) !important;}.has-x-large-font-size{font-size: var(--wp--preset--font-size--x-large) !important;}
:where(.wp-block-post-template.is-layout-flex){gap: 1.25em;}:where(.wp-block-post-template.is-layout-grid){gap: 1.25em;}
:where(.wp-block-columns.is-layout-flex){gap: 2em;}:where(.wp-block-columns.is-layout-grid){gap: 2em;}
:root :where(.wp-block-pullquote){font-size: 1.5em;line-height: 1.6;}
</style>
<link rel='stylesheet' id='cpschool-styles-css' href='https://sites.harvard.edu/airoldi/wp-content/themes/cpschool/css/theme.min.css?ver=0.9.0.5.1733816539' type='text/css' media='all' />
<link rel='stylesheet' id='dashicons-css' href='https://sites.harvard.edu/airoldi/wp-includes/css/dashicons.min.css?ver=6.7.2' type='text/css' media='all' />
<link rel='stylesheet' id='cpsc-styles-css' href='https://sites.harvard.edu/airoldi/wp-content/themes/harvard-flex/css/theme.min.css?ver=0.9.0.5.1727869985' type='text/css' media='all' />
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-includes/js/jquery/jquery.min.js?ver=3.7.1" id="jquery-core-js"></script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-includes/js/jquery/jquery-migrate.min.js?ver=3.4.1" id="jquery-migrate-js"></script>
<link rel="https://api.w.org/" href="https://sites.harvard.edu/airoldi/wp-json/" /><link rel="alternate" title="JSON" type="application/json" href="https://sites.harvard.edu/airoldi/wp-json/wp/v2/pages/49" /><meta name="generator" content="Harvard University Sites 6.7.2 - http://sites.harvard.edu/" />
<link rel="canonical" href="https://sites.harvard.edu/airoldi/" />
<link rel='shortlink' href='https://sites.harvard.edu/airoldi/' />
<link rel="alternate" title="oEmbed (JSON)" type="application/json+oembed" href="https://sites.harvard.edu/airoldi/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fsites.harvard.edu%2Fairoldi%2F" />
<link rel="alternate" title="oEmbed (XML)" type="text/xml+oembed" href="https://sites.harvard.edu/airoldi/wp-json/oembed/1.0/embed?url=https%3A%2F%2Fsites.harvard.edu%2Fairoldi%2F&#038;format=xml" />
	<style>
		@media screen and (max-width: 782px) {
			#wpadminbar li#wp-admin-bar-log-in,
			#wpadminbar li#wp-admin-bar-register {
				display: block;
			}

			#wpadminbar li#wp-admin-bar-log-in a,
			#wpadminbar li#wp-admin-bar-register a {
				padding: 0 8px;
			}
		}
	</style>
<meta property="og:title" content="Home" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://sites.harvard.edu/airoldi/" />
<meta property="og:description" content="bio I received a Ph.D. from Carnegie Mellon University in 2007, working at the intersection of statistical machine learning and computational social science with Stephen Fienberg and Kathleen Carley. My PhD thesis explored modeling approaches and inference strategies for analyzing social and biological networks. Till December 2008, I was a postdoctoral fellow in the Lewis-Sigler..." />
<meta property="og:site_name" content="Edoardo M Airoldi" />
<meta property="og:locale" content="en_US" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="Home" />
<meta name="twitter:description" content="bio I received a Ph.D. from Carnegie Mellon University in 2007, working at the intersection of statistical machine learning and computational social science with Stephen Fienberg and Kathleen Carley. My PhD thesis explored modeling approaches and inference strategies for analyzing social and biological networks. Till December 2008, I was a postdoctoral fellow in the Lewis-Sigler..." />

        <script type="text/javascript">
            var jQueryMigrateHelperHasSentDowngrade = false;

			window.onerror = function( msg, url, line, col, error ) {
				// Break out early, do not processing if a downgrade reqeust was already sent.
				if ( jQueryMigrateHelperHasSentDowngrade ) {
					return true;
                }

				var xhr = new XMLHttpRequest();
				var nonce = 'ce63540fcf';
				var jQueryFunctions = [
					'andSelf',
					'browser',
					'live',
					'boxModel',
					'support.boxModel',
					'size',
					'swap',
					'clean',
					'sub',
                ];
				var match_pattern = /\)\.(.+?) is not a function/;
                var erroredFunction = msg.match( match_pattern );

                // If there was no matching functions, do not try to downgrade.
                if ( null === erroredFunction || typeof erroredFunction !== 'object' || typeof erroredFunction[1] === "undefined" || -1 === jQueryFunctions.indexOf( erroredFunction[1] ) ) {
                    return true;
                }

                // Set that we've now attempted a downgrade request.
                jQueryMigrateHelperHasSentDowngrade = true;

				xhr.open( 'POST', 'https://sites.harvard.edu/airoldi/wp-admin/admin-ajax.php' );
				xhr.setRequestHeader( 'Content-Type', 'application/x-www-form-urlencoded' );
				xhr.onload = function () {
					var response,
                        reload = false;

					if ( 200 === xhr.status ) {
                        try {
                        	response = JSON.parse( xhr.response );

                        	reload = response.data.reload;
                        } catch ( e ) {
                        	reload = false;
                        }
                    }

					// Automatically reload the page if a deprecation caused an automatic downgrade, ensure visitors get the best possible experience.
					if ( reload ) {
						location.reload();
                    }
				};

				xhr.send( encodeURI( 'action=jquery-migrate-downgrade-version&_wpnonce=' + nonce ) );

				// Suppress error alerts in older browsers
				return true;
			}
        </script>

		<meta name="mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-capable" content="yes">
<meta name="apple-mobile-web-app-title" content="Edoardo M Airoldi - ">
					<!-- Google Analytics tracking code output by Beehive Analytics Pro -->
						<script async src="https://www.googletagmanager.com/gtag/js?id=G-GMEVN96PYN&l=beehiveDataLayer"></script>
		<script>
						window.beehiveDataLayer = window.beehiveDataLayer || [];
			function gaplusu() {beehiveDataLayer.push(arguments);}
			gaplusu('js', new Date())
						gaplusu('config', 'G-GMEVN96PYN', {
				'anonymize_ip': false,
				'allow_google_signals': false,
			})
					</script>
		<style type="text/css">
/* <![CDATA[ */
img.latex { vertical-align: middle; border: none; }
/* ]]> */
</style>
		<style type="text/css" id="wp-custom-css">
			@import url("https://harvard-web-publishing.github.io/flex-theme-flavors/global-styles.css");
@import url("https://harvard-web-publishing.github.io/flex-theme-flavors/crimson.css");


		</style>
		<style id="kirki-inline-styles">.logo-font{font-family:inter;}.container{max-width:1170px;}:root{--site-width:1170px;--content-width:1170px;--body-font-size:100;--roundness:3px;--shadows:4px;--color-bg:#ffffff;--color-bg-alt:#eeeeec;--color-bg-contrast:#000000;--color-bg-accent:#b41111;--color-bg-accent-a:rgba(180,17,17,0.5);--color-bg-accent-contrast:#ffffff;--color-bg-accent-hl:#216890;--color-bg-accent-hl-a:rgba(33,104,144,0.5);--color-bg-accent-hl-contrast:#ffffff;--color-bg-alt-contrast:#000000;--color-bg-alt-accent:#b11010;--color-bg-alt-accent-a:rgba(177,16,16,0.5);--color-bg-alt-accent-contrast:#ffffff;--color-bg-alt-accent-hl:#1d5c80;--color-bg-alt-accent-hl-a:rgba(29,92,128,0.5);--color-bg-alt-accent-hl-contrast:#ffffff;--header-main-height:117.562px;--header-main-gap-height:0px;--header-main-padding-v:100;--header-main-logo-font-size:100;--header-main-font-size:100;--header-secondary-font-size:100;--header-secondary-padding-v:0;--hero-main-bg-color:#b41111;--hero-main-bg-color-contrast:#ffffff;--hero-main-bg-color-accent:#fbeeee;--hero-main-bg-color-accent-a:rgba(251,238,238,0.5);--hero-main-bg-color-accent-contrast:#000000;--hero-main-bg-color-accent-hl:#eaf0f5;--hero-main-bg-color-accent-hl-a:rgba(234,240,245,0.5);--hero-main-bg-color-accent-hl-contrast:#000000;--hero-main-img-opacity:0.25;--hero-main-header-main-opacity:1;--headers-font-size:100;}body{font-family:inter;}#navbar-main .nav{font-family:inherit;}.h1,.h2,.h3,.h4,.h5,.h6,h1,h2,h3,h4,h5,h6{font-family:inter;}</style></head>

<body class="home page-template-default page page-id-49 wp-embed-responsive h-style-separator-bottom sidebars-check-sticky navbar-hover-enabled singular main-wrapper-margin-top-disabled">

<div class="site" id="page">
	<a class="skip-link sr-only sr-only-focusable" href="#content">Skip to content</a>

		<div id="branding_header">
	<div class="branding-container clearfix container">
		<div class="branding-left">
			<a href="https://www.harvard.edu/">
				<img src="https://sites.harvard.edu/airoldi/wp-content/themes/harvard-flex/img/harvard-logo.svg" alt="Harvard University homepage" width="235" height="32">
			</a>
		</div>
		<div class="branding-right">
			<a href="https://www.harvard.edu/">HARVARD.EDU</a>
		</div>
	</div>
</div>

	
	<div id="wrapper-navbar-main-top" ></div>
		<div id="wrapper-navbar-main" class="wrapper-navbar" itemscope itemtype="http://schema.org/WebSite">
		<nav id="navbar-main" class="navbar navbar-expand-md nav-styling-underline has-background has-header-main-bg-color-background-color" aria-label="main">
			<div class="navbar-container container">
				
				<!-- Your site title as branding in the menu -->
															<div class="navbar-brand-holder">
												
							<a class="navbar-brand logo-font custom-bg-disabled"  rel="home" href="https://sites.harvard.edu/airoldi/" title="Edoardo M Airoldi" itemprop="url">
								<span class="navbar-brand-text">Edoardo M Airoldi</span>

															</a>

															<a rel="home" href="https://sites.harvard.edu/airoldi/" title="Edoardo M Airoldi" class="navbar-brand-subtext">
																	</a>
							
																		</div>
									<!-- end custom logo -->

				
				<div class="navbar-navs-container">
					<div id="navbar-main-nav-desktop" class="navbar-nav-container"><ul id="menu-main-desktop" class="nav navbar-nav"><li itemscope="itemscope" itemtype="https://www.schema.org/SiteNavigationElement" id="menu-item-menu-main-desktop-320" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-320 nav-item"><a class="nav-link" title="Contact" href="https://sites.harvard.edu/airoldi/contact/"><span class="menu-item-text">Contact</span></a></li>
</ul></div>					
					<div id="navbar-main-nav-buttons">
						<ul class="nav navbar-nav navbar-button-nav-right">
															<li id="navbar-main-btn-search" >
									<button type="button" class="btn btn-secondary nav-link has-styling" data-toggle="modal" data-target="#modal-search">
										<i aria-hidden="true" class="cps-icon cps-icon-search"></i>
										<span aria-hidden="true" class="d-none">Search</span>
										<span class="sr-only">Toggle search interface</span>
									</button>
								</li>
							
														<li id="navbar-main-btn-slide-in-menu" class="btn-modal-slide-in-menu-holder has-mobile">
								<button type="button" class="btn btn-secondary nav-link has-styling" data-toggle="modal" data-target="#modal-slide-in-menu">
									<i aria-hidden="true" class="cps-icon cps-icon-menu"></i>
									<span aria-hidden="true" class="d-none">Menu</span>
									<span class="sr-only">Toggle extended navigation</span>
								</button>
							</li>
						</ul>
					</div>
				</div>

							</div><!-- #navbar-container -->
		</nav>
	</div><!-- #wrapper-navbar end -->

	
	
<div class="main-wrapper wrapper" id="page-wrapper">
	<div class="container" id="content">
		<div class="row">
			<!-- Do the left sidebar check -->
			

<div class="col-md-12 col-xl-12 content-area" id="primary">

			<main class="site-main" id="main">
									
<article class="post-49 page type-page status-publish hentry" id="post-49">
	
	<div class="entry-content">
		
		
<div class="wp-block-cover alignfull has-background-dim" style="min-height:500px;aspect-ratio:unset;"><img fetchpriority="high" decoding="async" width="1920" height="636" class="wp-block-cover__image-background wp-image-412" alt="" src="https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px.jpg" data-object-fit="cover" srcset="https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px.jpg 1920w, https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px-300x99.jpg 300w, https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px-1024x339.jpg 1024w, https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px-768x254.jpg 768w, https://sites.harvard.edu/airoldi/files/2022/06/smallgate2000px-1536x509.jpg 1536w" sizes="(max-width: 1920px) 100vw, 1920px" /><div class="wp-block-cover__inner-container is-layout-flow wp-block-cover-is-layout-flow">
<h1 class="has-text-align-center has-color-bg-color has-text-color wp-block-heading" style="font-size:49px"><meta charset="utf-8">Edoardo M Airoldi</h1>



<p class="has-text-align-center has-color-bg-color has-text-color"><meta charset="utf-8"></p>



<p></p>



<p class="has-color-bg-color has-text-color">Associate Professor of Statistics, Harvard University<br>Associate Faculty Member, The Broad Insititute of MIT &amp; Harvard<br>At Harvard, I lead a research group in Applied Statistical Methodology &amp; Data Science<br>&nbsp; (formerly known as the Harvard Laboratory for Applied Statistical Methodology &amp; Data Science, 2009-2017)</p>



<p class="has-color-bg-color has-text-color">mailing address: Department of Statistics, Harvard University, 1 Oxford Street, Cambridge, MA 02138, USA<br>phone: (617) 496-8318 &nbsp;<br>fax: (617) 496-8057<br>email: airoldi AT fas DOT harvard DOT edu</p>



<div class="wp-block-buttons is-content-justification-center is-layout-flex wp-block-buttons-is-layout-flex"></div>
</div></div>



<div class="wp-block-group"><div class="wp-block-group__inner-container is-layout-flow wp-block-group-is-layout-flow">
<h2 class="has-text-align-left wp-block-heading">bio</h2>



<p>I received a Ph.D. from Carnegie Mellon University in 2007, working at the intersection of statistical machine learning and computational social science with Stephen Fienberg and Kathleen Carley. My PhD thesis explored modeling approaches and inference strategies for analyzing social and biological networks. Till December 2008, I was a postdoctoral fellow in the Lewis-Sigler Institute for Integrative Genomics and the Department of Computer Science at Princeton University working with Olga Troyanskaya and David Botstein. We developed mechanistic models of regulation, leveraging of high-thoughput technology, to gain insights into aspects of cellular dynamics that are not directly measurable at the desired resolution, such as growth rate. I joined the Statistics Department at Harvard University in 2009. See my CV for more details.</p>
</div></div>



<div class="wp-block-group has-color-bg-alt-background-color has-background"><div class="wp-block-group__inner-container is-layout-flow wp-block-group-is-layout-flow">
<h2 class="wp-block-heading">research</h2>



<p>My research explores modeling, inferential, and other methodological issues that often arise in applied problems where network data (i.e., measurements on pairs of units, or tuples more generally) need to be considered, and standard statistical theory and methods are no longer adequate to support the goals of the analysis. More broadly, my research interests encompass statistical methodology and theory with application to molecular biology and computational social science, including Theory and methods for the analysis of network data</p>



<p>Design and analysis of experiments in the presence of interference</p>



<p>Inference from, and design and evaluation of, non-ignorable network sampling mechanisms</p>



<p>Geometry of inference in ill-posed inverse problems, including network tomography and contingency tables</p>



<p>Modeling and inference of regulation and signaling dynamics, including mass spectrometry and next-generation sequencing</p>



<p>Approximate inference strategies for data analysis at scale</p>



<p>Areas of technical interest include approximation theorems, inequalities, convex and combinatorial optimization, and geometry.</p>
</div></div>



<div class="wp-block-group"><div class="wp-block-group__inner-container is-layout-flow wp-block-group-is-layout-flow">
<h2 class="wp-block-heading">funding</h2>



<p>My work has been supported by an NSF Career Award, an ONR Young Investigator Award, a Sloan Research Fellowship, a Shutzer Fellowship, grants from the National Science Foundation, the National Institute of Health and the US Army, and gifts from the Broad Institute, Google, Microsoft Research, LinkedIn, Facebook, and AT&amp;T.</p>
</div></div>



<font size="+1"> &nbsp; <b>news</b></font>
      
	  <ul>
        <font size="+0"><!--li> Consider submitting your work to our workshop on <a href="http://tinyurl.com/NetworksNIPS2015">Networks in the Social and Information Sciences</a> held in conjunction with <a href="http://www.nips.cc/" target="_blank" rel="noopener">NIPS 2015</a>. <br><br-->
      
        <li> Postdoctoral positions available. (<a href="https://sites.harvard.edu/airoldi/airoldi_postdocAD_sep2016www.pdf" target="_blank" rel="noopener">postdoc flyer pdf</a>) <br><br>

		</li></font><li><font size="+0"> The Handbook of Mixex Membership Models (co-edited with David Blei, Elena Erosheva and Stephen Fienberg) is out. (<a href="https://www.crcpress.com/Handbook-of-Mixed-Membership-Models-and-Their-Applications/Airoldi-Blei-Erosheva-Fienberg/p/book/9781466504080" target="_blank" rel="noopener">www</a>) <br>
		  <br>
        
        <!--li>Research sponsored by NSF CAREER award IIS-1149662 is described <a href="https://sites.harvard.edu/airoldi/career.html" target="_blank" rel="noopener">here</a> <br><br> 

        <!--li> I recently co-organized a workshop on <a href="http://morrislab.med.utoronto.ca/~anna/networksnips2010/" target="_blank" rel="noopener">Networks Across Disciplines: Theory and Applications</a> held in conjunction with <a href="http://www.nips.cc/" target="_blank" rel="noopener">NIPS 2010</a>. <br><br-->
        
        <!--li> Check out the online proceedings of our workshop on <a href="http://research.yahoo.com/workshops/nipsgraphs2008/proceedings.html" target="_blank" rel="noopener">Analyzing Graphs: Theory and Applications</a>. It was held in December 2008, in conjunction with <a href="http://www.nips.cc/" target="_blank" rel="noopener">NIPS</a>. <br><br-->
      </font></li></ul><font size="+0">
    
      <font size="+1"> &nbsp; <b>recent and upcoming talks</b></font>
      
      </font><ul>

	    <li>Two model-assisted strategies for desgning  experiments on networks. <i>UC Berkeley (Econometrics &amp; Statistics of Networks)</i>, November 4-5, 2016, Berkeley, CA. <br>
	      <br>

	    </li><li>Optimal  design of experiments on social networks. <i>MIT (Conference on Digital Experimentation)</i>, October 14-15, 2016, Cambridge, MA. <br>
	      <br>

	    </li><li>Discussant. <i>University of Chicago (Machine Learning &amp; Economics)</i>, September 23-24, 2016, Chicago, IL. <br>
	      <br>

	    <!--li>Some fundamental ideas for causal inference on large networks. <i>Yale (Statistics)</i>, November 9, 2015, New Haven, CT. <br><br>

	    <li>Designing and analyzing experiments on large networks. <i>Yale (Biostatistics)</i>, September 29, 2015, New Haven, CT. <br>
	      <br>

	    <li>Some fundamental ideas for causal inference on large networks. <i>MIT (Stochastics & Statistics)</i>, September 25, 2015, Cambridge, MA. <br><br>
	         
	    <li>Some fundamental ideas for causal inference on large networks. <i>Cornell Day of Statistics</i>, September 11, 2015, Itahca, NY. <br><br>
	      
	    <li><font size="+0"> Some fundamental ideas for causal inference on large networks. <i>Harvard University (Center of Mathematical Sciences and Applications)</i>, August 24-26, 2015, Cambridge, MA. <br>
	      <br>

	    </font>
	    <li> Optimal design of experiments in the presence of network interference. <i>Microsoft Research New England (Statistics & Data Science Symposium)</i>, June 11-12, 2015, Cambridge, MA. <br>
	      <br>

	    <li><font size="+0"> Statistics and machine learning challenges in the analysis of large networks. <i>University of Washington (Computer Science)</i>, June 2, 2015, Seattle, WA. <br>
	      <br>

	    </font><li><font size="+0"> Optimal design of experiments in the presence of network interference. <i>University of Washington (Statistics)</i>, June 1, 2015, Seattle, WA. <br>
	      <br>

	    </font><!--li><font size="+0"> Optimal design of experiments in the presence of network interference. <i> University of Montreal (Centre de Recherches Matematiques)</i>, May 4 - 8, 2015, Montreal, Canada. <br>
	      <br-->

	    <!--/font><li><font size="+0"> Optimal design of experiments in the presence of network interference. <i>University of Chicago (Booth School of Business)</i>, April 2, 2015, Chicago, IL. <br>
	      <br>

	    </font><li>Optimal design of experiments in the presence of network interference. <i>National Academy of Sciences (Sackler Symposium on Drawing Causal Inference from Big Data)</i>, March 26-27, 2015, Washington, DC.<br><br-->

	    <!--/font><li><font size="+0"> Design and analysis of experiments in the presence of network interference. <i>University of Connecticut (Statistics)</i>, March 11, 2015, Storrs, CT. <br>
	      <br>
          
	    </font><li><font size="+0"> Valid statistical analyses and reproducible science in the era of high-throughput biology. <i>Harvard School of Public Health (PQG Short Course)</i>, March 10, 2015, Boston, MA. <br>
	      <br>          

	    </font><li><font size="+0"> Statistical and machine learning challenges in the analysis of large networks. <i>UC Berkeley (IEOR)</i>, February 13, 2015, Berkeley, CA. <br>
	      <br>

	    </font><li><font size="+0"> Statistical and machine learning challenges in the analysis of large networks. <i>Northeastern University (Mathematics)</i>, February 10, 2015, Boston, MA. <br>
	      <br>

	    </font><li><font size="+0"> Design and analysis of experiments in the presence of network interference. <i>Princeton University (Political Science)</i>, December 5, 2014, Princeton, NJ. <br>
	      <br>

	    </font><li><font size="+0"> Statistical and machine learning challenges in the analysis of large networks. <i>Princeton University (Computer Science)</i>, December 2, 2014, Princeton, NJ. <br>
	      <br>

	    </font><li><font size="+0"> Design and analysis of experiments in the presence of network interference. <i>Yale University (Statistics)</i>, December 2, 2013, New Haven, CT. <br>
	      <br>

	    </font><li><font size="+0"> Design and analysis of experiments with interfering units. <i>Simons Institute for the Theory of Computing</i>, November 18-21, 2013, UC Berkeley, CA. <br><br>

  	    <!--li> Inference from non-ignorable sampling designs. <i>DIMACS</i>, November 7-8, 2013, Piscataway, NJ. <br><br>

	    <li> Inference and algorithms for high-throughput biology. <i>Italian Embassy</i>, October 29, 2013, Washington, DC. <br><br>

	<li> Estimating causal effects with interfering units. <i>SAMSI</i>, October 21-23, 2013, RTP, NC. <br><br>
      
	<!--li> TBD. <i>Penn State University (Statistics)</i>, Spring 2012, University Park, PA.<br><br>

	<!--li> TBD. <i>University of Chicago (Statistics Group at Booth Business School)</i>, Spring 2012, Chicago, IL.<br><br-->
 
	<!--li> Graphlets: A scalable, multi-scale decomposition for large social and information networks. <i>Stanford University (Algorithms for Modern Massive Data Sets)</i>, July 10-13, 2012, Palo Alto, CA.<br><br>

	<li> Statistical analysis of populations with interacting and interfering units. <i>Modern Modeling Methods Conference</i>, May 23, 2012, Storrs, CT.<br><br>

	<li> Valid inference in high-throughput biology and social media marketing. <i>New England Machine Learning Day (Microsoft Research)</i>, May 16, 2012, Cambridge, MA.<br><br>

	<!--li> New tools for systems-level analysis of regulation and signaling dynamics. <i>International Biometric Society Annual Meeting (ENAR)</i>, April 1-4, 2012, Washington, DC.<br><br>

	<!--li> Statistical analysis of populations with interacting and interfering units. <i>University of Chicago (Econometrics & Statistics at Booth)</i>, March 30, 2012, Chicago, IL.<br><br>

	<li> Graphlets: A scalable, multi-scale decomposition for large social and information networks. <i>Rensselaer Polytechnic Institute (Computer Science)</i>, March 23, 2012, Troy, NY.<br>
	  <br-->

	<!--li> Inference from non-ignorable network sampling designs. <i>Harvard University (Institute for Quantitative Social Sciences)</i>, March 21, 2012, Cambridge, MA.<br><br-->

	<!--li> Inference and algorithms for high-throughput biology. <i>Microsoft Research</i>, March 20, 2012, Redmond, WA.<br><br>

    <!--li> Statistical methods for high-throughput biology. <i>Duke University (Institute for Genome Sciences)</i>, March 12, 2012, Durham, NC.<br><br-->

	<!--li> Statistical methods for high-throughput biology. <i>MIT (Bioinformatics at Math & CSAIL)</i>, February 22, 2012, Cambridge, MA.<br><br>

	<li> Statistical analysis of populations with interacting and interfering units. <i>Brown University (Applied Mathematics)</i>, February 1, 2012, Providence, RI.<br><br>

	<li> Statistical analysis of populations with interacting and interfering units. <i>Duke University (Statistics)</i>, January 20, 2012, Durham, NC.<br><br>

	<!--li> Statistical analysis of populations with interacting and interfering units. <i>University of Texas at Austin (Statistics and Scientific Computing)</i>, November 11, 2011, Austin, TX.<br><br>

	<!--li> Tutorial on statistical analysis of populations with interacting and interfering units. <i>University of Minnesota (Institute for Mathematics and its Applications)</i>, October 24, 2011, Minneapolis, MN.<br><br>

	<!--li> TBD. <i>Analysis of Dynamic Social and Technological Networks</i>, November 8-11th 2011, Dagstuhl, Germany.<br><br-->

	<!--li> Estimating latent processes on a graph from indirect measurements. <i>MIT Lincoln Labs (Endicott House)</i>, August 9-10th 2011, Needham, MA.<br><br>

    <li> Statistical models of complex networks. <i>Joint Statistical Meetings</i>, July 30th-August 4th, Miami Beach, FL.<br><br>
    
    <li> A geometric approach to estimation of matrix-variate processes from indirect measurements. <i>International Statistics Forum</i>, July 17-19th 2011, Wu Xi, China.<br><br>

    <!--li> TBD. <i>IMS Pacific Asia Rim Meeting</i>, July 4-6th 2011, Tokyo, Japan.<br><br-->

    <!--li> Estimating latent processes on a graph from indirect data. <i>Applied Statistics Symposium</i>, June 29th 2011, New York, NY.<br><br>

    <li> Social information, structure and interference. <i>Universita degli Studi Federico II</i>, June 25th 2011, Naples, Italy.<br><br>

    <!--li>Estimating latent processes on a graph from indirect data. <i>Harvard Workshop on Statistics and Engineering</i>, June 6th 2011, Cambridge, MA.<br><br-->

	<!--li> Evaluating topic models. <i>Harvard Institute for Quantiative Social Sciences</i>, May 20th 2011, Cambridge, MA.<br><br>

	<li> Analysis and design of RNA sequencing experiments. <i>MD Anderson Cancer Center</i>, May 11th 2011, Houston, TX.<br><br>

	<li> Estimating latent processes on a graph from indirect measurements. <i>Boston University (Mathematics & Statistics)</i>, April 28th 2011, Boston, MA.<br><br>

	<li> New tools for systems-level analysis of regulation and signaling dynamics. <i>The Broad Institute</i>, April 20th 2011, Cambridge, MA.<br><br>

	<li> Deconvolution of mixing time series on a network. <i>New England Statistics Symposium</i>, April 16th 2011, Storrs, CT.<br><br>

	<li> Estimating latent processes on a graph from indirect measurements. <i>University of Massachusetts, Amherst (Statistics & Computational Social Science)</i>, April 8th 2011, Amherst, MA. <br><br>

	<li> Deconvolution of mixing time series on a network. <i>Massachusetts Insititute of Technology (McGovern & CBCL)</i>, March 28th 2011, Cambridge, MA.<br><br>

	<li> New tools for systems-level analysis of regulation and signaling dynamics. <i>Harvard School of Public Health</i>, February 28th 2011, Boston, MA.<br><br>

	<li> Systems-level analysis of regulation and signaling dynamics. <i>Microsoft Research (Computational Aspects of Biological Information)</i>, December 8th 2010, Cambridge, MA. <br><br>

	<!--li> Network representation and modeling. <i>Statistical and Applied Mathematical Sciences Institute (Complex Networks program)</i>, October 20th 2010, RTP, NC. <br><br>

	<li> Dynamic network tomography. <i>Statistical and Applied Mathematical Sciences Institute (Complex Networks program)</i>, August 31st 2010, RTP, NC. <br><br>

	<li> Modeling approaches for analyzing complex networks. <i>Institute for Pure and Applied Mathematics at UCLA (Network Analysis for the Humanities)</i>, August 25th 2010, Los Angeles, CA. <br><br>

	<li> Estimating isoforms abundance from paired-end mRNA sequencing data. <i>Joint Statistical Metings (JSM)</i>, August 2nd 2010, Vancouver, Canada. <br><br>

	<!--li> Network statistics and processes on networks. <i>International Conference on Statistics and Society</i>, July 10th 2010, Beijing, China. <br><br>

	<li> Mini-course on social network analysis. (3 hours) <i>PIMS Summer School on Social Networks</i>, June 24th 2010, Whistler, Canada. <br><br>

	<li> Three statistical problems in complex networks. <i>Harvard Medical School</i>, June 17th 2010, Boston, MA. <br><br>

	<li> Representation and Bayesian analysis of integer-valued networks. <i>Valencia International Meeting on Bayesian Statistics (ISBA)</i>, June 7th 2010, Benidorm, Spain. <br><br>

	<li> Network analysis and tomography. <i>Oxford University</i>, May 20th 2010, Oxford, UK. <br><br>

	<li> A statistical perspective on complex networks. <i>University of Cambridge (Computer Science Laboratory)</i>, May 19th 2010, Cambridge, UK. <br><br>
    
	<li> Mini-course on statistical network analysis. (3 hours) <i>New England Statistics Symposium</i>, April 17th 2010, Cambridge, MA. <br><br>

	<li> Elements of statistical network analysis. <i>MIT Lincoln Labs (Endicott house)</i>, April 13th 2010, Needham, MA. <br><br>

	<li> Elements of statistical graph theory. <i>Northeastern University (Center for Complex Networks Research)</i>, December 2nd 2009, Boston, MA. <br><br>

	<li> A systems-level analysis of cellular proliferation. <i>Harvard University (School of Engineering and Applied Sciences)</i>, November 19th 2009, Cambridge, MA. <br><br>

	<!--li> Elements of statistical graph theory. <i>University of Connecticut (Department of Statistics)</i>, November 18th 2009, Storrs, CT. <br><br>

	<li> A statistical perspective on complex networks. <i>Harvard University (Institute for Quantitative Social Sciences)</i>, November 4th 2009, Cambridge, MA. <br><br>

	<li> Statistical analysis of graphs and node attributes. <i>International Meeting of the Psychometic Society</i>, July 23rd 2009, Cambrdge, UK. <br><br>

	<li> The exchangeable graph model. <i>University College Dublin</i>, June 16th 2009, Dublin, Ireland. <br><br>

	<li> A statistical perspective on cellular growth. <i>ETH meeting on Statistical Advances in Genome-scale Data Analysis</i>, May 5th, Ascona, Switzerland. <br><br>

	<li> The exchangeable graph model. <i>Harvard University (Radcliffe Institute for Advanced Studies)</i>, February 6-7th 2009, Cambridge, MA. <br><br>

	<li> The exchangeable graph model. <i>NIPS Workshop on Analyzing Graphs</i>, December 12th 2008, Whistler, Canada. <br><br>

	<li> Statistical analysis of complex biological systems. <i>University of New Mexico</i>, December 6-8th 2008, Albuquerque, NM. <br><br>

	<li> The exchangeable graph model. <i>Santa Fe Institute</i>, December 3-5th 2008, Santa Fe, NM. <br><br>

	<li> A statistical perspective on cellular growth. <i>Fred Hutchinson Cancer Research Center</i>, December 1-2th 2008, Seattle, WA. <br><br>

	<li> A statistical perspective on cellular growth. <i>New York Academy of Sciences (Machine Learning Symposium)</i>, October 10th 2008, New York, NY. <br><br>

	<!--li> Statistical analysis of complex biological systems. <i>Rutgers University</i>, September 24th 2008, New Brunswick, NJ. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>Joint Statistical Meetings (JSM)</i>, August 6th 2008, Denver, CO. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>International Sociaty for Bayesian Analysis (ISBA)</i>, July 23rd 2008, Hamilton Island, Australia. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>Cambridge University (JSM)</i>, June 25th 2008, Cambridge, United Kingdom. <br><br>

	<li> The exchangeable graph model for statistical network analysis. <i>Isaac Newton Institute</i>, June 24th 2008, Cambridge, United Kingdom. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>University of Pennsylvania</i>, March 5th 2008, Philadelphia, PA. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>Harvard University</i>, February 27th 2008, Cambridge, MA. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>University of Maryland</i>, February 19rd 2008, College Park, MD. <br><br>

	<li> Bayesian analysis of complex biological systems. <i>Duke University</i>, February 11th 2008, Durham, NC. <br><br>

	<li> Statistical models for static & dynamic network analysis. <i>Columbia University</i>, November 26th 2007, New York, NY. <br><br>

	<li> The exchangeable graph model. <i>Institute for Pure and Applied Mathematics (UCLA)</i>, November 6th, 2007, Los Angeles, CA. <br><br>

	<li> Mixed-membership stochastic blockmodels. <i>Joint Statistical Meetings (JSM)</i>, July 30th 2007, Salt Lake City UT. <br><br>

	<li> Biological context analysis of gene expression data. <i>International Biometrics Society (ENAR)</i>, March 13th 2007, Atlanta GA. <br><br>

	<li> Dynamic network analysis. <i>Institute for Operations Research and the Management Sciences (INFORMS)</i>, November 7th 2006, Pittsburgh PA. <br><br>

      	<li> Latent aspects models with applications to sequences & networks. <i>CSD & Lewis-Sigler Institute, Princeton University</i>, April 4th 2006, Princeton NJ. <br><br-->
      </li></ul>

      <font size="+1"> &nbsp; <b>pre-prints</b></font>
      
      <ul>	

     	<font size="+0"><!--li> Handbook of Mixed Membership Models and Their Applications
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM.pdf" target="_blank" rel="noopener">book</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Preface_MMM2014.pdf" target="_blank" rel="noopener">preface</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch1_MMM2014.pdf" target="_blank" rel="noopener">ch1</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch2_MMM2014.pdf" target="_blank" rel="noopener">ch2</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch3_MMM2014.pdf" target="_blank" rel="noopener">ch3</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch4_MMM2014.pdf" target="_blank" rel="noopener">ch4</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch5_MMM2014.pdf" target="_blank" rel="noopener">ch5</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch6_MMM2014.pdf" target="_blank" rel="noopener">ch6</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch7_MMM2014.pdf" target="_blank" rel="noopener">ch7</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch8_MMM2014.pdf" target="_blank" rel="noopener">ch8</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch9_MMM2014.pdf" target="_blank" rel="noopener">ch9</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch10_MMM2014.pdf" target="_blank" rel="noopener">ch10</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch11_MMM2014.pdf" target="_blank" rel="noopener">ch11</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch12_MMM2014.pdf" target="_blank" rel="noopener">ch12</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch13_MMM2014.pdf" target="_blank" rel="noopener">ch13</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch14_MMM2014.pdf" target="_blank" rel="noopener">ch14</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch15_MMM2014.pdf" target="_blank" rel="noopener">ch15</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch16_MMM2014.pdf" target="_blank" rel="noopener">ch16</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch17_MMM2014.pdf" target="_blank" rel="noopener">ch17</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch18_MMM2014.pdf" target="_blank" rel="noopener">ch18</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch19_MMM2014.pdf" target="_blank" rel="noopener">ch19</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch20_MMM2014.pdf" target="_blank" rel="noopener">ch20</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch21_MMM2014.pdf" target="_blank" rel="noopener">ch21</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch22_MMM2014.pdf" target="_blank" rel="noopener">ch22</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch23_MMM2014.pdf" target="_blank" rel="noopener">ch23</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch24_MMM2014.pdf" target="_blank" rel="noopener">ch24</a>)
        	 (<a href="https://sites.harvard.edu/airoldi/b02.AiroldiBleiEroshevaFienberg2014HandbookMMM/Ch25_MMM2014.pdf" target="_blank" rel="noopener">ch25</a>)
              <br><br-->
    
	<li> Optimal model-assisted design of experiments for network correlated outcomes suggests new notions of network balance. 
             (<a href="http://arxiv.org/abs/1507.00803" target="_blank" rel="noopener">pdf</a>) <br><br>          

		</li><li> Valid inference from non-ignorable network sampling designs. 
             (<a href="http://arxiv.org/abs/1401.4718" target="_blank" rel="noopener">pdf</a>) <br><br>          

	</li><li> Implicit stochastic approximation.
             (<a href="https://arxiv.org/abs/1510.00967" target="_blank" rel="noopener">pdf</a>) <br><br>

	</li><li> Non-standard conditionally specified models for non-ignorable missing data. 
             (<a href="http://arxiv.org/abs/1603.06045" target="_blank" rel="noopener">pdf</a>) <br><br>          

	</li><li> Causal inference for ordinal outcomes. 
             (<a href="http://arxiv.org/abs/1501.01234" target="_blank" rel="noopener">pdf</a>) <br><br>          

		<!--li> Inference of network summary statistics through network denoising. 
             (<a href="http://arxiv.org/abs/1310.0423" target="_blank" rel="noopener">pdf</a>) <br><br-->          

		<!--li> Sharp total variation bounds for finitely exchangeable arrays. 
             (<a href="http://arxiv.org/abs/1407.6092" target="_blank" rel="noopener">pdf</a>) <br><br-->          

		</li></font><li><font size="+0"> The geometry of 2&#215;2 contingency tables.
             <!-- (<a href="https://sites.harvard.edu/airoldi/AiroldiFienbergHashimotoSlavkovic2014StatSci.pdf" target="_blank" rel="noopener">pdf</a>) --> 
             (<a href="https://sites.harvard.edu/airoldi/src/tetra3d.jar" target="_blank" rel="noopener">java app</a>) 
             (<a href="https://sites.harvard.edu/airoldi/src/tetra3d.v31.zip" target="_blank" rel="noopener">source code</a>) <br><br>
             
		<!--li> Learning modular structures from network data and node variables. 
             (<a href="http://arxiv.org/abs/1405.2566" target="_blank" rel="noopener">pdf</a>)
             (<font color=red>Best student paper award, NESS 2014</font>)        
             (a shorter version appeared at ICML 2014) <br><br-->          

		<!--li> Estimating cellular pathways from an ensemble of heterogeneous data sources. 
             (<a href="http://biorxiv.org/content/early/2014/06/23/006478" target="_blank" rel="noopener">pdf</a>) <br><br-->          

		<!--li> A model of peptide charge and modification states for accurate quantification of low concentration proteins. <br><br>          

		<li> Template-based methods for the estimation of nucleosoem positioning at base-pair resolution. 
             (<font color=red>MSR best student paper award, NESS 2012</font>) <br><br>          

		<li> Variable stoichiometry among core ribosomal proteins. 
             (<a href="http://biorxiv.org/content/early/2014/05/26/005553" target="_blank" rel="noopener">pdf</a>) <br><br-->          

      </font></li></ul><font size="+0">

   	    <!--li> Airoldi, E.M., Fienberg, S.E., Joutard, C., Love, T.M., & Shringarpure, S.
        On the granularity of semantic themes in a database of scientific publications.
        (<a href="http://arxiv.org/" target="_blank" rel="noopener"><i>pdf soon</i></a>) <br><br-->

	    <!--li> *Airoldi, E.M., *Markowetz, F., Blei, D.M., & Troyanskaya, O.G. (manuscript).
	    (an <a href="http://www.cs.ubc.ca/~murphyk/nips07NetworkWorkshop/abstracts/airoldi.pathways.pdf" target="_blank" rel="noopener">earlier version</a> appeared at <i>Statistical Models of Networks</i> workshop, NIPS 2007) <br><br>

	    <li> Airoldi, E.M., Fienberg, S.E., & Krackhardt, D. (manuscript).
	    (an <a href="http://www.cs.ubc.ca/~murphyk/nips07NetworkWorkshop/abstracts/airoldi.soc_failure.pdf" target="_blank" rel="noopener">earlier version</a> appeared at <i>Statistical Models of Networks</i> workshop, NIPS 2007) <br><br-->
	
      <font size="+1"> &nbsp; <b>selected publications</b></font>
      (see my <a href="https://sites.harvard.edu/airoldi/airoldi_cv_sep2016www.pdf" target="_blank" rel="noopener">CV</a>
      or <a href="http://scholar.google.com/citations?user=XKCyZk0AAAAJ&amp;hl=en&amp;oi=ao" target="_blank" rel="noopener">Google Scholar</a>
         <!--a href="http://scholar.google.com/citations?sortby=pubdate&hl=en&user=XKCyZk0AAAAJ&pagesize=100&view_op=list_works" target="_blank">Google Scholar</a-->
      for more publications and bibliographic details) <br>
      <br>
      <font size="+0">

      &nbsp; <u><b>statistical inference strategies for massive data sets</b></u>
      <ul>	

		<li> Asymptotic and finite-sample properties of estimators based on stochastic gradients.
	      <i>Annals of Statistics</i>, 2016.
     	  (<a href="http://www.e-publications.org/ims/submission/AOS/user/submissionFile/23916?confirm=44d0228d" target="_blank" rel="noopener">pdf</a>) <br><br> 

		</li><li> Scalable estimation strategies based on stochastic approximations: Classical results and new insights.
          <i>Statistics and Computing</i>, 2015.
             (<a href="http://link.springer.com/article/10.1007/s11222-015-9560-y" target="_blank" rel="noopener">pdf</a>) <br>
             
      </li></ul>

      &nbsp; <u><b>theory and methods for network data analysis</b></u>
      <ul>	

		<li> Geometric representations of distributions on hypergraphs.
	 	     <i>Journal of the American Statistical Association</i>, 2016.
     		 (<a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2016.1141686" target="_blank" rel="noopener">pdf</a>) 
             <br><br>

		</li><li> Nonparametric estimation and testing of exchangeable graph models.
             <i>Journal of Machine Learning Research, W&amp;CP</i>, 2014.
             (<a href="http://jmlr.org/proceedings/papers/v33/yang14c.html" target="_blank" rel="noopener">pdf</a>)
             <br><br>

		</li><li> A consistent total variation estimator for exchangeable graph models.
             (<a href="http://arxiv.org/abs/1402.1888" target="_blank" rel="noopener">pdf</a>)
             (a shorter version appeared at <i>ICML</i> 2014) <br><br>

		</li><li> Stochastic blockmodel approximation of a graphon: Theory and consistent estimation.
             <i>NIPS</i>, 2013.
             (<a href="http://papers.nips.cc/paper/5047-stochastic-blockmodel-approximation-of-a-graphon-theory-and-consistent-estimation.pdf" target="_blank" rel="noopener">pdf</a>) <br><br>
         
		</li><li> Stochastic blockmodels with growing number of classes.
             <i>Biometrika</i>, 2012.
             <!-- (<a href="http://biomet.oxfordjournals.org/content/99/2/273.abstract" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j023.ChoiWolfeAiroldi2012Biometrika.pdf" target="_blank" rel="noopener">pdf</a>) <br><br>
         
		</li><li> Confidence sets for network structure.
             <i>Statistical Analysis and Data Mining</i>, 2011.
             <!-- (<a href="http://dx.doi.org/10.1002/sam.10136" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j022.AiroldiChoiWolfe2011SADM.pdf" target="_blank" rel="noopener">pdf</a>)
             (a shorter version appeared at <i>NIPS</i> 2011) <br><br>

		</li><li> Graphlets decomposition of a weighted network.
             <i>Journal of Machine Learning Research, W&amp;CP (AISTAT)</i>, 2011.
             <!-- (<a href="http://arxiv.org/abs/1203.2821" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j024.AzariAiroldi2012JMLRWCP.pdf" target="_blank" rel="noopener">pdf</a>)
             (<font color="red">MSR best student paper award, NESS 2012</font>) <br><br>

		<!--li> Network sampling and classification: An investigation of network model representations.
             <i>Decision Support Systems</i>, 2011.
             <!-- (<a href="http://dx.doi.org/10.1016/j.dss.2011.02.014" target="_blank" rel="noopener">link</a>)
             (<a href="https://sites.harvard.edu/airoldi/j020.AiroldiBaiCarley2011DSS.pdf" target="_blank" rel="noopener">pdf</a>) <br><br-->
         
		</li><li> A survey of statistical network models.
             <i>Foundations and Trends in Machine Learning</i>, 2010. 
             <!-- (<a href="http://arxiv.org/abs/0912.5410/" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j012.GoldenbergZhengFienbergAiroldi2010FTML.pdf" target="_blank" rel="noopener">pdf</a>) <br><br>

	    </li><li> Mixed-membership stochastic blockmodels.
             <i>Journal of Machine Learning Research</i>, 2008.
	         <!-- (<a href="http://jmlr.csail.mit.edu/papers/volume9/airoldi08a/airoldi08a.pdf" target="_blank" rel="noopener">link</a>) -->
	         (<a href="https://sites.harvard.edu/airoldi/j008.AiroldiBleiFienbergXing2008JMLR.pdf" target="_blank" rel="noopener">pdf</a>)
	         (<a href="http://cran.r-project.org/web/packages/lda/" target="_blank" rel="noopener">r code</a>)
	         (<a href="http://github.com/premgopalan/svinet" target="_blank" rel="noopener">fast code</a>)
             (<font color="red">John Van Ryzin award, 2006</font>)  <br>
         
	    <!--li> Statistical Network Analysis: Models, Issues & New Directions.
             <em>Lecture Notes in Computer Science</em>, 2007.
	         (<a href="http://dx.doi.org/10.1007/978-3-540-73133-7" target="_blank" rel="noopener">pdf</a>) <br><br>

		<li> Sampling algorithms for pure network topologies: Stability and separability of metric embeddings.        
             <i>ACM KDD Explorations</i>, 2005. <br-->    
      </li></ul>

      &nbsp; <u><b>geometry and inference in ill-posed inverse problems</b></u></font>
      <ul>	

		<li> Estimating latent processes on a network from indirect measurements.
             <i>Journal of the American Statistical Association</i>, 2013.
             <!-- (<a href="http://arxiv.org/abs/1212.0178" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j027.AiroldiBlocker2013JASA.pdf" target="_blank" rel="noopener">pdf</a>) 
             (<a href="https://sites.harvard.edu/airoldi/j027s.AiroldiBlocker2013JASA.pdf" target="_blank" rel="noopener">supp</a>) 
             (<a href="http://cran.r-project.org/web/packages/networkTomography/" target="_blank" rel="noopener">r code</a>)              
             (<font color="red">IBM best student paper award, NESS 2011</font>) <br><br>

	    </li><li> Polytope samplers for inference in ill-posed inverse problems.
             <i>Journal of Machine Learning Research, W&amp;CP (AISTAT)</i>, 2011.
             <!-- (<a href="http://jmlr.csail.mit.edu/proceedings/papers/v15/airoldi11a/airoldi11a.pdf" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j018.AiroldiHaas2011JMLRWCP.pdf" target="_blank" rel="noopener">pdf</a>) <br><br>
          
		</li><li> Tree preserving embedding
             <i>Proceedings of the National Academy of Sciences</i>, 2011.
             <!-- (<a href="http://www.pnas.org/content/108/41/16916.short" target="_blank" rel="noopener">link</a>) --> 
             (<a href="https://sites.harvard.edu/airoldi/j021.ShiehHashimotoAiroldi2011PNAS.pdf" target="_blank" rel="noopener">pdf</a>) 
             (<a href="http://cran.r-project.org/web/packages/tpe/" target="_blank" rel="noopener">r code</a>) 
             (a shorter version appeared at <i>ICML</i> 2011) <br>
          
		<!--li> Deconvolution of mixing time series on a graph.
             <i>Uncertainty in Artificial Intelligence (UAI)</i>, 2011.
             (<a href="http://uai.sis.pitt.edu/papers/11/p51-blocker.pdf" target="_blank" rel="noopener">pdf</a>) 
             (<font color=red>IBM best student paper award, NESS 2011</font>) <br><br-->

        <!--li> Network tomography with particle filters.          
             <i>ACM International Conference on Knowledge Discovery &amp; Data Mining (KDD)</i>, 2004.
	         (<a href="http://portal.acm.org/citation.cfm?id=1014052.1014059" target="_blank" rel="noopener">pdf</a>) <br-->    
      </li></ul>

      &nbsp; <u><b>modeling and inference in high-throughput biology</b></u></font>
      <ul>	

		<li> Template-based models for genome-wide analysis of next-generation sequencing data at base-pair resolution.
 		     <i>Journal of the American Statistical Association</i>, 2016.
     		 (<a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2016.1141095" target="_blank" rel="noopener">pdf</a>) <br><br> 

		</li><li> Estimating cellular pathways from an ensemble of heterogeneous data sources.
		     <i>Annals of Applied Statistics</i>, 2015.
     		 (<a href="https://sites.fas.harvard.edu/~airoldi/" target="_blank" rel="noopener">pdf</a>) <br><br>

		</li><li> Accounting for experimental noise reveals that mRNA levels, amplified by post-transcriptional processes, largely determine steady-state protein levels in yeast.
	     	 <i>PLoS Genetics</i>, 2015.
    	 	 (<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005206" target="_blank" rel="noopener">pdf</a>)
             (<a href="http://drummondlab.org/papers/paper/mrna-protein/" target="_blank" rel="noopener">data</a>)  <br><br>         	

		</li><li> Estimating a structured covariance matrix from multi-lab measurements in high-throughput biology.          
             <i>Journal of the American Statistical Association</i>, 2015. 
 			 (<a href="http://amstat.tandfonline.com/doi/full/10.1080/01621459.2014.964404" target="_blank" rel="noopener">pdf</a>)
             (<font color="red">IBM best student paper award, NESS 2013</font>)          
             (<font color="red">W. J. Youden Award in Interlaboratory Testing, ASA 2015</font>) <br><br>          

		</li><li> Generalized species sampling priors with latent beta reinforcements.
             <i>Journal of the American Statistical Association</i>, 2014. 
             (<a href="http://arxiv.org/abs/1012.0866/" target="_blank" rel="noopener">pdf</a>) <br><br> 

 	    </li><li> Multi-way blockmodels for analyzing coordinated high-dimensional responses.
             <i>Annals of Applied Statistics</i>, 2013. 
             (<a href="https://sites.harvard.edu/airoldi/j029.AiroldiWangLin2013AoAS.pdf" target="_blank" rel="noopener">pdf</a>)                    
             (<a href="https://sites.harvard.edu/airoldi/j029s.AiroldiWangLin2013AoAS.pdf" target="_blank" rel="noopener">supp</a>) <br><br>                      

	    </li><li> Analysis and design of RNA sequencing experiments for identifying mRNA isoform regulation.
             <i>Nature Methods</i>, 2010. 
             <!-- (<a href="http://www.nature.com/nmeth/journal/v7/n12/abs/nmeth.1528.html" target="_blank" rel="noopener">pdf</a>)
             (<a href="http://www.nature.com/nmeth/journal/v7/n12/extref/nmeth.1528-S1.pdf" target="_blank" rel="noopener">supplement</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j016.KatzWangAiroldiBurge2010NatMeth.pdf" target="_blank" rel="noopener">pdf</a>)
             (<a href="https://sites.harvard.edu/airoldi/j016s.KatzWangAiroldiBurge2010NatMeth.pdf" target="_blank" rel="noopener">supp</a>)           
             (<a href="http://genes.mit.edu/burgelab/miso/software.html" target="_blank" rel="noopener">code</a>) <br><br>           
         
 	    </li><li> Ranking relations using analogies in biological and information networks.
             <i>Annals of Applied Statistics</i>, 2010. 
             <!-- (<a href="http://arxiv.org/abs/0912.5193/" target="_blank" rel="noopener">link</a>) -->         
             (<a href="https://sites.harvard.edu/airoldi/j013.SilvaHellerGhahramaniAiroldi2010AoAS.pdf" target="_blank" rel="noopener">pdf</a>)
             (<a href="http://dx.doi.org/10.1214/09-AOAS321SUPP" target="_blank" rel="noopener">code</a>) <br><br>                      
          
	    </li><li> Predicting cellular growth from gene expression signatures.
             <i>PLoS Computational Biology</i>, 2009. 
             <!-- (<a href="http://dx.doi.org/10.1371/journal.pcbi.1000257" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j009.Airoldi_Troyanskaya2009PLoSCBi.pdf" target="_blank" rel="noopener">pdf</a>)
             (<a href="http://reducio.princeton.edu/growthrate/" target="_blank" rel="noopener">code &amp; data</a>)
	         (a shorter version appeared at NIPS 2008) <br>

	    <!--li> Getting started in probabilistic graphical models.
             <i>PLoS Computational Biology</i>, 2007.
	         <!-- (<a href="http://dx.doi.org/10.1371/journal.pcbi.0030252" target="_blank" rel="noopener">link</a>)
	         (<a href="https://sites.harvard.edu/airoldi/j006.Airoldi2007PLoSCBi.pdf" target="_blank" rel="noopener">pdf</a>) <br-->
      </li></ul>

      &nbsp; <u><b>applications in molecular biology</b></u><ul>	

		<font size="+0">
		<li> Defining the essential function of yeast Hsf1 reveals a compact transcriptional program for maintaining eukaryotic proteostasis. 
             <i>Molecular Cell</i>, 2016.
             (<a href="http://www.sciencedirect.com/science/article/pii/S1097276516301514" target="_blank" rel="noopener">pdf</a>)
	         (<a href="https://sites.harvard.edu/airoldi/ref/Triandafillou_and_Drummond_Mol_Cell_Preview_July_7_2016.pdf" target="_blank" rel="noopener"><font color="red">preview</font></a>) <br><br>
             
		</li><li> Reversible, specific, active aggregates of endogenous proteins assemble upon heat stress. 
             <i>Cell</i>, 2015.
             (<a href="http://www.cell.com/cell/abstract/S0092-8674(15)01093-4" target="_blank" rel="noopener">pdf</a>) <br><br>

		</li><li> Differential stoichiometry among core Ribosomal proteins. 
             <i>Cell Reports</i>, 2015.
             (<a href="http://biorxiv.org/content/early/2015/09/19/005553" target="_blank" rel="noopener">pdf</a>) <br><br>
             
        </li><li> Musashi proteins are post-transcriptional regulators of the epithelial-luminal cell state. 
             <i>eLife</i>, 2014.
             (<a href="http://elifesciences.org/content/early/2014/11/07/eLife.03915" target="_blank" rel="noopener">pdf</a>)
	         (<a href="https://sites.harvard.edu/airoldi/ref/MsiPaper_in_Science-2015-246.pdf" target="_blank" rel="noopener"><font color="red">editor&#8217;s choice in Science</font></a>) <br><br>

		<!--li> Quantifying condition-dependent intracellular protein levels enables high-precision fitness estimates.
             <i>PLoS One</i>, 2013.
             (<a href="https://sites.harvard.edu/airoldi/j030.Samerotte_Drummond2013PLoSOne.pdf" target="_blank" rel="noopener">pdf</a>) <br><br>          

		<!--li> Estimating selection on synonymous codon usage from noisy experimental data.
             <i>Molecular Biology and Evolution</i>, 2013.
             (<a href="http://mbe.oxfordjournals.org/content/30/6/1438.short" target="_blank" rel="noopener">pdf</a>)          
             (<a href="http://f1000.com/prime/718042321" target="_blank" rel="noopener"><font color=red>F1000</font></a>) <br><br-->          

		<!--li> A conserved cell growth cycle can account for the environmental stress responses of divergent eukaryotes.
             <i>Molecular Biology of the Cell</i>, 2012.
             <!-- (<a href="http://www.molbiolcell.org/content/23/10/1986.abstract" target="_blank" rel="noopener">link</a>)         
             (<a href="https://sites.harvard.edu/airoldi/j025.SlavovAiroldivanOudenaardenBotstein2012MBC.pdf" target="_blank" rel="noopener">pdf</a>) <br><br-->          

		<!--li> Small sets of interacting proteins suggest functional linkage mechanisms via Bayesian analogical reasoning.
             <i>Bioinformatics</i>, 2011.
             (<a href="http://bioinformatics.oxfordjournals.org/content/27/13/i374.abstract" target="_blank" rel="noopener">pdf</a>) <br><br>

	    <li> Mapping dynamic histone acetylation patterns to gene expression in Nanog-depleted murine embryonic stem cells.
             <i>PLoS Computational Biology</i>, 2010.
             (<a href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1001034" target="_blank" rel="noopener">pdf</a>)<br><br>
          
	    <li> Aneuploidy prediction and tumor classification with heterogeneous hidden conditional random fields.
             <i>Bioinformatics</i>, 2009.
             (<a href="http://dx.doi.org/10.1093/bioinformatics/btn585" target="_blank" rel="noopener">pdf</a>) <br><br-->
         
	    </li><li> Systems-level dynamic analyses of fate change in murine embryonic stem cells.
             <i>Nature</i>, 2009.
             <!-- (<a href="http://www.nature.com/nature/journal/v462/n7271/full/nature08575.html" target="_blank" rel="noopener">link</a>) -->
             (<a href="https://sites.harvard.edu/airoldi/j011.Lu_Lemischka2009Nature.pdf" target="_blank" rel="noopener">pdf</a>)
             (<a href="https://sites.harvard.edu/airoldi/j011s.Lu_Lemischka2009Nature.pdf" target="_blank" rel="noopener">supp</a>)
             (<a href="http://f1000.com/prime/1387966" target="_blank" rel="noopener"><font color="red">F1000</font></a>)             
             (<a href="http://www.nature.com/nbt/journal/v28/n2/full/nbt0210-146.html" target="_blank" rel="noopener"><font color="red">news &amp; views, Nat BT</font></a>)
             (<a href="http://stke.sciencemag.org/cgi/content/abstract/sigtrans;2/98/ec377" target="_blank" rel="noopener"><font color="red">editor&#8217;s choice, Sci Sig</font></a>) <br><br>
          
	    </li></font><li><font size="+0"> Coordination of growth rate, cell cycle, stress response and metabolic activity in yeast.     
             <i>Molecular Biology of the Cell</i>, 2008.
	         <!-- (<a href="http://www.molbiolcell.org/cgi/reprint/19/1/352" target="_blank" rel="noopener">link</a>) -->          
	         (<a href="https://sites.harvard.edu/airoldi/j007.Brauer_Botstein2008MBC.pdf" target="_blank" rel="noopener">pdf</a>)           
	         (<a href="http://growthrate.princeton.edu/transcriptome/" target="_blank" rel="noopener">code &amp; data</a>) <br>                              
          
      </font></li></ul><font size="+0">

      &nbsp; <u><b>applied methodology in computational social science</b></u></font>
      <ul>	

		<li> A regularization scheme on word occurrence rates that improves estimation and interpretation of topical content (with discussion). 
             <i>Journal of the American Statistical Association</i>, 2016. 
             (<a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2015.1051182" target="_blank" rel="noopener">pdf</a>) 
             (<font color="red">Oustanding Statistical Application Award, ASA 2016</font>)          
             <br><br>

		</li><li> A model of text for experimentation in the social sciences. 
             <i>Journal of the American Statistical Association</i>, 2016. 
             (<a href="http://www.tandfonline.com/doi/abs/10.1080/01621459.2016.1141684" target="_blank" rel="noopener">pdf</a>) 
             <br><br>          

	    </li><li> Predicting traffic volumes and estimating the effects of shocks in massive transportation systems.         
             <i>Proceedings of the National Acadamy of Sciences</i>, 2015. <br>
	         <br>

	    </li><li> A natural experiment of social network formation and dynamics.         
             <i>Proceedings of the National Acadamy of Sciences</i>, 2015. <br>
	         <br>

	    <!--li> Discussion of Hennig and Liao 'How to find an appropriate clustering for mixed-type variables with application to socio-economic stratification'.         
             <i>Journal of the Royal Statistical Society, Series C</i>, 2013.
	         (<a href="https://sites.harvard.edu/airoldi/j026.FellerAiroldi2013JRSSC.pdf" target="_blank" rel="noopener">pdf</a>)
	         (<a href="https://sites.harvard.edu/airoldi/ref/HennigLiao2013JRSSCwDisc.pdf" target="_blank" rel="noopener">article</a>) <br>
	         <br-->

	    </li><li> Reconceptualizing the classification of PNAS articles.
             <i>Proceedings of the National Academy of Sciences</i>, 2010.
             <!-- (<a href="http://www.pnas.org/content/107/49/20899.abstract" target="_blank" rel="noopener">link</a>) --> 
             (<a href="https://sites.harvard.edu/airoldi/j014.Airoldi_Shringarpure2010PNAS.pdf" target="_blank" rel="noopener">pdf</a>) 
             (<a href="http://www.pnas.org/content/107/49/20847.full" target="_blank" rel="noopener"><font color="red">editorial feature</font></a>) <br><br>

	    <!--li> Discussion of Handcock, Raftery and Tantrum 'Model-based clustering for social networks'.         
             <i>Journal of the Royal Statistical Society, Series A</i>, 2007.
	         (<a href="https://sites.harvard.edu/airoldi/j004.Airoldi2007JRSSA.pdf" target="_blank" rel="noopener">pdf</a>)
	         (<a href="https://sites.harvard.edu/airoldi/ref/HandcockRafteryTantrum2007JRSSAwDisc.pdf" target="_blank" rel="noopener">article</a>) <br>
	         <br-->

	    </li><li> Whose ideas? Whose words? Authorship of the Ronald Reagan radio addresses.          
             <i>Political Science &amp; Politics</i>, 2007.
	         <!-- (<a href="https://sites.harvard.edu/airoldi/AiroldiFienbergSkinner2007PS.pdf" target="_blank" rel="noopener">link</a>) --> 
	         (<a href="https://sites.harvard.edu/airoldi/j005.AiroldiFienbergSkinner2007PS.pdf" target="_blank" rel="noopener">pdf</a>) 
             (<a href="http://www.nytimes.com/2007/09/15/opinion/15skinner.html" target="_blank" rel="noopener"><font color="red">op-ed by Skinner &amp; Rice</font></a>)<br>
             <br>

	    </li><li> Who wrote Ronald Reagan&#8217;s radio addresses?
             <i>Bayesian Analysis</i>, 2006.
	         <!-- (<a href="http://ba.stat.cmu.edu/journal/2006/vol01/issue02/airoldi289-320.pdf" target="_blank" rel="noopener">link</a>) -->
	         (<a href="https://sites.harvard.edu/airoldi/j003.AiroldiAndersonFienbergSkinner2006BA.pdf" target="_blank" rel="noopener">pdf</a>)
	         (<a href="http://www.stat.cmu.edu/tr/tr789/tr789.pdf" target="_blank" rel="noopener">tr with detailed predictions</a>)
	         (<a href="https://sites.harvard.edu/airoldi/Airoldi2005NegBinom.pdf" target="_blank" rel="noopener">notes on Negative Binomial</a>) <br><!--br-->

	    <!--li> Bayesian methods for frequent terms in text.         
             <i>CSNA & INTERFACE Annual Meetings</i>, 2005.
	         (<a href="papers/airo.cohe.fien.2005.pdf" target="_blank" rel="noopener">pdf</a>) <br><br-->

        <!--li> A social network analysis model for name disambiguation in lists.         
             <i>Journal of Computational and Mathematical Organization Theory</i>, 2005. 
             (<a href="http://dx.doi.org/10.1007/s10588-005-3940-3" target="_blank" rel="noopener">pdf</a>) <br-->
      </li></ul>

      &nbsp; <u><b>theses</b></u>
      <ul>	

	    <li> Bayesian mixed-membership models of complex and evolving networks.
	         <em>Doctoral dissertation</em>, 2007.
	         <!-- Committee: <a href="http://www.stat.cmu.edu/~fienberg/" target="_blank" rel="noopener">Fienberg, S.E.</a> (co-chair),
	                <a href="http://www.casos.cs.cmu.edu/bios/carley/carley.html" target="_blank" rel="noopener">Carley, K.M.</a> (co-chair),
	                <a href="http://www.cs.princeton.edu/~blei/" target="_blank" rel="noopener">Blei, D.M.</a> (external),
	                <a href="http://www.cs.cmu.edu/~christos/" target="_blank" rel="noopener">Faloutsos, C.</a>, &
	                <a href="http://www.cs.cmu.edu/~epxing/" target="_blank" rel="noopener">Xing, E.P.</a>. -->
	         (<font color="red">Savage award honorable mention, 2007</font>)<br><br>
      
        </li><li> The theory of weak convergence of probability measures and its applications in statistics.
	         <em>Undergraduate thesis</em>, 1999.
	         (<font color="red">Gold medal for best graduates, 1999</font>)
             <!-- Committee: Cifarelli, D.M., Regazzini, E., Veronese, P., &amp; Corielli, F. --> <br><br>          
      </li></ul>

		
			</div><!-- .entry-content -->

	</article><!-- #post-## -->

												</main><!-- #main -->

			<!-- Do the right sidebar check -->
			
</div><!-- #closing the primary container from /templates/global-templates/left-sidebar-check.php -->

		</div><!-- .row -->
	</div><!-- #content -->
</div><!-- #page-wrapper -->




<footer class="wrapper has-background has-footer-main-bg-color-background-color" id="wrapper-footer">
	<div id="footer-content">
		

					
		<script type="text/javascript" id="blog_templates_front-js-extra">
/* <![CDATA[ */
var blog_templates_params = {"type":""};
/* ]]> */
</script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-content/plugins/blogtemplates//blogtemplatesfiles/assets/js/front.js?ver=2.6.8.2" id="blog_templates_front-js"></script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/js/coblocks-animation.js?ver=3.1.15" id="coblocks-animation-js"></script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/js/vendors/tiny-swiper.js?ver=3.1.15" id="coblocks-tiny-swiper-js"></script>
<script type="text/javascript" id="coblocks-tinyswiper-initializer-js-extra">
/* <![CDATA[ */
var coblocksTinyswiper = {"carouselPrevButtonAriaLabel":"Previous","carouselNextButtonAriaLabel":"Next","sliderImageAriaLabel":"Image"};
/* ]]> */
</script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-content/plugins/coblocks/dist/js/coblocks-tinyswiper-initializer.js?ver=3.1.15" id="coblocks-tinyswiper-initializer-js"></script>
<script type="text/javascript" id="cpschool-scripts-js-extra">
/* <![CDATA[ */
var cpSchoolData = {"parallaxHeader":"","animations":""};
/* ]]> */
</script>
<script type="text/javascript" src="https://sites.harvard.edu/airoldi/wp-content/themes/cpschool/js/theme.min.js?ver=0.9.0.5.1733816539" id="cpschool-scripts-js"></script>
							<div id="custom-footer-html"><p><img class="alignnone size-full wp-image-178" src="https://sites.harvard.edu/group-site/files/2021/08/harvard-logo.png" alt="harvard university logo" width="186" height="47" /></p>
<p>Harvard University<br />Massachusetts Hall<br />Cambridge, MA 02138<br /><a href="mailto:your_name@harvard.edu">your_name@harvard.edu</a></p></div>
		
		<div id="footer-site-info">
			<div class="row">
				<div class="col-md-12">
					<div class="site-footer" id="colophon">
						<div class="site-info">
							<div id="branding_footer">
	<div class="branding-container container">
		<div class="copyright">
			<span class="harvard-copyright">Copyright &copy; 2025 The President and Fellows of Harvard College</span> |
			<a href="https://accessibility.harvard.edu/">Accessibility</a> |
			<a href="https://accessibility.huit.harvard.edu/digital-accessibility-policy">Digital Accessibility</a>  |
			<a href="https://www.harvard.edu/copyright-issue/">Report Copyright Infringement</a>
		</div>
		<div class="admin-login">
			<a href="https://sites.harvard.edu/airoldi/wp-login.php?redirect_to=https%3A%2F%2Fsites.harvard.edu%2Fairoldi%2F&#038;action=shibboleth" alt="Login">
				Log In			</a>
		</div>
	</div>
</div>
						</div><!-- .site-info -->
					</div><!-- #colophon -->
				</div><!--col end -->
			</div><!-- row end -->
		</div><!-- container end -->
	</div>

	</footer><!-- wrapper end -->

</div><!-- #page we need this extra closing tag here -->

<div id="modal-slide-in-menu" class="modal fade modal-slide-in-right modal-full-height nav-styling-underline" tabindex="-1" role="dialog" aria-label="slide-in menu" aria-hidden="true">
	<div class="modal-dialog" role="document">
		<div class="modal-content has-background has-header-main-bg-color-background-color">
			<div class="modal-header">
				<button type="button" class="close" data-dismiss="modal">
					<i aria-hidden="true" class="cps-icon cps-icon-close"></i>
					<span class="sr-only">close menu</span>
				</button>
			</div>
			<div class="modal-body">
									<nav class="menu-mobile" aria-label="Mobile" role="navigation">
						<div class="menu-primary-menu-container"><ul id="menu-main-mobile" class="nav flex-column"><li itemscope="itemscope" itemtype="https://www.schema.org/SiteNavigationElement" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-320 nav-item"><a class="nav-link" title="Contact" href="https://sites.harvard.edu/airoldi/contact/"><span class="menu-item-text">Contact</span></a></li>
</ul></div>
					</nav>
								</div>
		</div>
	</div>
</div><!-- #modal-slide-in-menu -->
<div id="modal-search" class="modal fade modal-slide-in-top modal-close-inline modal-site-width modal-padding-lg" tabindex="-1" role="dialog" aria-label="search" aria-hidden="true">
	<div class="modal-dialog site-width-max" role="document">
		<div class="modal-content has-background has-header-main-bg-color-background-color">
			<div class="modal-header pb-0">
				<button type="button" class="close" data-dismiss="modal">
					<i aria-hidden="true" class="cps-icon cps-icon-close"></i>
					<span class="sr-only">close search</span>
				</button>
			</div>
			<div class="modal-body">
				
<form method="get" id="searchform" action="https://sites.harvard.edu/airoldi/" role="search">
	<label class="sr-only" for="searchform-s">Search</label>
	<div class="input-group">
		<input class="field form-control form-control-lg" id="searchform-s" name="s" type="text" placeholder="Search..." value="">
		<span class="input-group-append">
			<button class="submit btn btn-primary btn-lg" id="searchform-submit" type="submit">
				<i aria-hidden="true" class="cps-icon cps-icon-search"></i>
				<span class="sr-only">Search Site</span>
			</button>
		</span>
	</div>
</form>
			</div>
		</div>
	</div>
</div><!-- #modal-search -->

</body>

</html>