

    <!DOCTYPE html>
<html class="">

        

<head>
    <title>Peer Review Examples - F1000Research</title>
    <meta charset="utf-8">
    <!--<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>-->
    <!--<meta lang="$locale">-->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="google-site-verification" content="9-QVycOO2_ob3Z9QzRmXv2CF08A9oyYXqWyTiVdKPlU" />
    <!-- This is commented out to fix display problems on mobile devices.
    We may use it again once we implement a responsive design that supports native device resolutions.
    <meta name="viewport" content="width=device-width"> -->

    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">



    <link rel="alternate" title="Recent articles published in F1000Research" href="/rss" type="application/rss+xml">
    
            <meta name="description" content="F1000Research is an innovative open access publishing platform offering immediate
publication, open peer review and full data deposition and sharing." />
    
    

    
    <link href="/img/favicon-research.ico" rel="shortcut icon" type="image/ico">
    <link href="/img/favicon-research.ico" rel="icon" type="image/ico">

        <link rel="stylesheet" href="/1746712139455/css/mdl/material-design-lite-research.css" type="text/css" media="all" />
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" rel="stylesheet">

        
            
    <link rel="stylesheet" href="/1746712139455/css/Research.css" type="text/css" media="all" />
    <link rel="stylesheet" href="/css/F1000ResearchFontIcons/F1000ResearchFontIcons.css" type="text/css" media="all" />
    <link rel="stylesheet" href="/css/F1000ResearchFontIcons/animation.css" type="text/css" media="all" />

        <!--[if IE 7]><link rel="stylesheet" href="/css/F1000ResearchFontIcons/F1000ResearchFontIcons-ie7.css" media="all" /><![endif]-->

    <!-- Transcend cookies consent for F1000Research -->
<script type="text/javascript">"use strict";function _typeof(t){return(_typeof="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(t){return typeof t}:function(t){return t&&"function"==typeof Symbol&&t.constructor===Symbol&&t!==Symbol.prototype?"symbol":typeof t})(t)}!function(){var t=function(){var t,e,o=[],n=window,r=n;for(;r;){try{if(r.frames.__tcfapiLocator){t=r;break}}catch(t){}if(r===n.top)break;r=r.parent}t||(!function t(){var e=n.document,o=!!n.frames.__tcfapiLocator;if(!o)if(e.body){var r=e.createElement("iframe");r.style.cssText="display:none",r.name="__tcfapiLocator",e.body.appendChild(r)}else setTimeout(t,5);return!o}(),n.__tcfapi=function(){for(var t=arguments.length,n=new Array(t),r=0;r<t;r++)n[r]=arguments[r];if(!n.length)return o;"setGdprApplies"===n[0]?n.length>3&&2===parseInt(n[1],10)&&"boolean"==typeof n[3]&&(e=n[3],"function"==typeof n[2]&&n[2]("set",!0)):"ping"===n[0]?"function"==typeof n[2]&&n[2]({gdprApplies:e,cmpLoaded:!1,cmpStatus:"stub"}):o.push(n)},n.addEventListener("message",(function(t){var e="string"==typeof t.data,o={};if(e)try{o=JSON.parse(t.data)}catch(t){}else o=t.data;var n="object"===_typeof(o)&&null!==o?o.__tcfapiCall:null;n&&window.__tcfapi(n.command,n.version,(function(o,r){var a={__tcfapiReturn:{returnValue:o,success:r,callId:n.callId}};t&&t.source&&t.source.postMessage&&t.source.postMessage(e?JSON.stringify(a):a,"*")}),n.parameter)}),!1))};"undefined"!=typeof module?module.exports=t:t()}();</script>
<script data-cfasync="false" data-tcf-ui-dismissed-view-state="Collapsed" src="https://transcend-cdn.com/cm/ef49a3f1-d8c1-47d6-88fc-50e41130631f/airgap.js"></script>

            <script>
    dataLayer = [];
    </script>

<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-PCBS9JK');</script>
<!-- End Google Tag Manager -->
        <script type="text/javascript">
;window.NREUM||(NREUM={});NREUM.init={distributed_tracing:{enabled:true},privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]}};

;NREUM.loader_config={accountID:"438030",trustKey:"438030",agentID:"772317073",licenseKey:"97f8f67f26",applicationID:"772317073"}
;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"97f8f67f26",applicationID:"772317073",sa:1}
;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();
</script>
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="/js/vendor/jquery-1.8.1.min.js"><\/script>')</script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.1/moment.min.js"></script>
    <script src="/1746712139455/js/vendor/modernizr-2.6.1-respond-1.1.0.min.js"></script>
    <script src="/1746712139455/js/shared_scripts/sticky.js"></script>
    <script src="/1746712139455/js/shared_scripts/helpers.js"></script>
    <script src="/1746712139455/js/shared_scripts/menu.js"></script>
    <script src="/1746712139455/js/shared_scripts/navbar.js"></script>
    <script src="/1746712139455/js/shared_scripts/platforms.js"></script>
    <script src="/1746712139455/js/shared_scripts/object-polyfills.js"></script>
            <script src="/1746712139455/js/vendor/underscore-umd.js"></script>
        <script>CKEDITOR_BASEPATH='https://f1000research.com/js/vendor/ckeditor/'</script>
    <script src="/1746712139455/js/plugins.js"></script>
    <script src="/1746712139455/js/shared_scripts/helpers.js"></script>
    <script src="/1746712139455/js/app/research.js"></script>
    <script>window.reactTheme = 'research';</script>
    <script src="/js/public/bundle.js"></script>

    <script src="/1746712139455/js/app/research.ui.js"></script>
    <script src="/1746712139455/js/app/login.js"></script>
    <script src="/1746712139455/js/app/main.js"></script>
    <script src="/1746712139455/js/app/js-date-format.min.js"></script>
    <script src="/1746712139455/js/app/search.js"></script>
    <script src="/1746712139455/js/mdl/mdl.min.js"></script>

    <script>
    window.MathJax = {
        CommonHTML: { linebreaks: { automatic: true } },
        'HTML-CSS': { linebreaks: { automatic: true } },
               SVG: { linebreaks: { automatic: true } },
        AuthorInit: function() {
            MathJax.Hub.Register.MessageHook('End Process',
            function () {
                let timeout = false; // holder for timeout id
                const delay = 250; // delay after event is "complete" to run callback
                const reflowMath = function() {
                    const dispFormulas = document.querySelectorAll('.disp-formula.panel');

                    if (!dispFormulas) {
                        return;
                    }

                    for (const dispFormula of dispFormulas) {
                        const child = dispFormula.querySelector('.MathJax_Preview').nextSibling.firstChild;
                        const isMultiline = MathJax.Hub.getAllJax(dispFormula)[0].root.isMultiline;
                        if (dispFormula.offsetWidth < child.offsetWidth || isMultiline) {
                            MathJax.Hub.Queue(['Rerender', MathJax.Hub, dispFormula]);
                        }
                    }
                };

                window.addEventListener('resize', function() {
                    clearTimeout(timeout); // clear the timeout
                    timeout = setTimeout(reflowMath, delay); // start timing for event "completion"
                });
            });
        },
    };
</script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
    
    <script src="/js/namespace.js"></script>
    <script src="/js/constants.js"></script>
    <script src="/js/utilities.js"></script>

    
            
                                                                            <script>
            if (window.location.hash == '#_=_'){
                window.location = window.location.href.split('#')[0]
            }
        </script>

                    
        
    <!-- pixelId: 1641728616063202 :: assetPixelId: 6034867600215 :: funderPixelId: $!fbFunderPixelId -->

            <!-- Facebook pixel code (merged with EP GTM code) -->
        <script >
            !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function()

            {n.callMethod? n.callMethod.apply(n,arguments):n.queue.push(arguments)}
            ;if(!f._fbq)f._fbq=n;
            n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
            t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
            document,'script','https://connect.facebook.net/en_US/fbevents.js');

            fbq('init', '1641728616063202');

            
            fbq('track', "PixelInitialized", {});
        </script>

        <noscript><img height="1" width="1" style="display:none"
            src="https://www.facebook.com/tr?id=1641728616063202&noscript=1&amp;ev=PixelInitialized"
        /></noscript>
        <!-- End Facebook Pixel Code -->
            <meta name="facebook-domain-verification" content="0ba5yc8np4c67n85z5ma2vtck11by4" />

                <script>
            (function(h,o,t,j,a,r){
                h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
                h._hjSettings={hjid:2318163,hjsv:6};
                a=o.getElementsByTagName('head')[0];
                r=o.createElement('script');r.async=1;
                r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
                a.appendChild(r);
            })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
        </script>
    
</head>
<body  class="o-page-container no-js p-for-referees o-layout-reset  ">

    
            <!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PCBS9JK"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->    
        
    <div class="o-page">

        <div id="notify-container"></div>
        <div id="pageWarning"></div>
        <div id="pageMessage"></div>
        <div id="pageFooterMessage"></div>

                
        
        <div class="header-wrapper   js-navbar-space ">
                            


    










    
  
    
              
      





<nav class="c-navbar js-navbar js-mini-nav js-sticky c-navbar--js-sticky c-navbar--userSite c-navbar__platform-bgcolor  c-navbar--bg-f1000research ">

    <div class="c-navbar__content">

                                <div class="c-navbar__extras">
            <div class="o-wrapper">
                <div class="o-actions o-actions--middle c-navbar__extras-row">
                    <div class="o-actions__primary">
                        
                                            </div>
                                    </div>
            </div>
        </div>

        <div class="o-wrapper t-inverted js-sticky-start">

            <div class="c-navbar__branding-row">
                <div class="c-navbar__row">


                                        
                    <div class="c-navbar__primary u-mr--2">

                                                                                                                                                                                                    <a href="/" class="c-navbar__branding u-ib u-middle"   data-test-id="nav_branding"  >
                                <img class="u-ib u-middle" src="/img/research/F1000Research_white_solid.svg" alt="F1000Research">
                            </a>
                                            </div>

                    <div class="c-navbar__secondary c-navbar__row">


                                                
                                                    <form action="/search" class="-navbar__secondary u-mr--2 c-search-form js-search-form u-hide u-show@navbar">
                                <label for="searchInput" class="c-search-form__label _mdl-layout">
                                    <input name="q" type="search" class="c-search-form__input" id="searchInput" placeholder="Search">
                                    <button type="submit" class="c-search-form__submit mdl-button mdl-js-button mdl-button--icon"><i class="material-icons">search</i></button>
                                </label>
                            </form>
                        
                                                
                                                    <div class="c-navbar__primary u-hide u-show@navbar">
                                <div class="_mdl-layout c-navbar__cta">
                                    <a class="mdl-button mdl-js-button mdl-button--raised mdl-button--no-shadow mdl-button--multi-line mdl-js-ripple-effect mdl-button--inverted c-navbar__submit" href="/for-authors/publish-your-research"   data-test-id="nav_submit_research"  ><i class="material-icons">file_upload</i>Submit your research</a>
                                </div>
                            </div>
                        

                                                
                        <span class="u-hide@navbar _mdl-layout u-nowrap">

                                                            <button type="button" class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-button--icon c-navbar__toggle c-navbar__toggle--menu js-navbar-toggle" data-focus="#navbar_mob_search_input" data-toggle="navbarMenu" data-target="navbarMenu" aria-controls="navbarMenu" aria-expanded="false" aria-label="Toggle navigation"   data-test-id="nav_menu_search_mob"  ><i class="material-icons">search</i></button>
                            
                                                            <button class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-button--icon mdl-button--multi-line c-navbar__toggle c-navbar__toggle--menu js-navbar-toggle" type="button" data-toggle="navbarMenu" data-target="navbarMenu" aria-controls="navbarMenu" aria-expanded="false" aria-label="Toggle navigation"   data-test-id="nav_menu_toggle_mob"  >
                                    <i class="material-icons c-navbar__toggle-open">menu</i>
                                    <i class="material-icons c-navbar__toggle-close">close</i>
                                </button>
                                                    </span>
                    </div>

                </div>
            </div>

                        
                            <div class="c-navbar__menu-row js-navbar-block is-collapsed" id="navbarMenu">

                                                                                                                                                
                                                                    
                                                                    
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                

                                                                
                                                                                                                                                                                                                                
                                            
                    <div class="c-navbar__menu-row-content">

                                                                            <div class="u-hide@navbar c-navbar__menu-bar-spacing">
                                <form action="/search" class="c-search-form js-search-form">
                                    <label for="navbar_mob_search_input" class="c-search-form__label _mdl-layout">
                                        <input id="navbar_mob_search_input" name="q" type="search" class="c-search-form__input" placeholder="Search">
                                        <button type="submit" class="c-search-form__submit mdl-button mdl-js-button mdl-button--icon"><i class="material-icons">search</i></button>
                                    </label>
                                </form>
                            </div>
                        
                        <div class="o-actions o-actions--middle">

                            <div class="o-actions__primary">

                                                                
    <ul class="c-menubar c-navbar__menu-bar js-main-menu"   id="main-menu"   role="menubar" aria-label="Main Navigation"  data-menu-group="navbar" >

        
            
                                                                
                                
                                
                                    <li role="none"
                        data-index="0"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/browse/articles" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="0"
                              data-test-id="nav_browse"                              >Browse</a>

                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="1"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/gateways" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="-1"
                              data-test-id="nav_gatewaysViewAndBrowse"                              >Gateways & Collections</a>

                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="2"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="-1"
                              data-test-id="nav_for-authors"                              >How to Publish</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="How to Publish">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_submit-manuscript"                      href="/for-authors/publish-your-research"
                    role="menuitem"
                    tabindex="0">Submit your Research</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-submissions"                      href="/for-authors/my-submissions"
                    role="menuitem"
                    tabindex="-1">My Submissions</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-guidelines"                      href="/for-authors/article-guidelines"
                    role="menuitem"
                    tabindex="-1">Article Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-guidelines-new-versions"                      href="/for-authors/article-guidelines-new-versions"
                    role="menuitem"
                    tabindex="-1">Article Guidelines (New Versions)</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_data-guidelines"                      href="/for-authors/data-guidelines"
                    role="menuitem"
                    tabindex="-1">Open Data, Software and Code Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_data-guidelines-hss"                      href="/for-authors/data-guidelines-hss"
                    role="menuitem"
                    tabindex="-1">Open Data and Accessible Source Materials Guidelines (HSS)</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_data-guidelines-pse"                      href="/for-authors/data-guidelines-pse"
                    role="menuitem"
                    tabindex="-1">Open Data, Software and Code Guidelines (PSE)</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_prepublication-checks"                      href="/for-authors/prepublication-checks"
                    role="menuitem"
                    tabindex="-1">Prepublication Checks</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-guidelines-the-production-process"                      href="/for-authors/article-guidelines-the-production-process"
                    role="menuitem"
                    tabindex="-1">Production Process</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_asset-guidelines"                      href="/for-authors/posters-and-slides-guidelines"
                    role="menuitem"
                    tabindex="-1">Posters and Slides Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_document-guidelines"                      href="/for-authors/document-guidelines"
                    role="menuitem"
                    tabindex="-1">Document Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-processing-charges"                      href="/for-authors/article-processing-charges"
                    role="menuitem"
                    tabindex="-1">Article Processing Charges</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_peer-review"                      href="/for-authors/peer-review"
                    role="menuitem"
                    tabindex="-1">Peer Review</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_finding-referees"                      href="/for-authors/tips-for-finding-referees"
                    role="menuitem"
                    tabindex="-1">Finding Article Reviewers</a>

                </li>

                        </ul>
                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="3"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="-1"
                              data-test-id="nav_about-contact"                              >About</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="About">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_about-page"                      href="/about"
                    role="menuitem"
                    tabindex="0">How it Works</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_referee-guidelines"                      href="/for-referees/guidelines"
                    role="menuitem"
                    tabindex="-1">For Reviewers</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_advisoryPanel"                      href="/advisors"
                    role="menuitem"
                    tabindex="-1">Our Advisors</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_policy-page"                      href="/about/policies"
                    role="menuitem"
                    tabindex="-1">Policies</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_glossary-page"                      href="/glossary"
                    role="menuitem"
                    tabindex="-1">Glossary</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_faqs-page"                      href="/faqs"
                    role="menuitem"
                    tabindex="-1">FAQs</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              u-hide u-show@navbar"
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      u-hide u-show@navbar"
                                          data-test-id="nav_for-developers"                      href="/developers"
                    role="menuitem"
                    tabindex="-1">For Developers</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_newsroom-page"                      href="/newsroom"
                    role="menuitem"
                    tabindex="-1">Newsroom</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_contact-page"                      href="/contact"
                    role="menuitem"
                    tabindex="-1">Contact</a>

                </li>

                        </ul>
                                            </li>
                
            
        
    </ul>



                            </div>

                            <div class="o-actions__secondary">

                                                                
    <ul class="c-menubar c-navbar__menu-bar js-main-menu"   id="secondary-items"   role="menubar" aria-label="My Account"  data-menu-group="navbar" >

        
            
                                                                
                                
                                
                                    <li role="none"
                        data-index="0"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="0"
                              data-test-id="nav_my-research"                              >My Research</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="My Research">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-submissions"                      href="/login?originalPath=%2Fmy%2Fsubmissions"
                    role="menuitem"
                    tabindex="0">Submissions</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-email-alerts"                      href="/login?originalPath=%2Fmy%2Femail-alerts"
                    role="menuitem"
                    tabindex="-1">Content and Tracking Alerts</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-user-details"                      href="/login?originalPath=%2Fmy%2Fuser-details"
                    role="menuitem"
                    tabindex="-1">My Details</a>

                </li>

                        </ul>
                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="1"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/login?originalPath=%2Ffor-referees%2Fpeer-reviewing-tips%2Fexamples" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="-1"
                              data-test-id="nav_sign-in"                              >Sign In</a>

                                            </li>
                
            
        
    </ul>



                            </div>

                                                                                        <div class="_mdl-layout c-navbar__cta u-hide@navbar c-navbar__menu-bar-spacing">
                                    <a class="mdl-button mdl-js-button mdl-button--raised mdl-button--multi-line mdl-button--no-shadow mdl-js-ripple-effect mdl-button--inverted c-navbar__submit" href="/for-authors/publish-your-research"   data-test-id="nav_submit_research_mob"  ><i class="material-icons">file_upload</i>Submit your research</a>
                                </div>
                                                    </div>

                    </div>

                </div>
            
        </div>

    </div>

</nav>
                    </div>

        <div class="content-wrapper o-page__main row homepage">
            <div id="highlight-area" class="content ">
                







<div id="peer-review-examples" class="row research-layout _mdl-layout research-info-pages no-margins">

  <div class="o-wrapper u-mt--2">
                
            
            
            
                                                                    
  <script type="application/ld+json">
  {
    "@context": "http://schema.org",
    "@type": "BreadcrumbList",
    "itemListElement": [
              {
          "@type": "ListItem",
          "position": "1",
          "item": {
            "@id": "https://f1000research.com/",
            "name": "Home"
          }
        },              {
          "@type": "ListItem",
          "position": "2",
          "item": {
            "@id": "https://f1000research.com/for-referees/guidelines",
            "name": "For Reviewers"
          }
        },              {
          "@type": "ListItem",
          "position": "3",
          "item": {
            "@id": "https://f1000research.com/for-referees/peer-reviewing-tips/examples",
            "name": "Examples"
          }
        }          ]
  }
  </script>
      <div class="breadcrumbs u-pt--1 js-breadcrumbs">
                  <a href="/" class="f1r-standard-link">Home</a> <span class="item_separator"></span>
                        <a href="/for-referees/guidelines" class="f1r-standard-link">For Reviewers</a> <span class="item_separator"></span>
                        Examples
            </div>
    
            <div class="heading2 f1r-page-heading  with-breadcrumb ">
            <span class="t-h2">For Reviewers</span>
            <span class="is-hidden float-right is-right margin-top heading5 red header-update-message"></span>
        </div>
      </div>
  
        
    <div class="p-for-referees__nav">
        <div class="u-hide@md">
            <nav class="c-browse-tabs">
                <ul class="c-browse-tabs__list">

                    
                                                
                        <li class="js-browse-tabs__item c-browse-tabs__item ">
                            <a href="/for-referees/guidelines" class="c-browse-tabs__link"   >Reviewer Guidelines</a>
                        </li>

                        
                    
                                                
                        <li class="js-browse-tabs__item c-browse-tabs__item c-browse-tabs__item--selected is-selected">
                            <a href="/for-referees/peer-reviewing-tips" class="c-browse-tabs__link"   >Peer Reviewing Tips</a>
                        </li>

                        
                    
                                                
                        <li class="js-browse-tabs__item c-browse-tabs__item ">
                            <a href="/for-referees/incentives" class="c-browse-tabs__link"   >Benefits for Reviewers</a>
                        </li>

                        
                                    </ul>
            </nav>
        </div>
    </div>

    

  <div class="p-for-referees__content">
    <div class="o-wrapper">

      
    
        
    <div class="u-hide u-show@md">
        <div class="section-nav heading11 js-sticky-nav">
            <ul class="section-nav-list">

                
                                        
                    <a href="/for-referees/guidelines" class="sidebar-navigation-link ">
                        <li class="">Reviewer Guidelines</li>
                    </a>

                                            <ul class="subsection-nav-list ">
                                                            <li href="#prm" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="prm" >Peer review model</li>
                                                            <li href="#sae" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="sae" >Scope &amp; article eligibility</li>
                                                            <li href="#rel" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="rel" >Reviewer eligibility</li>
                                                            <li href="#con" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="con" >Peer reviewer code of conduct</li>
                                                            <li href="#gfr" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="gfr" >Guidelines for reviewing</li>
                                                            <li href="#hts" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="hts" >How to submit</li>
                                                            <li href="#prp" class="subsection-item js-ref-scroll-to" data-gatewaypath="" data-sectionid="prp" >The peer-review process</li>
                                                    </ul>
                                    
                                        
                    <a href="/for-referees/peer-reviewing-tips" class="sidebar-navigation-link is-selected">
                        <li class="is-selected">Peer Reviewing Tips</li>
                    </a>

                                    
                                        
                    <a href="/for-referees/incentives" class="sidebar-navigation-link ">
                        <li class="">Benefits for Reviewers</li>
                    </a>

                                                </ul>
        </div>
    </div>

      <div class="page-content">
                <div class="t-h3 u-weight--md p-for-referees__title">Peer Review Examples</div>

        <div class="row example-detail">
          <a id="good-explanation"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Good explanation of relevance of article</div>
            <div class="example-authors">Annalisa Pastore says:</div>

            <div class="collapsed-section">
              <p>The genesis of this paper is the proposal that genomes containing a poor percentage of guanosine and cytosine (GC) nucleotide pairs lead to proteomes more prone to aggregation than those encoded by GC-rich genomes. As a consequence these organisms are also more dependent on the protein folding machinery. If true, this interesting hypothesis could establish a direct link between the tendency to aggregate and the genomic code.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The genesis of this paper is the proposal that genomes containing a poor percentage of guanosine and cytosine (GC) nucleotide pairs lead to proteomes more prone to aggregation than those encoded by GC-rich genomes. As a consequence these organisms are also more dependent on the protein folding machinery. If true, this interesting hypothesis could establish a direct link between the tendency to aggregate and the genomic code.</p>

              <p>In their paper, the authors have tested the hypothesis on the genomes of eubacteria using a genome-wide approach based on multiple machine learning models. Eubacteria are an interesting set of organisms which have an appreciably high variation in their nucleotide composition with the percentage of CG genetic material ranging from 20% to 70%. The authors classified different eubacterial proteomes in terms of their aggregation propensity and chaperone-dependence. For this purpose, new classifiers had to be developed which were based on carefully curated data. They took account for twenty-four different features among which are sequence patterns, the pseudo amino acid composition of phenylalanine, aspartic and glutamic acid, the distribution of positively charged amino acids, the FoldIndex score and the hydrophobicity. These classifiers seem to be altogether more accurate and robust than previous such parameters.</p>

              <p>The authors found that, contrary to what expected from the working hypothesis, which would predict a decrease in protein aggregation with an increase in GC richness, the aggregation propensity of proteomes increases with the GC content and thus the stability of the proteome against aggregation increases with the decrease in GC content. The work also established a direct correlation between GC-poor proteomes and a lower dependence on GroEL. The authors conclude by proposing that a decrease in eubacterial GC content may have been selected in organisms facing proteostasis problems. A way to test the overall results would be through <em>in vitro</em> evolution experiments aimed at testing whether adaptation to low GC content provide folding advantage.</p>

              <p>The main strengths of this paper is that it addresses an interesting and timely question, finds a novel solution based on a carefully selected set of rules, and provides a clear answer. As such this article represents an excellent and elegant bioinformatics genome-wide study which will almost certainly influence our thinking about protein aggregation and evolution. Some of the weaknesses are the not always easy readability of the text which establishes unclear logical links between concepts.</p>

              <p>Another possible criticism could be that, as any <em>in silico</em> study, it makes strong assumptions on the sequence features that lead to aggregation and strongly relies on the quality of the classifiers used. Even though the developed classifiers seem to be more robust than previous such parameters, they remain only overall indications which can only allow statistical considerations. It could of course be argued that this is good enough to reach meaningful conclusions in this specific case.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-137" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="detailed-analysis"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Detailed analysis puts article in context</div>
            <div class="example-authors">Eva Delpon and Ricardo Caballero say:</div>
            <div class="collapsed-section">
              <p>The paper by Chevalier <em>et al.</em> analyzed whether late sodium current (I<sub>NaL</sub>) can be assessed using an automated patch-clamp device. To this end, the I<sub>NaL</sub> effects of ranolazine (a well known I<sub>NaL</sub> inhibitor) and veratridine (an I<sub>NaL</sub> activator) were described. The authors tested the CytoPatch automated patch-clamp equipment and performed whole-cell recordings in HEK293 cells stably transfected with human Nav1.5. Furthermore, they also tested the electrophysiological properties of human induced pluripotent stem cell-derived cardiomyocytes (hiPS) provided by Cellular Dynamics International. The title and abstract are appropriate for the content of the text. Furthermore, the article is well constructed, the experiments were well conducted, and analysis was well performed.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The paper by Chevalier <em>et al.</em> analyzed whether late sodium current (I<sub>NaL</sub>) can be assessed using an automated patch-clamp device. To this end, the I<sub>NaL</sub> effects of ranolazine (a well known I<sub>NaL</sub> inhibitor) and veratridine (an I<sub>NaL</sub> activator) were described. The authors tested the CytoPatch automated patch-clamp equipment and performed whole-cell recordings in HEK293 cells stably transfected with human Nav1.5. Furthermore, they also tested the electrophysiological properties of human induced pluripotent stem cell-derived cardiomyocytes (hiPS) provided by Cellular Dynamics International. The title and abstract are appropriate for the content of the text. Furthermore, the article is well constructed, the experiments were well conducted, and analysis was well performed.</p>

              <p>I<sub>NaL</sub> is a small current component generated by a fraction of Nav1.5 channels that instead to entering in the inactivated state, rapidly reopened in a burst mode. I<sub>NaL</sub> critically determines action potential duration (APD), in such a way that both acquired (myocardial ischemia and heart failure among others) or inherited (long QT type 3) diseases that augmented the I<sub>NaL</sub> magnitude also increase the susceptibility to cardiac arrhythmias. Therefore, I<sub>NaL</sub> has been recognized as an important target for the development of drugs with either antiischemic or antiarrhythmic effects. Unfortunately, accurate measurement of I<sub>NaL</sub> is a time consuming and technical challenge because of its extra-small density. The automated patch clamp device tested by Chevalier et al. resolves this problem and allows fast and reliable I<sub>NaL</sub> measurements.</p>

              <p>The results here presented merit some comments and arise some unresolved questions. First, in some experiments (such is the case in experiments B and D in Figure 2) current recordings obtained before the ranolazine perfusion seem to be quite unstable. Indeed, the amplitude progressively increased to a maximum value that was considered as the control value (highlighted with arrows). Can this problem be overcome? Is this a consequence of a slow intracellular dialysis? Is it a consequence of a time-dependent shift of the voltage dependence of activation/inactivation? Second, as shown in Figure 2, intensity of drug effects seems to be quite variable. In fact, experiments A, B, C, and D in Figure 2 and panel 2D, demonstrated that veratridine augmentation ranged from 0-400%. Even assuming the normal biological variability, we wonder as to whether this broad range of effect intensities can be justified by changes in the perfusion system. Has been the automated dispensing system tested? If not, we suggest testing the effects of several K<sup>+</sup> concentrations on inward rectifier currents generated by Kir2.1 channels (I<sub>Kir2.1</sub>).</p>

              <p>The authors demonstrated that the recording quality was so high that the automated device allows to the differentiation between noise and current, even when measuring currents of less than 5 pA of amplitude. In order to make more precise mechanistic assumptions, the authors performed an elegant estimation of current variance (σ<sup>2</sup>) and macroscopic current (I) following the procedure described more than 30 years ago by Van Driessche and Lindemann <a href="#chevref"><sup>1</sup></a>. By means of this method, Chevalier <em>et al.</em> reducing the open channel probability, while veratridine increases the number of channels in the burst mode. We respectfully would like to stress that these considerations must be put in context from a pharmacological point of view. We do not doubt that ranolazine acts as an open channel blocker, what it seems clear however, is that its onset block kinetics has to be &ldquo;ultra&rdquo; slow, otherwise ranolazine would decrease peak I<sub>NaL</sub> even at low frequencies of stimulation. This comment points towards the fact that for a precise mechanistic study of ionic current modifying drugs it is mandatory to analyze drug effects with much more complicated pulse protocols. Questions thus are: does this automated equipment allow to the analysis of the frequency-, time-, and voltage-dependent effects of drugs? Can versatile and complicated pulse protocols be applied? Does it allow to a good voltage control even when generated currents are big and fast? If this is not possible, and by means of its extraordinary discrimination between current and noise, this automated patch-clamp equipment will only be helpful for rapid I<sub>NaL</sub>-modifying drug screening. Obviously it will also be perfect to test HERG blocking drug effects as demanded by the regulatory authorities.</p>

              <p>Finally, as cardiac electrophysiologists, we would like to stress that it seems that our dream of testing drug effects on human ventricular myocytes seems to come true. Indeed, human atrial myocytes are technically, ethically and logistically difficult to get, but human ventricular are almost impossible to be obtained unless from the explanted hearts from patients at the end stage of cardiac diseases. Here the authors demonstrated that ventricular myocytes derived from hiPS generate beautiful action potentials that can be recorded with this automated equipment. The traces shown suggested that there was not alternation in the action potential duration. Is this a consistent finding? How long do last these stable recordings? The only comment is that resting membrane potential seems to be somewhat  variable. Can this be resolved? Is it an unexpected veratridine effect? Standardization of maturation methods of ventricular myocytes derived from hiPS will be a big achievement for cardiac cellular electrophysiology which was obliged for years to the imprecise extrapolation of data obtained from a combination of several species none of which was representative of human electrophysiology. The big deal will be the maturation of human atrial myocytes derived from hiPS that fulfil the known characteristics of human atrial cells.</p>

              <div class="detail-report-subheading">Minor points:</div>
              <p>We suggest suppressing the initial sentence of section 3. We surmise that results obtained from the experiments described in this section cannot serve to understand the role of I<sub>NaL</sub> in arrhythmogenesis.</p>

              <div class="detail-report-subheading" id="chevref" name="chevref">Reference</div>
              <p>1. Van Driessche W, Lindemann B: Concentration dependence of currents through single sodium-selective pores in frog skin. <em>Nature</em>. 1979; 282 (5738): 519-520 <a href="http://www.ncbi.nlm.nih.gov/pubmed/315521" target="_blank">PubMed Abstract</a> | <a href="https://doi.org/10.1038/282519a0" target="_blank">Publisher Full Text</a></p>

              <div class="link-row expanded-example">
                <a href="/articles/3-245" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>
            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="critical-rereview"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Critical re-review with well-organized points of concern</div>
            <div class="example-authors">Sam Schwarzkopf says:</div>
            <div class="collapsed-section">
              <p>The authors have clarified several of the questions I raised in my previous review. Unfortunately, most of the major problems have not been addressed by this revision. As I stated in my previous review, I deem it unlikely that all those issues can be solved merely by a few added paragraphs. Instead there are still some fundamental concerns with the experimental design and, most critically, with the analysis. This means the strong conclusions put forward by this manuscript are not warranted and I cannot approve the manuscript in this form.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The authors have clarified several of the questions I raised in my previous review. Unfortunately, most of the major problems have not been addressed by this revision. As I stated in my previous review, I deem it unlikely that all those issues can be solved merely by a few added paragraphs. Instead there are still some fundamental concerns with the experimental design and, most critically, with the analysis. This means the strong conclusions put forward by this manuscript are not warranted and I cannot approve the manuscript in this form.</p>

              <ul class="number-list" style="margin-top: 10px;">
                <li>The greatest concern is that when I followed the description of the methods in the previous version it was possible to decode, with almost perfect accuracy, any arbitrary stimulus labels I chose. See <a href="https://doi.org/10.6084/m9.figshare.1167456" target="_blank">https://doi.org/10.6084/m9.figshare.1167456</a> for examples of this reanalysis. Regardless of whether we pretend that the actual stimulus appeared at a later time or was continuously alternating between signal and silence, the decoding is always close to perfect. This is an indication that the decoding has <strong>nothing to do with the actual stimulus</strong> heard by the Sender but is opportunistically exploiting some other features in the data. The control analysis the authors performed, reversing the stimulus labels, cannot address this problem because it suffers from the exact same problem. Essentially, what the classifier is presumably using is the time that has passed since the recording started.</li>

                <li>The reason for this is presumably that the authors used non-independent data for training and testing. Assuming I understand correctly (see point 3), random sampling one half of data samples from an EEG trace are <strong>not independent data</strong>. Repeating the analysis five times – the control analysis the authors performed – is not an adequate way to address this concern. Randomly selecting samples from a time series containing slow changes (such as the slow wave activity that presumably dominates these recordings under these circumstances) will inevitably contain strong temporal correlations. See TemporalCorrelations.jpg in <a href="https://doi.org/10.6084/m9.figshare.1185723" target="_blank">https://doi.org/10.6084/m9.figshare.1185723</a> for 2D density histograms and a correlation matrix demonstrating this.</li>

                <li>While the revised methods section provides more detail now, it still is unclear about exactly what data were used. Conventional classification analysis report what data features (usual columns in the data matrix) and what observations (usual rows) were used. Anything could be a feature but typically this might be the different EEG channels or fMRI voxels etc. Observations are usually time points. Here I assume the authors transformed the raw samples into a different space using principal component analysis. It is not stated if the dimensionality was reduced using the eigenvalues. Either way, I assume the data samples (collected at 128 Hz) were then used as observations and the EEG channels transformed by PCA were used as features. The stimulus labels were assigned as ON or OFF for each sample. A set of 50% of samples (and labels) was then selected at random for training, and the rest was used for testing. Is this correct?</li>

                <li>A powerful non-linear classifier can capitalise on such correlations to discriminate arbitrary labels. In my own analyses I used both an SVM with RBF as well as a k-nearest neighbour classifier, both of which produce excellent decoding of arbitrary stimulus labels (see point 1). Interestingly, linear classifiers or less powerful SVM kernels fare much worse – a clear indication that the classifier learns about the complex non-linear pattern of temporal correlations that can describe the stimulus label. This is further corroborated by the fact that when using stimulus labels that are chosen completely at random (i.e. with high temporal frequency) decoding does not work.</li>

                <li>The authors have mostly clarified how the correlation analysis was performed. It is still left unclear, however, how the correlations for individual pairs were averaged. Was Fisher&rsquo;s z-transformation used, or were the data pooled across pairs? More importantly, it is not entirely surprising that under the experimental conditions there will be some correlation between the EEG signals for different participants, especially in low frequency bands. Again, this further supports the suspicion that the classification utilizes slow frequency signals that are unrelated to the stimulus and the experimental hypothesis. In fact, a quick spot check seems to confirm this suspicion: correlating the time series separately for each channel from the Receiver in pair 1 with those from the Receiver in pair 18 reveals 131 significant (p&lsaquo;0.05, Bonferroni corrected) out of 196 (14x14 channels) correlations… One could perhaps argue that this is not surprising because both these pairs had been exposed to identical stimulus protocols: one minute of initial silence and only one signal period (see point 6). However, it certainly argues strongly against the notion that the decoding is any way related to the mental connection between the particular Sender and Receiver in a given pair because it clearly works between Receivers in different pairs! However, to further control for this possibility I repeated the same analysis but now comparing the Receiver from pair 1 to the Receiver from pair 15. This pair was exposed to a different stimulus paradigm (2 minutes of initial silence and a longer paradigm with three signal periods). I only used the initial 3 minutes for the correlation analysis. Therefore, both recordings would have been exposed to only one signal period but at different times (at 1 min and 2 min for pair 1 and 15, respectively). Even though the stimulus protocol was completely different the time courses for all the channels are highly correlated and 137 out of 196 correlations are significant. Considering that I used the <strong>raw data</strong> for this analysis it should not surprise anyone that extracting power from different frequency bands in short time windows will also reveal significant correlations. Crucially, it demonstrates that correlations between Sender and Receiver are artifactual and trivial.</li>

                <li>The authors argue in their response and the revision that predictive strategies were unlikely. After having performed these additional analyses I am inclined to agree. The excellent decoding almost certainly has nothing to do with expectation or imagery effects and it is irrelevant whether participants could guess the temporal design of the experiment. Rather, the results are almost entirely an artefact of the analysis. However, this does not mean that predictability is not an issue. The figure StimulusTimecourses.jpg in <a href=" https://doi.org/10.6084/m9.figshare.1185723" target="_blank">https://doi.org/10.6084/m9.figshare.1185723</a> plots the stimulus time courses for all 20 pairs as can be extracted from the newly uploaded data. This confirms what I wrote in my previous review, in fact, with the corrected data sets the problem with predictability is even greater. Out of the 20 pairs, 13 started with 1 min of initial silence. The remaining 7 had 2 minutes of initial silence. Most of the stimulus paradigms are therefore perfectly aligned and thus highly correlated. This also proves incorrect the statement that initial silence periods were 1, 2, or 3 minutes. No pair had 3 min of initial silence. It would therefore have been very easy for any given Receiver to correctly guess the protocol. It should be clear that this is far from optimal for testing such an unorthodox hypothesis. Any future experiments should employ more randomization to decrease predictability. Even if this wasn&rsquo;t the underlying cause of the present results, this is simply not great experimental design.</li>

                <li>The authors now acknowledge in their response that all the participants were authors. They say that this is also acknowledged in the methods section, but I did not see any statement about that in the revised manuscript. As before, I also find it highly questionable to include <strong>only</strong> authors in an experiment of this kind. It is not sufficient to claim that Receivers weren&rsquo;t guessing their stimulus protocol. While I am giving the authors (and thus the participants) the benefit of the doubt that they actually believe they weren&rsquo;t guessing/predicting the stimulus protocols, this does not rule out that they did. It may in fact be possible to make such predictions subconsciously (Now, if you ask me, <strong>this</strong> is an interesting scientific question someone should do an experiment on!). The fact familiar with the protocol may help that. Any future experiments should take steps to prevent this.</li>

                <li>I do not follow the explanation for the binomial test the authors used. Based on the excessive Bayes Factor of 390,625 it is clear that the authors assumed a chance level of 50% on their binomial test. Because the design is not balanced, this is not correct.</li>

                <li>In general, the Bayes Factor and the extremely high decoding accuracy should have given the authors reason to start. Considering the unusual hypothesis did the authors not at any point wonder if these results aren&rsquo;t just far too good to be true? Decoding mental states from brain activity is typically extremely noisy and hardly affords accuracies at the level seen here. Extremely accurate decoding and Bayes Factors in the hundreds of thousands should be a tell-tale sign to check that there isn&rsquo;t an analytical flaw that makes the result entirely trivial. I believe this is what happened here and thus I think this experiment serves as a very good demonstration for the pitfalls of applying such analysis without sanity checks. In order to make claims like this, the experimental design must contain control conditions that can rule out these problems. Presumably, recordings without any Sender, and maybe even when the &ldquo;Receiver&rdquo; is aware of this fact, should produce very similar results.</li>
              </ul>

              <p>Based on all these factors, it is impossible for me to approve this manuscript. I should however state that it is laudable that the authors chose to make all the raw data of their experiment publicly available. Without this it would have impossible for me to carry out the additional analyses, and thus the most fundamental problem in the analysis would have remained unknown. I respect the authors&rsquo; patience and professionalism in dealing with what I can only assume is a rather harsh review experience. I am honoured by the request for an adversarial collaboration. I do not rule out such efforts at some point in the future. However, for all of the reasons outlined in this and my previous review, I do not think the time is right for this experiment to proceed to this stage. Fundamental analytical flaws and weaknesses in the design should be ruled out first. An adversarial collaboration only really makes sense to me for paradigms were we can be confident that mundane or trivial factors have been excluded.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-182" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-feedback"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Positive feedback with specific examples</div>
            <div class="example-authors">Gregg Roman says:</div>

            <div class="collapsed-section">
              <p>This manuscript does an excellent job demonstrating significant strain differences in Burdian's paradigm. Since each Drosophila lab has their own wild type (usually Canton-S) isolate, this issue of strain differences is actually a very important one for between lab reproducibility. This work is a good reminder for all geneticists to pay attention to the population effects in the background controls, and presumably the mutant lines we are comparing.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>This manuscript does an excellent job demonstrating significant strain differences in Burdian's paradigm. Since each Drosophila lab has their own wild type (usually Canton-S) isolate, this issue of strain differences is actually a very important one for between lab reproducibility. This work is a good reminder for all geneticists to pay attention to the population effects in the background controls, and presumably the mutant lines we are comparing.</p>

              <p>I was very pleased to see the within-isolate behavior was consistent in replicate experiments one year apart. The authors further argue that the between-isolate differences in behavior arise from a Founder's effect, at least in the differences in locomotor behavior between the Paris lines CS_TP and CS_JC. I believe this is a very reasonable and testable hypothesis. It predicts that genetic variability for these traits exist within the populations. It should now be possible to perform selection experiments from the original CS_TP population to replicate the founding event and estimate the heritability of these traits.</p>

              <p>Two other things that I liked about this manuscript are the ability to adjust parameters in figure 3, and our ability to download the raw data. After reading the manuscript, I was a little disappointed that the performance of the five strains in each 12 behavioral variables weren't broken down individually in a table or figure. I thought this may help us readers understand what the principle components were representing. The authors have made this data readily accessible in a downloadable spreadsheet.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-176" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="concise-review"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Concise review with suggestions for a few minor revisions</div>
            <div class="example-authors">Gerald Watts says:</div>

            <div class="collapsed-section">
              <p>This is an exceptionally good review and balanced assessment of the status of CETP inhibitors and ASCVD from a world authority in the field. The article highlights important data that might have been overlooked when promulgating the clinical value of CETPIs and related trials.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>This is an exceptionally good review and balanced assessment of the status of CETP inhibitors and ASCVD from a world authority in the field. The article highlights important data that might have been overlooked when promulgating the clinical value of CETPIs and related trials.</p>

              <p>Only 2 areas need revision:</p>

              <ul class="number-list">
                <li>Page 3, para 2: the notion that these data from Papp <em>et al</em>. convey is critical and the message needs an explicit sentence or two at end of paragraph.</li>

                <li>Page 4, Conclusion: the assertion concerning the ethics of the two Phase 3 clinical trials needs toning down. Perhaps rephrase to indicate that the value and sense of doing these trials is open to question, with attendant ethical implications, or softer wording to that effect.</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/3-124" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>
              
            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="clear-report"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Clear report emphasizes relevance of the article</div>
            <div class="example-authors">Jasper Rine says:</div>
            <div class="collapsed-section t-body u-mt--1">
              <p>The Wiley <em>et al</em>. manuscript describes a beautiful synthesis of contemporary genetic approaches to, with astonishing efficiency, identify lead compounds for therapeutic approaches to a serious human disease. I believe the importance of this paper stems from the applicability of the approach to the several thousand of rare human disease genes that Next-Gen sequencing will uncover in the next few years and the challenge we will have in figuring out the function of these genes and their resulting defects. This work presents a paradigm that can be broadly and usefully applied.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The Wiley <em>et al</em>. manuscript describes a beautiful synthesis of contemporary genetic approaches to, with astonishing efficiency, identify lead compounds for therapeutic approaches to a serious human disease. I believe the importance of this paper stems from the applicability of the approach to the several thousand of rare human disease genes that Next-Gen sequencing will uncover in the next few years and the challenge we will have in figuring out the function of these genes and their resulting defects. This work presents a paradigm that can be broadly and usefully applied.</p>

              <p>In detail, the authors begin with gene responsible for X-linked spinal muscular atrophy and express both the wild-type version of that human gene as well as a mutant form of that gene in <em>S. pombe</em>. The conceptual leap here is that progress in genetics is driven by phenotype, and this approach involving a yeast with no spine or muscles to atrophy is nevertheless and N-dimensional detector of phenotype.</p>

              <p>The study is not without a small measure of luck in that expression of the wild-type <em>UBA1</em> gene caused a slow growth phenotype which the mutant did not. Hence there was something in <em>S. pombe</em> that could feel the impact of this protein. Given this phenotype, the authors then went to work and using the power of the synthetic genetic array approach pioneered by Boone and colleagues made a systematic set of double mutants combining the human expressed UBA1 gene with knockout alleles of a plurality of <em>S. pombe</em> genes. They found well over a hundred mutations that either enhanced or suppressed the growth defect of the cells expressing UBI1. Most of these have human orthologs. My hunch is that many human genes expressed in yeast will have some comparably exploitable phenotype, and time will tell.</p>

              <p>Building on the interaction networks of <em>S. pombe</em> genes already established, augmenting these networks by the protein interaction networks from yeast and from human proteome studies involving these genes, and from the structure of the emerging networks, the authors deduced that an E3 ligase modulated <em>UBA1</em> and made the leap that it therefore might also impact X-linked Spinal Muscular Atrophy.</p>

              <p>Here, the awesome power of the model organism community comes into the picture as there is a zebrafish model of spinal muscular atrophy. The principle of phenologs articulated by the Marcotte group inspire the recognition of the transitive logic of how phenotypes in one organism relate to phenotypes in another. With this zebrafish model, they were able to confirm that an inhibitor of E3 ligases and of the Nedd8-E1 activating suppressed the motor axon anomalies, as predicted by the effect of mutations in <em>S. pombe</em> on the phenotypes of the UBA1 overexpression.</p>

              <p>I believe this is an important paper to teach in intro graduate courses as it illustrates beautifully how important it is to know about and embrace the many new sources of systematic genetic information and apply them broadly.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-121" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="detailed-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Detailed feedback about statistics in a critical review</div>
            <div class="example-authors">Michael McCarthy says:</div>

            <div class="collapsed-section">
              <p>This paper by Amrhein <em>et al.</em> criticizes a paper by Bradley Efron that discusses Bayesian statistics (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron, 2013a</a>), focusing on a particular example that was also discussed in <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a>. The example concerns a woman who is carrying twins, both male (as determined by sonogram and we ignore the possibility that gender has been observed incorrectly). The parents-to-be ask Efron to tell them the probability that the twins are identical.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>This paper by Amrhein <em>et al.</em> criticizes a paper by Bradley Efron that discusses Bayesian
              statistics (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron, 2013a</a>), focusing on a particular example that was also discussed in <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a>. The example concerns a woman who is carrying twins, both male (as determined by sonogram and we ignore the possibility that gender has been observed incorrectly). The parents-to-be ask Efron to tell them the probability that the twins are identical.</p>

              <p>This is my first open review, so I'm not sure of the protocol. But given that there appears to be errors in both <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a> and the paper under review, I am sorry to say that my review might actually be longer than the article by <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron (2013a)</a>, the primary focus of the critique, and the critique itself. I apologize in advance for this. To start, I will outline the problem being discussed for the sake of readers.</p>

              <p>This problem has various parameters of interest. The primary parameter is the genetic composition of the twins in the mother&rsquo;s womb. Are they identical (which I describe as the state <em>x</em> = 1) or fraternal twins (<em>x</em> = 0)? Let <em>y</em> be the data, with <em>y</em> = 1 to indicate the twins are the same gender. Finally, we wish to obtain Pr(<em>x</em> = 1 | <em>y</em> = 1), the probability the twins are identical given they are the same gender<a href="#footnote1" class="footnote-link">1</a>. Bayes&rsquo; rule gives us an expression for this:</p>

              <p>Pr(<em>x</em> = 1 | <em>y</em> = 1) = Pr(<em>x</em>=1) Pr(<em>y</em> = 1 | <em>x</em> = 1) / {Pr(<em>x</em>=1) Pr(<em>y</em> = 1 | <em>x</em> = 1) + Pr(<em>x</em>=0) Pr(<em>y</em> = 1 | <em>x</em> = 0)}</p>

              <p>Now we know that Pr(<em>y</em> = 1 | <em>x</em> = 1) = 1; twins must be the same gender if they are identical. Further, Pr(<em>y</em> = 1 | <em>x</em> = 0) = 1/2; if twins are not identical, the probability of them being the same gender is 1/2.</p>

              <p>Finally, Pr(<em>x</em> = 1) is the prior probability that the twins are identical. The bone of contention in the Efron papers and the critique by Amrhein <em>et al.</em> revolves around how this prior is treated. One can think of Pr(<em>x</em> = 1) as the population-level proportion of twins that are identical for a mother like the one being considered.</p>

              <p>However, if we ignore other forms of twins that are extremely rare (equivalent to ignoring coins finishing on their edges when flipping them), one incontrovertible fact is that Pr(<em>x</em> = 0) = 1 − Pr(<em>x</em> = 1); the probability that the twins are fraternal is the complement of the probability that they are identical.</p>

              <p>The above values and expressions for Pr(<em>y</em> = 1 | <em>x</em> = 1), Pr(<em>y</em> = 1 | <em>x</em> = 0), and Pr(<em>x</em> = 0) leads to a simpler expression for the probability that we seek &hyphen;
              the probability that the twins are identical given they have the same gender:</p>

              <p>Pr(<em>x</em> = 1 | <em>y</em> = 1) = 2 Pr(<em>x</em>=1) / [1 + Pr(<em>x</em>=1)] (1)</p>

              <p>We see that the answer depends on the prior probability that the twins are identical, Pr(<em>x</em>=1). The paper by Amrhein <em>et al.</em> points out that this is a mathematical fact. For example, if identical twins were impossible (Pr(<em>x</em> = 1) = 0), then Pr(<em>x</em> = 1| <em>y</em> = 1) = 0. Similarly, if all twins were identical (Pr(<em>x</em> = 1) = 1), then Pr(<em>x</em> = 1| <em>y</em> = 1) = 1. The &ldquo;true&rdquo; prior lies somewhere in between. Apparently, the doctor knows that one third of twins are identical<a href="#footnote2" class="footnote-link">2</a>. Therefore, if we assume Pr(<em>x</em> = 1) = 1/3, then Pr(<em>x</em> = 1| <em>y</em> = 1) = 1/2.</p>

              <p>Now, what would happen if we didn't have the doctor's knowledge? Laplace's &ldquo;Principle of Insufficient Reason&rdquo; would suggest that we give equal prior probability to all possibilities, so Pr(<em>x</em> = 1) = 1/2 and Pr(<em>x</em> = 1| <em>y</em> = 1) = 2/3, an answer different from 1/2 that was obtained when using the doctor's prior of 1/3.</p>

              <p><a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron(2013a)</a> highlights this sensitivity to the prior, representing someone who defines an uninformative prior as a &ldquo;violator&rdquo;, with Laplace as the &ldquo;prime violator&rdquo;. In contrast, Amrhein <em>et al.</em> correctly points out that the difference in the posterior probabilities is merely a consequence of mathematical logic. No one is violating logic &ndash; they are merely expressing ignorance by specifying equal probabilities to all states of nature. Whether this is philosophically valid is debatable (<a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1539-6924.2008.01058.x" target="_blank" class="in-text-link tiny">Colyvan 2008</a>), but weight to that question, and it is well beyond the scope of this review. But setting Pr(<em>x</em> = 1) = 1/2 is not a violation; it is merely an assumption with consequences (and one that in hindsight might be incorrect<a href="#footnote2" class="footnote-link">2</a>).</p>

              <p>Alternatively, if we don't know Pr(<em>x</em> = 1), we could describe that probability by its own probability distribution. Now the problem has two aspects that are uncertain. We don&rsquo;t know the true state <em>x</em>, and we don&rsquo;t know the prior (except in the case where we use the doctor&rsquo;s knowledge that Pr(<em>x</em> = 1) = 1/3). Uncertainty in the state of <em>x</em> refers to uncertainty about this particular set of twins. In contrast, uncertainty in Pr(<em>x</em> = 1) reflects uncertainty in the population-level frequency of identical twins. A key point is that the state of one particular set of twins is a different parameter from the frequency of occurrence of identical twins in the population.</p>

              <p>Without knowledge about Pr(<em>x</em> = 1), we might use Pr(<em>x</em> = 1) ~ dunif(0, 1), which is consistent with Laplace. Alternatively, Efron (2013b) notes another alternative for an uninformative prior: Pr(<em>x</em> = 1) ~ dbeta(0.5, 0.5), which is the Jeffreys prior for a probability.</p>

              <p>Here I disagree with Amrhein <em>et al.</em>; I think they are confusing the two uncertain parameters. Amrhein <em>et al.</em> state:</p>

              <p><em>&ldquo;We argue that this example is not only flawed, but useless in illustrating Bayesian data analysis because it does not rely on any data. Although there is one data point (a couple is due to be parents of twin boys, and the twins are fraternal), Efron does not use it to update prior knowledge. Instead, Efron combines different pieces of expert knowledge from the doctor and genetics using Bayes&rsquo; theorem.&rdquo;</em></p>

              <p>This claim might be correct when describing uncertainty in the population-level frequency of identical twins. The data about the twin boys is not useful by itself for this purpose &ndash; they are a biased sample (the data have come to light because their gender is the same; they are not a random sample of twins). Further, a sample of size one, especially if biased, is not a firm basis for inference about a population parameter. While the data are biased, the claim by Amrheim <em>et al.</em> that there are no data is incorrect.</p>

              <p>However, the data point (the twins have the same gender) is entirely relevant to the question about the state of this particular set of twins. And it does update the prior. This updating of the prior is given by equation (1) above. The doctor&rsquo;s prior probability that the twins are identical (1/3) becomes the posterior probability (1/2) when using information that the twins are the same gender. The prior is clearly updated with Pr(<em>x</em> = 1| <em>y</em> = 1) &ne; Pr(<em>x</em> = 1) in all but trivial cases; Amrheim <em>et al.</em>&rsquo;s statement that I quoted above is incorrect in this regard.</p>

              <p>This possible confusion between uncertainty about these twins and uncertainty about the population level frequency of identical twins is further suggested by Amrhein <em>et al.</em>&rsquo;s statements:</p>

              <p>&ldquo;Second, for the uninformative prior, Efron mentions erroneously that he used a uniform distribution between zero and one, which is clearly different from the value of 0.5 that was used. Third, we find it at least debatable whether a prior can be called an uninformative prior if it has a fixed value of 0.5 given without any measurement of uncertainty.&rdquo;</p>

              <p>Note, if the prior for Pr(<em>x</em> = 1) is specified as 0.5, or dunif(0,1), or dbeta(0.5, 0.5), the posterior probability that these twins are identical is 2/3 in all cases. <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a> says the different priors lead to different results, but this result is incorrect, and the correct answer (2/3) is given in <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron (2013a)</a><a href="#footnote3" class="footnote-link">3</a>. Nevertheless, a prior that specifies Pr(<em>x</em> = 1) = 0.5 does indicate uncertainty about whether this particular set of twins is identical (but certainty in the population level frequency of twins). And <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron&rsquo;s (2013a)</a> result is consistent with Pr(<em>x</em> = 1) having a uniform prior. Therefore, both claims in the quote above are incorrect.</p>

              <p>It is probably easiest to show the (lack of) influence of the prior using MCMC sampling. Here is WinBUGS code for the case using Pr(<em>x</em> = 1) = 0.5.</p>
<pre>
model
{
  pr_ident_twins &lt;- 0.5   # prior probability that the twins are identical
  x ~ dbern(pr_ident_twins)   # are they identical? If so, x = 1, and 0 otherwise
 
  pr_same_gender &lt;- x + (1-x)*0.5   # the probability that the twins have the same gender. It equals 1 if x = 1, and 0.5 otherwise (i.e., if x = 0)
 
  same_gender &lt;- 1  # the single data point - the twins are the same gender
  same_gender ~ dbern(pr_same_gender)   # those data arise as a Bernoulli sample with probability pr_same_gender
}
</pre>

              <p>Running this model in WinBUGS shows that the posterior mean of <em>x</em> is 2/3; this is the posterior probability that <em>x</em> = 1.</p>

              <p>Instead of using pr_ident_twins &lt;- 0.5, we could set this probability as being uncertain and define pr_ident_twins ~ dunif(0,1), or pr_ident_twins ~ dbeta(0.5,0.5). In either case, the posterior mean value of <em>x</em> remains 2/3 (contrary to <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron 2013b</a>, but in accord with the correction in <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron 2013a</a>).</p>

              <p>Note, however, that the value of the population level parameter pr_ident_twins is different in all three cases. In the first it remains unchanged at 1/2 where it was set. In the case where the prior distribution for pr_ident_twins is uniform or beta, the posterior distributions remain broad, but they differ depending on the prior (as they should &ndash; different priors lead to different posteriors <a href="#footnote4" class="footnote-link">4</a>). However, given the biased sample size of 1, the posterior distribution for this particular parameter is likely to be misleading as an estimate of the population-level frequency of twins.</p>

              <p>So why doesn&rsquo;t the choice of prior influence the posterior probability that these twins are identical? Well, for these three priors, the prior probability that any single set of twins is identical is 1/2 (this is essentially the mean of the prior distributions in these three cases).</p>

              <p>If, instead, we set the prior as dbeta(1,2), which has a mean of 1/3, then the posterior probability that these twins are identical is 1/2. This is the same result as if we had set Pr(<em>x</em> = 1) = 1/3. In both these cases (choosing dbeta(1,2) or 1/3), the prior probability that a single set of twins is identical is 1/3, so the posterior is the same (1/2) given the data (the twins have the same gender).</p>
              
              <p>Further, Amrhein <em>et al.</em> also seem to misunderstand the data. They note:</p>
              
              <p>&ldquo;Although there is one data point (a couple is due to be parents of twin boys, and the twins are fraternal)...&rdquo;</p>
              
              <p>This is incorrect. The parents simply know that the twins are both male. Whether they are fraternal is unknown (fraternal twins being the complement of identical twins) &ndash; that is the question the parents are asking. This error of interpretation makes the calculations in Box 1 and subsequent comments irrelevant.</p>
              
              <p>Box 1 also implies Amrhein <em>et al.</em> are using the data to estimate the population frequency of identical twins rather than the state of this particular set of twins. This is different from the aim of <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron (2013a)</a> and the stated question.</p>

              <p>Efron suggests that Bayesian calculations should be checked with frequentist methods when priors are uncertain. However, this is a good example where this cannot be done easily, and Amrhein <em>et al.</em> are correct to point this out. In this case, we are interested in the probability that the hypothesis is true given the data (an inverse probability), not the probabilities that the observed data would be generated given particular hypotheses (frequentist probabilities). If one wants the inverse probability (the probability the twins are identical given they are the same gender), then Bayesian methods (andtherefore a prior) are required. A logical answer simply requires that the prior is constructed logically. Whether that answer is &ldquo;correct&rdquo; will be, in most cases, only known in hindsight.</p>
              
              <p>However, one possible way to analyse this example using frequentist methods would be to assess the likelihood of obtaining the data for each of the two hypothesis (the twins are identical or fraternal). The likelihood of the twins having the same gender under the hypothesis that they are identical is 1. The likelihood of the twins having the same gender under the hypothesis that they are fraternal is 0.5. Therefore, the weight of evidence in favour of identical twins is twice that of fraternal twins. Scaling these weights so they sum to one (<a href="http://www.springer.com/statistics/statistical+theory+and+methods/book/978-0-387-95364-9" target="_blank" class="in-text-link tiny">Burnham and Anderson 2002</a>), gives a weight of 2/3 for identical twins and 1/3 for fraternal twins. These scaled weights have the same numerical values as the posterior probabilities based on either a Laplace or Jeffreys prior. Thus, one might argue that the weight of evidence for each hypothesis when using frequentist methods is equivalent to the posterior probabilities derived from an uninformative prior. So, as a final aside in reference to <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron (2013a)</a>, if we are being &ldquo;violators&rdquo; when using a uniform prior, are we also being &ldquo;violators&rdquo; when using frequentist methods to weigh evidence? Regardless of the answer to this rhetorical question, &ldquo;checking&rdquo; the results with frequentist methods doesn&rsquo;t give any more insight than using uninformative priors (in this case). However, this analysis shows that the question can be analysed using frequentist methods; the single data point is not a problem for this. The claim in Armhein <em>et al.</em> that a frequentist analyis "is impossible because there is only one data point, and frequentist methods generally cannot handle such situations" is not supported by this example.</p>

              <p>In summary, the comment by Amrhein <em>et al.</em> raises some interesting points that seem worth discussing, but it makes important errors in analysis and interpretation, and misrepresents the results of <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank" class="in-text-link tiny">Efron (2013a)</a>. This means the current version should not be approved.</p>

              <div class="detail-report-subheading">References</div>
              <p>Burnham, K.P. & D.R. Anderson. 2002. Model Selection and Multi-model Inference: a Practical Information-theoretic Approach. Springer-Verlag, New York.</p>

              <p>Colyvan, M. 2008. Is Probability the Only Coherent Approach to Uncertainty? <em>Risk Anal. </em> 28: 645-652.</p>

              <p>Efron B. (2003a) Bayes&rsquo; Theorem in the 21st Century. <em>Science</em> 340(6137): 1177-1178.</p>
              
              <p>Efron B. (2013b) A 250-year argument: Belief, behavior, and the bootstrap. <em>Bull Amer. Math Soc. </em> 50: 129-146.</p>

              <div class="detail-report-subheading">Footnotes</div>

              <ul class="number-list">
                <li><a id="footnote1"></a>The twins are both male. However, if the twins were both female, the statistical results would be the same, so I will simply use the data that the twins are the same gender.</li>
                
                <li><a id="footnote2"></a>In reality, the frequency of twins that are identical is likely to vary depending on many factors but we will accept 1/3 for now.</li>
                
                <li><a id="footnote3"></a><a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a> reports the posterior probability for these twins being identical as &ldquo;a whopping 61.4% with a flat Laplace prior&rdquo; but as 2/3 in <a href="http://www.ncbi.nlm.nih.gov/pubmed/23744934" target="_blank">Efron (2013a)</a>. The latter (I assume 2/3 is &ldquo;even more whopping&rdquo;!) is the correct answer, which I confirmed via email with Professor Efron. Therefore, <a href="http://www.ams.org/journals/bull/2013-50-01/S0273-0979-2012-01374-5/home.html" target="_blank" class="in-text-link tiny">Efron (2013b)</a> incorrectly claims the posterior probability is sensitive to the choice between a Jeffreys or Laplace uninformative prior.</li>

                <li><a id="footnote4"></a>When the data are very informative relative to the different priors, the posteriors will be similar, although not identical.</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/2-278" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="constructive-feedback"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Constructive feedback on an opinion article</div>
            <div class="example-authors">Mark Parsons and Peter Fox say:</div>
            
            <div class="collapsed-section">
              <p>I am very glad the authors wrote this essay. It is a well-written, needed, and useful summary of the current status of &ldquo;data publication&rdquo; from a certain perspective. The authors, however, need to be bolder and more analytical. This is an opinion piece, yet I see little opinion. A certain view is implied by the organization of the paper and the references chosen, but they could be more explicit.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>I am very glad the authors wrote this essay. It is a well-written, needed, and useful summary of the current status of &ldquo;data publication&rdquo; from a certain perspective. The authors, however, need to be bolder and more analytical. This is an opinion piece, yet I see little opinion. A certain view is implied by the organization of the paper and the references chosen, but they could be more explicit.</p> 
              
              <p>The paper would be both more compelling and useful to a broad readership if the authors moved beyond providing a simple summary of the landscape and examined why there is controversy in some areas and then use the evidence they have compiled to suggest a path forward. They need to be more forthright in saying what data publication means to them, or what parts of it they do not deal with. Are they satisfied with the Lawrence <em>et al.</em> definition? Do they accept the critique of Parsons and Fox? What is the scope of their essay?</p>

              <p>The authors take a rather narrow view of data publication, which I think hinders their analyses. They describe three types of (digital) data publication: Data as a supplement to an article; data as the subject of a paper; and data independent of a paper. The first two types are relatively new and they represent very little of the data actually being published or released today. The last category, which is essentially an &ldquo;other&rdquo; category, is rich in its complexity and encompasses the vast majority of data released. I was disappointed that the examples of this type were only the most bare-bones (Zenodo and Figshare). I think a deeper examination of this third category and its complexity would help the authors better characterize the current landscape and suggest paths forward.</p>
              
              <p>Some questions the authors might consider: Are these really the only three models in consideration or does the publication model overstate a consensus around a certain type of data publication? Why are there different models and which approach is better for different situations? Do they have different business models or imply different social contracts? Might it also be worthy of typing &ldquo;publishers&rdquo; instead of &ldquo;publications&rdquo;? For example, do domain repositories vs. institutional repositories vs. publishers address the issues differently? Are these models sustaining models or just something to get us through the next 5-10 years while we really figure it out?</p>
              
              <p>I think this oversimplification inhibited some deeper analysis in other areas as well. I would like to see more examination of the validation requirement beyond the lens of peer review, and I would like a deeper examination of incentives and credit beyond citation.</p>

              <p>I thought the validation section of the paper was very relevant, but somewhat light. I like the choice of the term validation as more accurate than &ldquo;quality&rdquo; and it fits quite well with Callaghan&rsquo;s useful distinction between technical and scientific review, but I think the authors overemphasize the peer-review style approach. The authors rightly argue that &ldquo;peer-review&rdquo; is where the publication metaphor leads us, but it may be a false path. They overstate some difficulties of peer-review (No-one looks at every data value? No, they use statistics, visualization, and other techniques.) while not fully considering who is responsible for what. We need a closer examination of different roles and who are appropriate validators (not necessarily conventional peers). The narrowly defined models of data publication may easily allow for a conventional peer-review process, but it is much more complex in the real-world &ldquo;other&rdquo; category. The authors discuss some of this in what they call &ldquo;independent data validation,&rdquo; but they don&rsquo;t draw any conclusions.</p>

              <p>Only the simplest of research data collections are validated only by the original creators. More often there are teams working together to develop experiments, sampling protocols, algorithms, etc. There are additional teams who assess, calibrate, and revise the data as they are collected and assembled. The authors discuss some of this in their examples like the PDS and tDAR, but I wish they were more analytical and offered an opinion on the way forward. Are there emerging practices or consensus in these team-based schemes? The level of service concept illustrated by Open Context may be one such area. Would formalizing or codifying some of these processes accomplish the same as peer-review or more? What is the role of the curator or data scientist in all of this? Given the authors&rsquo;s backgrounds, I was surprised this role was not emphasized more. Finally, I think it is a mistake for science review to be the main way to assess reuse value. It has been shown time and again that data end up being used effectively (and valued) in ways that original experts never envisioned or even thought valid.</p>

              <p>The discussion of data citation was good and captured the state of the art well, but again I would have liked to see some views on a way forward. Have we solved the basic problem and are now just dealing with edge cases? Is the &ldquo;just-in-time identifier&rdquo; the way to go? What are the implications? Will the more basic solutions work in the interim? More critically, are we overemphasizing the role of citation to provide academic credit? I was gratified that the authors referenced the Parsons and Fox paper which questions the whole data publication metaphor, but I was surprised that they only discussed the &ldquo;data as software&rdquo; alternative metaphor. That is a useful metaphor, but I think the ecosystem metaphor has broader acceptance. I mention this because the authors critique the software metaphor because &ldquo;using it to alter or affect the academic reward system is a tricky prospect&rdquo;. Yet there is little to suggest that data publication and corresponding citation alters that system either. Indeed there is little if any evidence that data publication and citation incentivize data sharing or stewardship. As Christine Borgman suggests, <em>we need to look more closely at who we are trying to incentivize to do what.</em> There is no reason to assume it follows the same model as research literature publication. It may be beyond the scope of this paper to fully examine incentive structures, but it at least needs to be acknowledged that building on the current model doesn&rsquo;t seem to be working.</p>

              <p>Finally, what is the takeaway message from this essay? It ends rather abruptly with no summary, no suggested directions or immediate challenges to overcome, no call to action, no indications of things we should stop trying, and only brief mention of alternative perspectives. What do the authors want us to take away from this paper?</p>

              <p>Overall though, this is a timely and needed essay. It is well researched and nicely written with rich metaphor. With modifications addressing the detailed comments below and better recognizing the complexity of the current data publication landscape, this will be a worthwhile review paper. With more significant modification where the authors dig deeper into the complexities and controversies and truly grapple with their implications to suggest a way forward, this could be a very influential paper. It is possible that the definitions of &ldquo;publication&rdquo; and &ldquo;peer-review&rdquo; need not be just stretched but changed or even rejected.</p>

              <div class="detail-report-subheading">Detailed Comments:</div>

              <ul class="bullet-list">
                <li>The whole paper needs a quick copy edit. There are a few typos, missing words, and wrong verb tenses. Note the word &ldquo;data&rdquo; is a plural noun. E.g., Data are not software, nor are they literature. (NSICD, instead of NSIDC)</li>

                <li><span class="bold">Page 2, para 2:</span> &ldquo;citability is addressed by assigning a PID.&rdquo; This is not true, as the authors discuss on page 4, para 4. Indeed, page 4, para 4 seems to contradict itself. Citation is more than a locator/identifier.</li>

                <li>In the discussion of &ldquo;Data independent of any paper&rdquo; it is worth noting that there may often be linkages between these data and myriad papers. Indeed a looser concept of a data paper has existed for some time, where researchers request a citation to a paper even though it is not the data nor fully describes the data (e.g the CRU temp records)</li>

                <li><span class="bold">Page 4, para 1:</span> I&rsquo;m not sure it&rsquo;s entirely true that published data cannot involve requesting permission. In past work with Indigenous knowledge holders, they were willing to publish summary data and then provide the details when satisfied the use was appropriate and not exploitive. I think those data were &ldquo;published&rdquo; as best they could be. A nit, perhaps, but it highlights that there are few if any hard and fast rules about data publication.</li>

                <li><span class="bold">Page 4, para 2:</span> You may also want to mention the WDS certification effort, which is combining with the DSA via an RDA Working Group:</li>

                <li><span class="bold">Page 4, para 2:</span> The joint declaration of data citation principles involved many more organizations than Force11, CODATA, and DCC. Please credit them all (maybe in a footnote). The glory of the effort was that it was truly a joint effort across many groups. There is no leader. Force11 was primarily a convener.</li>

                <li><span class="bold">Page 4, para 6:</span> The deep citation approach recommended by ESIP is not to just to list variables or a range of data. It is to identify a &ldquo;structural index&rdquo; for the data and to use this to reference subsets. In Earth science this structural index is often space and time, but many other indices are possible--location in a gene sequence, file type, variable, bandwidth, viewing angle, etc. It is not just for &ldquo;straightforward&rdquo; data sets.</li>
                
                <li><span class="bold">Page 5, para 5:</span> I take issue with the statement that few repositories provide scientific review. I can think of a couple dozen that do just off the top of my head, and I bet most domain repositories have some level of science review. The &ldquo;scientists&rdquo; may not always be in house, but the repository is a team facilitator. See my general comments.</li>
                
                <li><span class="bold">Page 5, para 10:</span> The PDS system is only unusual in that it is well documented and advertised. As mentioned, this team style approach is actually fairly common.</li>
                
                <li><span class="bold">Page 6, para 3:</span> Parsons and Fox don&rsquo;t just argue that the data publication metaphor is limiting. They also say it is misleading. That should be acknowledged at least, if not actively grappled with.</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/3-94" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="software-review"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Response to authors who did not make sufficient changes to article</div>
            <div class="example-authors">Leo McHugh says:</div>

            <div class="collapsed-section">

              <ul class="number-list">
                <li><span class="bold">Artifact removal:</span> Unfortunately the authors have not updated the paper with a 2x2 table showing guns and smiles by removed data points. This could dispel criticism that an asymmetrical expectation bias that has been shown to exist in similar experiments is not driving a bias leading to inappropriate conclusions.</li>
              </ul>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <ul class="number-list">
                <li><span class="bold">Artifact removal:</span> Unfortunately the authors have not updated the paper with a 2x2 table showing guns and smiles by removed data points. This could dispel criticism that an asymmetrical expectation bias that has been shown to exist in similar experiments is not driving a bias leading to inappropriate conclusions. This is my strongest criticism of the paper and should be easily addressed as per my previous review comment. The fact that this simple data presentation was not performed to remove a clear potential source of spurious results is disappointing.</li>

                <li>The authors have added 95% CIs to figures S1 and S2. This clarifies the scope for expectation bias in these data. The addition of error bars permits the authors&rsquo; assumption of a linear trend, indicating that the effect of sequences of either guns or smiles may not skew results. Equally, there could be either a downwards or upwards trend fitting within the confidence intervals that could be indicative of a cognitive bias that may violate the assumptions of the authors, leading to spurious results. One way to remove these doubts could be to stratify the analyses by the length of sequences of identical symbols. If the results hold up in each of the strata, this potential bias could be shown to not be present in the data. If the bias is strong, particularly in longer runs, this could indicate that the positive result was due to small numbers of longer identical runs combined with a cognitive bias rather than an ability to predict future events.</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/2-262" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Suggestions for further improvement of an approved article</div>
            <div class="example-authors">Gavin Simpson says:</div>
            
            <div class="collapsed-section">
              <p>Chamberlain and Szöcs present the taxize R package, a set of functions that provides interfaces to several web tools and databases, and simplifies the process of checking, updating, correcting and manipulating taxon names for researchers working with ecological/biological data. A key feature that is repeated throughout is the need for reproducibility of science workflows and taxize provides a means to achieve this within the R software ecosystem for taxonomic search.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>Chamberlain and Szöcs present the taxize R package, a set of functions that provides interfaces to several web tools and databases, and simplifies the process of checking, updating, correcting and manipulating taxon names for researchers working with ecological/biological data. A key feature that is repeated throughout is the need for reproducibility of science workflows and taxize provides a means to achieve this within the R software ecosystem for taxonomic search.</p>

              <p>The manuscript is well-written and nicely presented, with a good balance of descriptive text and discourse and practical illustration of package usage. A number of examples illustrate the scope of the package, something that is fully expanded upon in the two appendices, which are a welcome addition to the paper.</p>

              <p>As to the package, I am not overly fond of long function names; the authors should consider dropping the data source abbreviations from the function names in a future update/revision of the package. Likewise there is some inconsistency in the naming conventions used. For example there is the &rsquo;tpl_search()&rsquo; function to search The Plant List, but the equivalent function to search uBio is &rsquo;ubio_namebank()&rsquo;. Whilst this may reflect specific aspects of terminology in use at the respective data stores, it does not help the user gain familiarity with the package by having them remember inconsistent function names.</p>

              <p>One advantage of taxize is that it draws together a rich selection of data stores to query. A further suggestion for a future update would be to add generic function names, that apply to a database connection/information object. The latter would describe the resource the user wants to search and any other required information, such as the API key, etc., for example:</p>

              <code>foo &lt;- taxizeDB(what = "uBio", key = "1646546164694")</code>

              <p>The user function to search would then be &rsquo;search(foo, "Abies")&rsquo;. Similar generically named functions would provide the primary user-interface, thus promoting a more consistent toolbox at the R level. This will become increasingly relevant as the scope of taxize increases through the addition of new data stores that the package can access.</p>

              <p>In terms of presentation in the paper, I really don&rsquo;t like the way the R code inputs merge with the R outputs. I know the author of Knitr doesn&rsquo;t like the demarcation of output being polluted by the R prompt, but I do find it difficult parsing the inputs/outputs you show because often there is no space between them and users not familiar with R will have greater difficulties than I. Consider adding in more conventional indications of R outputs, or physically separate input from output by breaking up the chunks of code to have whitespace between the grey-background chunks. Related, in one location I noticed something amiss with the layout; in the first code block at the top of page 5, the printed output looks wrong here. I would expect the attributes to print on their own line and the data in the attribute to also be on its own separate line.</p>
              
              <p>Note also, the inconsistency in the naming of the output object columns. For example, in the two code chunks shown in column 1 of page 4, the first block has an object printed with column names &rsquo;matched_name&rsquo; and &rsquo;data_source_title&rsquo;, whilst camelCase is used in the outputs shown in the second block. As the package is revised and developed, consider this and other aspects of providing a consistent presentation to the user.</p>
              
              <p>I was a little confused about the example in the section Resolve Taxonomic Names on page 4. Should the taxon name be <em>&ldquo;Helianthus annuus&rdquo;</em> or <em>&ldquo;Helianthus annus&rdquo;</em>? In the &lsquo;mynames&rsquo; definition you include <em>&lsquo;Helianthus annuus&rsquo;</em> in the character vector but the output shown suggests that the submitted name was <em>&lsquo;Helianthus annus&rsquo;</em> (1 &ldquo;u&rdquo;) in rows with rownames 9 and 10 in the output shown.</p>
              
              <p>Other than that there were the following minor observations:</p>
              
              <ul class="number-list">
                <li>Abstract: replace &ldquo;easy&rdquo; with &ldquo;simple&rdquo; in <em>&ldquo;...fashion that&rsquo;s easy...&rdquo;</em>, and move the details about availability and the URI to the end of the sentence.</li>

                <li>Page 2, Column 1, Paragraph 2: You <em>have &ldquo;In addition, there is no one authoritative taxonomic names source...&rdquo;</em>, which is a little clumsy to read. How about <em>&ldquo;In addition, there is no one authoritative source of taxonomic names... &rdquo;</em>?</li>
                
                <li>Pg 2, C1, P2-3: The abbreviated data sources are presented first (in paragraph 2) and subsequently defined (in para 3). Restructure this so that the abbreviated forms are explained upon first usage.</li>

                <li>Pg 2, C2, P2: Most R packages are &ldquo;in development&rdquo; so I would drop the qualifier and reword the opening sentence of the paragraph.</li>

                <li>Pg 2, C2, P6: Change <em>&ldquo;and more can easily be added&rdquo;</em> to <em>&ldquo;and more can be easily added&rdquo;</em> seems to flow better?</li>

                <li>Pg 5, paragraph above Figure 1: You refer to converting the object to an **ape** *phylo* object and then repeat essentially the same information in the next sentence. Remove the repetition.</li>

                <li>Pg 6, C1: The header may be better as <em>&ldquo;Which taxa are children of the taxon of interest&rdquo;</em>.</li>

                <li>Pg 6: In the section &ldquo;IUCN status&rdquo;, the term &ldquo;we&rdquo; is used to refer to both the authors and the user. This is confusing. Reserve &ldquo;we&rdquo; for reference to the authors and use something else (&ldquo;a user&rdquo; perhaps) for the other instances. Check this throughout the entire manuscript.</li>

                <li>Pg 6, C2: in the paragraph immediately below the &lsquo;grep()&rsquo; for &ldquo;RAG1&rdquo;, two consecutive sentences begin with &ldquo;However&rdquo;.</li>

                <li>Pg 7: The first sentence of &ldquo;Aggregating data....&rdquo; reads <em>&ldquo;In biology, one can asks questions...&rdquo;</em>. It should be <em>&ldquo;one asks&rdquo;</em> or <em>&ldquo;one can ask&rdquo;</em>.</li>

                <li>Pg 7, Conclusions: The first sentence reads <em>&ldquo;information is increasingly sought out by biologists&rdquo;</em>. I would drop &ldquo;out&rdquo; as &ldquo;sought&rdquo; is sufficient on its own.</li>

                <li>Appendices: Should the two figures in the Appendices have a different reference to differentiate them from Figure 1 in the main body of the paper? As it stands, the paper has two Figure 1s, one on page 5 and a second on page 12 in the Appendix.</li>

                <li>On Appendix Figure 2: The individual points are a little large. Consider reducing the plotting character size. I appreciate the effect you were going for with the transparency indicating density of observation through overplotting, but the effect is weakened by the size of the individual points.</li>
                
                <li>Should the phylogenetic trees have some scale to them? I presume the height of the stems is an indication of phylogenetic distance but the figure is hard to calibrate without an associated scale. A quick look at <a href="http://www.springer.com/life+sciences/evolutionary+%26+developmental+biology/book/978-1-4614-1742-2" target="_blank">Paradis (2012) Analysis of Phylogenetics and Evolution with R</a> would suggest however that a scale is not consistently applied to these trees. I am happy to be guided by the authors as they will be more familiar with the conventions than I.</li>
              </ul>
              
              <div class="link-row expanded-example">
                <a href="/articles/2-191" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Peer review of a review article</div>
            <div class="example-authors">Gustavo Gutierrez Gonzalez says:</div>
            <div class="collapsed-section">
              <p>Hydbring and Badalian-Very summarize in this review, the current status in the potential development of clinical applications based on miRNAs&rsquo; biology. The article gives an interesting historical and scientific perspective on a field that has only recently boomed.</p>
              
              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>Hydbring and Badalian-Very summarize in this review, the current status in the potential development of clinical applications based on miRNAs&rsquo; biology. The article gives an interesting historical and scientific perspective on a field that has only recently boomed; focusing mostly on the two main products in the pipeline of several biotech companies (in Europe and USA) which work with miRNAs-based agents, disease diagnostics and therapeutics. Interestingly, not only the specific agents that are being produced are mentioned, but also clever insights in the important cellular pathways regulated by key miRNAs are briefly discussed.</p>

              <p>Minor points to consider in subsequent versions:</p>

              <ul class="number-list">
                <li>Page 2; paragraph <em>&lsquo;Genomic location and transcription of microRNAs&rsquo;</em>: the concept of miRNA clusters and precursors could be a bit better explained.</li>

                <li>Page 2; paragraph <em>&lsquo;Genomic location and transcription of microRNAs&rsquo;</em>: when discussing the paper by the laboratory of Richard Young (reference 16); I think it is important to mention that that particular study refers to stem cells.</li>

                <li>Page 2; paragraph <em>&lsquo;Processing of microRNAs&rsquo;</em>: &ldquo;Argonate&rdquo; should be replaced by &ldquo;Argonaute&rdquo;.</li>

                <li>Page 3; paragraph <em>&lsquo;MicroRNAs in disease diagnostics&rsquo;</em>: are miR-15a and 16-1 two different miRNAs? I suggest mentioning them as: miR-15a and miR-16-1 and not using a slash sign (/) between them.</li>

                <li>Page 4; paragraph <em>&lsquo;Circulating microRNAs&rsquo;</em>: I am a bit bothered by the description of multiple sclerosis (MS) only as an autoimmune disease. Without being an expert in the field, I believe that there are other hypotheses related to the etiology of MS.</li>

                <li>Page 5; paragraph <em>&lsquo;Clinical microRNA diagnostics&rsquo;</em>: Does &lsquo;hsa&rsquo; in hsa-miR-205 mean something?</li>

                <li>Page 5; paragraph <em>&lsquo;Clinical microRNA diagnostics&rsquo;</em>: the authors mention the company Asuragen, Austin, TX, USA but they do not really say anything about their products. I suggest to either remove the reference to that company or to include their current pipeline efforts.</li>

                <li>Page 6; paragraph <em>&lsquo;MicroRNAs in therapeutics&rsquo;</em>: in the first paragraph the authors suggest that miRNAs-based therapeutics should be able to be applied with &ldquo;minimal side-effects&rdquo;. Since one miRNA can affect a whole gene program, I found this a bit counterintuitive; I was wondering if any data has been published to support that statement. Also, in the same paragraph, the authors compare miRNAs to protein inhibitors, which are described as more specific and/or selective. I think there are now good indications to think that protein inhibitors are not always that specific and/or selective and that such a property actually could be important for their evidenced therapeutic effects.</li>

                <li>Page 6; paragraph <em>&lsquo;MicroRNAs in therapeutics&rsquo;</em>: I think the concept of &ldquo;antagomir&rdquo; is an important one and could be better highlighted in the text.</li>

                <li>Throughout the text (pages 3, 5, 6, and 7): I am a bit bothered by separating the word &ldquo;miRNA&rdquo; or &ldquo;miRNAs&rdquo; at the end of a sentence in the following way: &ldquo;miR-NA&rdquo; or &ldquo;miR-NAs&rdquo;. It is a bit confusing considering the particular nomenclature used for miRNAs. That was probably done during the formatting and editing step of the paper.</li>

                <li>I was wondering if the authors could develop a bit more the general concept that seems to indicate that in disease (and in particular in cancer) the expression and levels of miRNAs are in general downregulated. Maybe some papers have been published about this phenomenon?</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/2-136" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Review of an article describing negative results</div>
            <div class="example-authors">Christine Mummery says:</div>

            <div class="collapsed-section">
              <p>The authors describe their attempt to reproduce a study in which it was claimed that mild acid treatment was sufficient to reprogramme postnatal splenocytes from a mouse expressing GFP in the oct4 locus to pluripotent stem cells. The authors followed a protocol that has recently become available as a technical update of the original publication.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The authors describe their attempt to reproduce a study in which it was claimed that mild acid treatment was sufficient to reprogramme postnatal splenocytes from a mouse expressing GFP in the oct4 locus to pluripotent stem cells. The authors followed a protocol that has recently become available as a technical update of the original publication.</p>

              <p>They report obtaining no pluripotent stem cells expressing GFP driven over the same time period of several days described in the original publication. They describe observation of some green fluorescence that they attributed to autofluorescence rather than GFP since it coincided with PI positive dead cells. They confirmed the absence of oct4 expression by RT-PCR and also found no evidence for Nanog or Sox2, also markers of pluripotent stem cells.</p>

              <p>The paper appears to be an authentic attempt to reproduce the original study, although the study might have had additional value with more controls: &ldquo;failure to reproduce&rdquo; studies need to be particularly well controlled.</p>

              <p>Examples that could have been valuable to include are:</p>

              <ul class="number-list">
                <li>For the claim of autofluorescence: the emission spectrum of the samples would likely have shown a broad spectrum not coincident with that of GFP.</li>

                <li>The reprogramming efficiency of postnatal mouse splenocytes using more conventional methods in the hands of the authors would have been useful as a comparison. Idem the lung fibroblasts.</li>
                
                <li>There are no positive control samples (conventional mESC or miPSC) in the qPCR experiments for pluripotency markers. This would have indicated the biological sensitivity of the assay.</li>

                <li>Although perhaps a sensitive issue, it might have been helpful if the authors had been able to obtain samples of cells (or their mRNA) from the original authors for simultaneous analysis.</li>
              </ul>

              <p>In summary, this is a useful study as it is citable and confirms previous blog reports, but it could have been improved by more controls.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-102" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Comments on a case report</div>
            <div class="example-authors">Atanaska Elenkova says:</div>

            <div class="collapsed-section">
              <p>The article is well written, treats an actual problem (the risk of development of valvulopathy after long-term cabergoline treatment in patients with macroprolactinoma) and provides evidence about the reversibility of valvular changes after timely discontinuation of DA treatment.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>The article is well written, treats an actual problem (the risk of development of valvulopathy after long-term cabergoline treatment in patients with macroprolactinoma) and provides evidence about the reversibility of valvular changes after timely discontinuation of DA treatment.</p>
             
              <p><span class="bold">Title and abstract:</span> The title is appropriate for the content of the article. The abstract is concise and accurately summarizes the essential information of the paper although it would be better if the authors define more precisely the anatomic specificity of valvulopathy &ndash; mild mitral regurgitation.</p>
              
              <p><span class="bold">Case report:</span> The clinical case presentation is comprehensive and detailed but there are some minor points that should be clarified:</p>
              
              <ul class="number-list">
                <li>Please clarify the prolactin levels at diagnosis. In the <em>Presentation</em> section (line 3) &ldquo;At presentation, prolactin level was found to be greater than 1000 ng/ml on diluted testing&rdquo; but in the section describing the laboratory evaluation at diagnosis (line 7) &ldquo;Prolactin level was 55 ng/ml&rdquo;. Was the difference due to so called &ldquo;hook effect&rdquo;?</li>

                <li>Figure 1: In the text the follow-up MR imaging is indicated to be <em>&ldquo;after 10 months of cabergoline treatment&rdquo;</em>. However, the figures 1C and 1D represent 2 years post-treatment MR images. Please clarify.</li>

                <li>Figure 2: Echocardiograms 2A and 2B are defined as baseline but actually they correspond to the follow-up echocardiographic assessment at the 4th year of cabergoline treatment. Did the patient undergo a baseline (prior to dopamine agonist treatment) echocardiographic evaluation? If he did not, it should be mentioned as study limitation in the <em>Discussion</em> section.</li>

                <li>The mitral valve thickness was mentioned to be normal. Did the echographic examination visualize increased echogenicity (hyperechogenicity) of the mitral cusps?</li>

                <li>How could you explain the decrease of LV ejection fraction (from 60-65% to 50-55%) after switching from cabergoline to bromocriptine treatment and respectively its increase to 62% after doubling the bromocriptine daily dose? Was LV function estimated always by the same method during the follow-up?</li>

                <li><span class="bold">Final paragraph:</span> Authors conclude that early discontinuation and management with bromocriptine may be effective in reversing cardiac valvular dysfunction. Even though, regular echocardiographic follow up should be considered in patients who are expected to be on long-term high dose treatment with bromocriptine regarding its partial 5-HT2b agonist activity.</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/3-171" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Discussing sample size of a science communication study</div>
            <div class="example-authors">Sarah Davies says:</div>

            <div class="collapsed-section">
              <p>This is an interesting topic: as the authors note, the way that communicators imagine their audiences will shape their output in significant ways. And I enjoyed what clearly has the potential to be a very rich data set.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>This is an interesting topic: as the authors note, the way that communicators imagine their audiences will shape their output in significant ways. And I enjoyed what clearly has the potential to be a very rich data set. But I have some reservations about the adequacy of that data set, as it currently stands, given the claims the authors make; the relevance of the analytical framework(s) they draw upon; and the extent to which their analysis has offered significant new insights &hyphen; by which I mean, I would be keen to see the authors push their discussion further. My suggestions are essentially that they extend the data set they are working with to ensure that their analysis is both rigorous and generalisable, an re-consider the analytical frame they use. I will make some more concrete comments below.</p>
              
              <p>With regard to the data: my feeling is that 14 interviews is a rather slim data set, and that this is heightened by the fact that they were all carried out in a single location, and recruited via snowball sampling and personal contacts. What efforts have the authors made to ensure that they are not speaking to a single, small, sub-community in the much wider category of science communicators? &hyphen; a case study, if you like, of a particular group of science communicators in North Carolina? In addition, though the authors reference grounded theory as a method for analysis, I got little sense of the data reaching saturation. The reliance on one-off quotes, and on the stories and interests of particular individuals, left me unsure as to how <em>representative</em> interview extracts were. I would therefore recommend either that the data set is extended by carrying out more interviews, in a wider variety of locations (e.g. other sites in the US), or that it is redeveloped as a case study of a particular local professional community. (Which would open up some fascinating questions &hyphen; how many of these people know each other? What spaces, online or offline, do they interact in, and do they share knowledge, for instance about their audiences? Are there certain touchstone events or publics they communally make reference to?)</p>
              
              <p>As a more minor point with regard to the data set and what the authors want it to do, there were some inconsistencies as to how the study was framed. On p.2 they variously describe the purpose as to &ldquo;understand the experiences and perspectives of science communicators&rdquo; and the goals as identifying &ldquo;the basic interests and value orientations attributed to lay audiences by science communicators&rdquo;. Later, on p.5, they note that the &ldquo;research is inductive and seeks to build theory rather than generalizable claims&rdquo;, while in the Discussion they talk again about having identified communicators&lsquo; &ldquo;personal motivations&rdquo; (p.12). There are a number of questions left hanging: is the purpose to understand communicator experiences &hyphen; in which case why focus on perceptions of audiences? Where is theory being built, and in what ways can this be mobilised in future work? The way that the study is framed and argued as a whole needs, I would suggest, to be clarified.</p>

              <p>Relatedly, my sense is that some of this confusion is derived from what I find a rather busy analytical framework. I was not convinced of the value of combining inductive and deductive coding: if the &lsquo;human value typology&rsquo; the authors use is &lsquo;universal&rsquo;, then what is added by open coding? Or, alternatively, why let their open coding, and their findings from this, be constrained by an additional, rather rigid, framework? The addition of the considerable literature on news values to the mix makes the discussion more confusing again. I would suggest that the authors either make much more clear the value of combining these different approaches &hyphen; building new theory outlining how they relate, and can be jointly mobilised in practice &hyphen; or fix on one. (My preference would be to focus on the findings from the open coding &hyphen; but that reflects my own disciplinary biases.)</p>

              <p>A more minor analytical point: the authors note that their interviewees come from slightly different professions, and communicate through different formats, have different levels of experience, and different educational backgrounds &hyphen; but as far as I can see there is no comparative analysis based on this. Were there noticeable differences in the interview talk based on these categorisations? Or was the data set too small to identify any potential contrasts or themes? A note explaining this would be useful.</p>

              <p>My final point has reference to the potential that this data set has, particularly if it is extended and developed. I would like to encourage the authors to take their analysis further: at the moment, I was not particularly surprised by the ways in which the communicators referenced news values or imagined their audiences. But it seems to me that the analytical work is not yet complete. What does it <em>mean</em> that communicators imagine audience values and preferences in the way that they do &hyphen; who is included and excluded by these imaginations? One experiment might be to consider what &lsquo;ideal type&rsquo; publics are created in the communicators&rsquo; talk. What are the characteristics of the audiences constructed in the interviews and &hyphen; presumably &hyphen; in the communicative products of interviewees? What would these people look like? There are also some tantalizing hints in the Discussion that are not really discussed in the Findings &hyphen; of, for instance, the way in which communicator&rsquo;s personal motivations may combine with their perceptions of audiences to shape their products. How does this happen? These are, of course, suggestions. But my wider point is that the authors need to show more clearly what is original and useful in their findings &hyphen; what it is, exactly, that will be important to other scholars in the field.</p>

              <p>I hope my comments make sense &hyphen; please do not hesitate to contact me if not.</p>

              <div class="link-row expanded-example">
                <a href="/articles/3-128" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Reviewing bioinformatics software as well as the article</div>
            <div class="example-authors">Jordi Deu-Pons says:</div>

            <div class="collapsed-section">
              <p>This is an interesting article and piece of software. I think it contributes towards further alternatives to easily visualize high dimensionality data on the web. It&rsquo;s simple and easy to embed into other web frameworks or applications.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>This is an interesting article and piece of software. I think it contributes towards further alternatives to easily visualize high dimensionality data on the web. It&rsquo;s simple and easy to embed into other web frameworks or applications.</p>

              <div class="detail-report-subheading">Minor revisions</div>
              
              <p class="bold">a) About the software</p>
              
              <ul class="number-list">
                <li><span class="underlined">CSV format</span>. It was hard to guess the expected format. The authors need to add a syntax description of the CSV format at the help page.</li>

                <li><span class="underlined">Simple HTML example</span>. It will be easy to test HeatmapViewer (HmV) if you add a simple downloadable example file with the minimum required HTML-JavaScript to set up a HmV (without all the CSV import code).</li>

                <li><span class="underlined">Color scale</span>. HmV only implements a simple three point linear color scale. For me this is the major weakness of HmV. It will be very convenient that in the next HmV release the user can give as a parameter a function that manages the score to color conversion.</li>
              </ul>

              <p class="bold">b) About the paper</p>

              <ul class="number-list">
                <li>
                  <span class="underlined">Introduction (4<sup>th</sup> paragraph):</span> There are many alternatives to explore a dataset using heat maps. The author only cites two and it&rsquo;s not clear if you refer to &ldquo;JavaScript&rdquo; or &ldquo;web&rdquo; alternatives. I think that you have to emphasize the strengths of HmV in comparison to other alternatives (in my opinion, one strength is that it is a good lightweight alternative to embed heat maps in a web report). Example of alternatives that I know of (but I&rsquo;m sure that there are many more) are:
                  
                  <ul class="bullet-list">
                    <li><a target="_blank" href="http://www.broadinstitute.org/gsea">http://www.broadinstitute.org/gsea (desktop)</a></li>

                    <li><a target="_blank" href="http://jheatmap.github.io/jheatmap/">http://jheatmap.github.io/jheatmap/ (website)</a></li>

                    <li><a target="_blank" href="http://www.gitools.org/">http://www.gitools.org/ (desktop)</a></li>
                    
                    <li><a target="_blank" href="http://blog.nextgenetics.net/demo/entry0044/">http://blog.nextgenetics.net/demo/entry0044/ (website)</a></li>
                    
                    <li><a target="_blank" href="http://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram2d.html">http://docs.scipy.org/doc/numpy/reference/generated/numpy.histogram2d.html (python)</a></li>
                    
                    <li><a target="_blank" href="http://matplotlib.org/api/pyplot_api.html">http://matplotlib.org/api/pyplot_api.html (python)</a></li>
                  </ul>

                </li>

                <li><span class="underlined">Predicted protein mutability landscape:</span> The authors say: &ldquo;Without using a tool such as the HeatmapViewer, we could hardly obtain an overview of the protein mutability landscape&rdquo;. This paragraph seems to suggest that you can explore the data with HmV. I think that HmV is a good tool to report your data, but not to explore it.</li>

                <li><span class="underlined">Conclusions:</span> The authors say: &ldquo;... provides a new, powerful way to generate and display matrix data in web presentations and in publications.&rdquo; To use heat maps in web presentations and publications is nothing new. I think that HmV makes it easier and user-friendly, but it&rsquo;s not new.</li>

              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/3-48" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments u-mt--2"></a>
          <div class="one columns"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">A helpful review with advice for improvement</div>
            <div class="example-authors">John Banks says:</div>
            <div class="collapsed-section">
              <p>This article addresses the links between habitat condition and an endangered bird species in an important forest reserve (ASF) in eastern Kenya. It addresses an important topic, especially given ongoing anthropogenic pressures on this and similar types of forest reserves in eastern Kenya and throughout the tropics. Despite the rather small temporal and spatial extent of the study, it should make an important contribution to bird and forest conservation.</p>
              
              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
             <p> This article addresses the links between habitat condition and an endangered bird species in an important forest reserve (ASF) in eastern Kenya. It addresses an important topic, especially given ongoing anthropogenic pressures on this and similar types of forest reserves in eastern Kenya and throughout the tropics. Despite the rather small temporal and spatial extent of the study, it should make an important contribution to bird and forest conservation. There are a number of issues with the methods and analysis that need to be clarified/addressed however; furthermore, some of the conclusions overreach the data collected, while other important results are given less emphasis that they warrant. Below are more specific comments by section:</p>

            <div class="detail-report-subheading">Abstract:</div>
              
            <p>The conclusion that human-driven tree removal is an important contributor to the degradation of ASF is reasonable given the data reported in the article. Elephant damage, while clearly likely a very big contributor to habitat modification in ASF, was not the focus of the study (the authors state clearly in the Discussion that elephant damage was not systematically quantified, and thus no data were analyzed) &hyphen; and thus should only be mentioned in passing here &hyphen; if at all.</p>
            
            <div class="detail-report-subheading">Introduction:</div>
            
            <p>More information about the life history ecology of <em>A. Sokokensis</em> would provide welcome context here. A bit more detail about breeding sites as well as dispersal behavior etc. would be helpful &ndash; and especially why these and other aspects render the Pipit a good indicator species/proxy for habitat condition. This could be revisited in the Discussion as links are made between habitat conditions and occurrence of the bird (where you discuss the underlying mechanisms for why it thrives in some parts of ASF and not others, and why it&rsquo;s abundance correlate strongly with some types of disturbance and not others). Again, you reference other studies that have explored other species in ASF and forest disturbance, but do not really explicitly state why the Pipit is a particularly important indicator of forest condition.</p>
            
            <div class="detail-report-subheading">Methods:</div>
            
            <ul class="bullet-list">
              <li>Bird Survey: As described, all sightings and calls were recorded and incorporated into distance analysis &ndash; but it is not clear here whether or not distances to both auditory and visual encounters were measured the same way (i.e., with the rangefinder). Please clarify.</li>

              <li>Floor litter sampling: Not clear here whether or not litter cover was recorded as a continuous or categorical variable (percentage). If not, please describe percentage &ldquo;categories&rdquo; used.</li>

            </ul>

            <div class="detail-report-subheading">Results:</div>
              <ul class="bullet-list">
                <li>Mean litter depth graph (Figure 2) and accompanying text reports the means and sd but no post-hoc comparison test (e.g. Tukey HSD) &ndash; need to report the stats on which differences were/were not significant.</li>

                <li>Figure 3 &ndash; you indicate litter depth was better predictor of bird abundance than litter cover, but r-squared is higher for litter cover. Need to clarify (and also indicate why you chose only to shown depth values in Figure 3.</li>

                <li>The linear equation can be put in Figure 3 caption (not necessary to include in text).</li>

                <li>Figure 4 &ndash; stats aren&rsquo;t presented here; also, the caption states that tree loss and leaf litter are inversely correlated &ndash; this might be taken to mean, given discussion (below) about pruning, that there could be a poaching threshold below which poaching may pay dividends to Pipits (and above which Pipits are negatively affected). This warrants further exploration/elaboration.</li>

                <li>The pruning result is arguably the most important one here &ndash; this suggests an intriguing trade-off between poaching and bird conservation (in particular, the suggestion that pruning by poachers may bolster Pipit populations &ndash; or at the very least mitigate against other aspects of habitat degradation). Worth highlighting this more in Discussion.</li>
              </ul>

              <div class="detail-report-subheading">Discussion:</div>

              <ul class="bullet-list">
                <li>Last sentence on p. 7 suggests causality (&ldquo;That is because…&rdquo;) &ndash; but your data only support correlation (one can imagine that there may have been other extrinsic or intrinsic drivers of population decline).</li>

                <li>P. 8: discussion of classification of habitat types in ASF is certainly interesting, but could be made much more succinct in keeping with focus of this paper.</li>

                <li>P. 9, top: first paragraph could be expanded &ndash; as noted before, tradeoff between poaching/pruning and Pipit abundance is worth exploring in more depth. Could your results be taken as a prescription for understory pruning as a conservation tool for the Sokoke Pipit or other threatened species? More detail here would be welcome (and also in Conclusion); in subsequent paragraph about Pipit foraging behavior and specific relationship to understory vegetation at varying heights could be incorporated into this discussion. Is there any info about optimal perch height for foraging or for flying through the understory? Linking to results of other studies in ASF, is there potential for positive correlations with optimal habitat conditions for the other important bird species in ASF in order to make more general conclusions about management?</li>
              </ul>

              <div class="link-row expanded-example">
                <a href="/articles/3-59" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny"> Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">This well-organized review helped the authors improve their article</div>
            <div class="example-authors">Lisa Locatello says:</div>
            <div class="collapsed-section">
              <p>Bierbach and co-authors investigated the topic of the evolution of the audience effect in live bearing fishes, by applying a comparative method. They specifically focused on the hypothesis that sperm competition risk, arising from male mate choice copying, and avoidance of aggressive interactions play a key role in driving the evolution of audience-induced changes in male mate choice behavior.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>Bierbach and co-authors investigated the topic of the evolution of the audience effect in live bearing fishes, by applying a comparative method. They specifically focused on the hypothesis that sperm competition risk, arising from male mate choice copying, and avoidance of aggressive interactions play a key role in driving the evolution of audience-induced changes in male mate choice behavior. The authors found support to their hypothesis of an influence of SCR on the evolution of deceptive behavior as their findings at species level showed a positive correlation between mean sexual activity and the occurrence of deceptive behavior. Moreover, they found a positive correlation between mean aggressiveness and sexual activity but they did not detect a relationship between aggressiveness and audience effects.</p>

              <p>The manuscript is certainly well written and attractive, but I have some major concerns on the data analyses that prevent me to endorse its acceptance at the present stage.</p>

              <p>I see three main problems with the statistics that could have led to potentially wrong results and, thus, to completely misleading conclusions.</p>

              <ul class="bullet-list">
                <li>First of all the Authors cannot run an ANCOVA in which there is a significant interaction between
                  factor and covariate Tab. 2 (a). Indeed, when the assumption of common slopes is violated (as in
                  their case), all other significant terms are meaningless. They might want to consider alternative
                  statistical procedures, e.g. Johnson—Neyman method.</li>

                <li>Second, the Authors cannot retain into the model a non significant interaction term, as this may
                  affect estimations for the factors Tab. 2 (d). They need to remove the species x treatment
                  interaction (as they did for other non significant terms, see top left of the same page 7).</li>

                <li>The third problem I see regards all the GLMs in which species are compared. Authors entered the
                  'species' level as fixed factor when species are clearly a random factor. Entering species as fixed
                  factors has the effect of badly inflating the denominator degrees of freedom, making authors&rsquo;
                  conclusions far too permissive. They should, instead, use mixed LMs, in which species are the random
                  factor. They should also take care that the degrees of freedom are approximately equal to the number
                  of species (not the number of trials). To do so, they can enter as random factor the interaction
                  between treatment and species.</li>
              </ul>

              <p>Data need to be re-analyzed relying on the proper statistical procedures to confirm results and
              conclusions.</p>

              <p>A more theoretical objection to the authors&rsquo; interpretation of results (supposing that results will be confirmed by the new analyses) could emerge from the idea that male success in mating with the preferred female may reduce the probability of immediate female&rsquo;s re-mating, and thus reduce the risk of sperm competition on the short term. As a consequence, it may be not beneficial to significantly increase the risk of losing a high quality and inseminated female for a cost that will not be paid with certainty. The authors might want to consider also this for discussion.</p>

              <p>Lastly, I think that the scenario generated from comparative studies at species level may be explained by phylogenetic factors other than sexual selection. Only the inclusion of phylogeny, that allow to account for the shared history among species, into data analyses can lead to unequivocal adaptive explanations for the observed patterns. I see the difficulty in doing this with few species, as it is the case of the present study, but I would suggest the Authors to consider also this future perspective. Moreover, a phylogenetic comparative study would be aided by the recent development of a well-resolved phylogenetic tree for the genus Poecilia (Meredith 2011).</p>

              <div>Minor comments:</div>
              <p>Page 3: the authors should specify that also part of data on male aggressiveness (3 species from Table 1) come from previous studies, as they do for data on deceptive male mating behavior.</p>

              <p>Page 5: since data on mate choice come from other studies is it so necessary to report a detailed description of methods for this section? Maybe the authors could refer to the already published methods and only give a brief additional description.</p>
              
              <p>Page 6: how do the authors explain the complete absence of aggressive displays between the focal male and the audience male during the mate choice experiments? This sounds curious if considering that in all the examined species aggressive behaviors and dominance establishment are always observed during dyadic encounters.</p>

              <div class="link-row expanded-example">
                <a href="/articles/2-75" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">An example of addressing statistical analysis</div>
            <div class="example-authors">Chris Baker says:</div>

            <div class="collapsed-section">
              <p>In their response to my previous comments, the authors have clarified that only the data from the &ldquo;Experimental phase&rdquo; were used to calculate prediction accuracy. However, if I now understand the analysis procedure correctly, there are serious concerns with the approach adopted.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>In their response to my previous comments, the authors have clarified that only the data from the &ldquo;Experimental phase&rdquo; were used to calculate prediction accuracy. However, if I now understand the analysis procedure correctly, there are serious concerns with the approach adopted.</p>

              <p>First, let me state what I now understand the analysis procedure to be:</p>

              <ul class="number-list">
                <li>For each subject the PD values across the 20 trials were converted to z-scores.</li>
                
                <li>For each stimulus, the mean z-score was calculated.</li>

                <li>The sign of the mean z-score for each stimulus was used to make predictions.</li>
                
                <li>For each of the 20 trials, if the sign of the z-score on that trial was the same as for the mean z-score for that stimulus, a hit (correct prediction) was assigned. In contrast, if the sign of the z-score on that trial was the opposite as for the mean z-score for that stimulus, a miss (incorrect prediction) was assigned.</li>

                <li>For each stimulus the total hits and misses were calculated.</li>

                <li>Average hits (correct prediction) for each stimulus was calculated across subjects.</li>
              </ul>
              
              <p>If this is a correct description of the procedure, the problem is that the same data were used to determine the sign of the z-score that would be associated with a correct prediction and to determine the actual correct predictions. This will effectively guarantee a correct prediction rate above chance.</p>

              <p>To check if this is true, I quickly generated random data and used the analysis procedure as laid out above (see MATLAB code below). Across 10,000 iterations of 100 random subjects, the average &ldquo;prediction&rdquo; accuracy was ~57% for each stimulus (standard deviation, 1.1%), remarkably similar to the values reported by the authors in their two studies. In this simulation, I assumed that all subjects contributed 20 trials, but in the actual data analyzed in the study, some subjects contributed fewer than 20 trials due to artifacts in the pupil measurements.</p>
              
              <p>If the above description of the analysis procedure is correct, then I think the authors have provided no evidence to support pupil dilation prediction of random events, with the results reflecting circularity in the analysis procedure.</p>

              <p>However, if the above description of the procedure is incorrect, the authors need to clarify exactly what the analysis procedure was, perhaps by providing their analysis scripts.</p>

              <div class="detail-report-subheading">MATLAB code:</div>
              
<pre class="pre-code">
nTrials = 10; % 10 trials for each stimulus/condition
 
for boot = 1:10000 % 10,000 iterations for bootstrapping
   
  for i = 1:100 % 100 subjects
 
    data = randn(nTrials,2); % generate random values for each trial

    meandata = squeeze(mean(data(:))); % calculate mean
    stddata = std(data(:)); % calculate standard deviation

    zdata = (data - meandata)/(stddata); % convert to z-scores

    meancond1 = mean(zdata(:,1)); % calculate mean for each stimulus/condition
    meancond2 = mean(zdata(:,2));

    if meancond1 > 0 % evaluate sign of the mean values
      conscore = 1; % conscore indicates for which condition, positive z-values will indicate correctness
      conscoreB = 2; % conscoreB indicates for which condition, negative z-values will indicate correctness
    elseif meancond2 > 0
      conscore = 2;
      conscoreB = 1;
    else
      error = 'They are equal' % if mean z-values are equal, arbitrarily assign correctness
      conscore = 1;
      conscoreB = 2;
    end

    accScores(i) = sum(squeeze(zdata(:,conscore))>0)./nTrials; %calculate average correct for each condition for each subject
    accScoresB(i) = sum(squeeze(zdata(:,conscoreB))<0)./nTrials;

  end

  mAcc(boot) = mean(accScores); % calculate average correct for each condition across subjects for each iteration
  mAccB(boot) = mean(accScoresB);
 
end
 
meanBoot = mean(mAcc)% calculate mean correct for each condition across iterations
meanBootB = mean(mAccB)
 
stdBoot = std(mAcc)% calculate standard deviation for each condition across iterations
stdBootB = std(mAccB)

</pre>
              <div class="link-row expanded-example">
                <a href="/articles/2-262" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

        <div class="row example-detail padding-top-30">
          <a id="positive-comments"></a>
          <div class="one columns u-mt--2"><span class="comment-box example-page"></span></div>
          <div class="ten columns example-colstart">
            <div class="example-heading u-mt--2">Positive comments in a detailed review</div>
            <div class="example-authors">Sue Griffin says:</div>

            <div class="collapsed-section">
              <p>I think this paper excellent and is an important addition to the literature. I really like the conceptualization of a self-replicating cycle as it illustrates the concept that the &ldquo;problem&rdquo; starts with the neuron, i.e., due to one or more of a variety of insults, the neuron is negatively impacted and releases H1, which in turn activates microglia with over expression of cytokines that may, when limited, foster repair but when activated becomes chronic (as is demonstrated here with the potential of cyclic H1 release) and thus facilitates neurotoxicity. I hope the authors intend to measure cytokine expression soon, especially IL-1 and TNF in both astrocytes and microglia, and S100B in astrocytes.</p>

              <div class="link-row">
                <a href="#" title="Read full report" class="more-less-link more icon-link tiny">
                  Read full report <span class="f1r-icon icon-27_sub_marker link-arrow-tiny orange"></span>
                </a>
              </div>
            </div>

            <div class="detail-report-text expanded-section">
              <p>I think this paper excellent and is an important addition to the literature. I really like the conceptualization of a self-replicating cycle as it illustrates the concept that the &ldquo;problem&rdquo; starts with the neuron, i.e., due to one or more of a variety of insults, the neuron is negatively impacted and releases H1, which in turn activates microglia with over expression of cytokines that may, when limited, foster repair but when activated becomes chronic (as is demonstrated here with the potential of cyclic H1 release) and thus facilitates neurotoxicity. I hope the authors intend to measure cytokine expression soon, especially IL-1 and TNF in both astrocytes and microglia, and S100B in astrocytes.</p>

              <div class="detail-report-subheading">02/08/2013 &hyphen; additional comments</div>

              <p>In more detail, Gilthorpe and colleagues provide novel experimental data that demonstrate a new role for a specific histone protein—the linker histone, H1—in neurodegeneration. This study, which was originally designed to identify axonal chemorepellents, actually provided a previously unknown role for H1, as well as other novel and thought provoking results. Fortuitously, as sometimes happens, the authors had a pleasant surprise: their results set some old dogmas on their respective ears and opened up new avenues of approach for studying the role of histones in self-amplification of neurodegenerative cycles. In point, they show that H1 is not just a nice little partner of nuclear DNA as previously thought. H1 is released from &lsquo;damaged&rsquo; (or leaky) neurons, kills adjacent healthy neurons, and promotes a proinflammatory profile in both microglia and astrocytes.</p>

              <p>Interestingly, the authors&rsquo; conceptualization of a damaged neuron &rarr; H1 release &rarr; healthy neuron killing cycle does not take into account the H1-mediated proinflammatory glial response. This facet of the study opens for these investigators a new avenue they may wish to follow: the role of H1 in stimulation of neuroinflammation with overexpression of cytokines. This is interesting, as neuronal injury has been shown to set in motion an acute phase response that activates glia, increases their expression of cytokines (interleukin-1 and S100B), which, in turn, induce neurons to produce excess Alzheimer-related proteins such as &beta;APP and ApoE (favoring formation of mature A&beta;/ApoE plaques), activated MAPK-p38 and hyperphosphorylated tau (favoring formation of neurofibrillary tangles), and &alpha; synuclein (favoring formation of Lewy bodies). To date, the neuronal response shown responsible for stimulating glia is neuronal stress related release of sAPP, but these H1 results from Gilthorpe and colleagues may contribute to or exacerbate the role of sAPP.</p>

              <div class="link-row expanded-example">
                <a href="/articles/2-148" title="View article and reports" class="examples-link show-arrows">View Article &amp; Reports</a>
                <div class="show-less-container">
                  <a href="#" title="Show less" class="more-less-link less icon-link tiny">Show less <span class="f1r-icon icon-77_sub_marker_up link-arrow-tiny orange"></span>
                  </a>
                </div>
              </div>

            </div>
          </div>
          <div class="one columns"></div>
        </div>

      </div>

            <div class="site-nav u-hide u-show--ib@md">

    <div class="site-nav-heading heading5">
                
        About        F1000Research
    </div>

    <div class="site-nav-option">
        <a href="/about" class="sidebar-link" title="Open the How it Works page.">How it Works</a>
    </div>
    <div class="site-nav-option">
        <a href="/for-referees/guidelines" class="sidebar-link" title="Open the Reviewer Guidelines page.">For Reviewers</a>
    </div>
            <div class="site-nav-option">
            <a href="/advisors" class="sidebar-link" title="Open the advisors page.">
                                    Our Advisors
                            </a>
        </div>
        <div class="site-nav-option">
        <a href="/about/policies" class="sidebar-link" title="Open the policies page.">Policies</a>
    </div>
    <div class="site-nav-option">
        <a href="/glossary" class="sidebar-link" title="Open the glossary page.">Glossary</a>
    </div>
    <div class="site-nav-option">
        <a href="/faqs" class="sidebar-link" title="Open the FAQs page.">FAQs</a>
    </div>
    <div class="site-nav-option">
        <a href="/contact" class="sidebar-link" title="Open the contact page.">Contact</a>
    </div>

    
            <div class="sidebar-info-box for-referees-sidebar">
            <div class="sidebar-heading-box">Other Resources</div>
            <div class="sidebar-item-wrapper first">
                <div class="sidebar-item-title">Peer Review: The Nuts And Bolts</div>
                <div class="sidebar-item-published">Published by Sense About Science</div>
                <a href="https://senseaboutscience.org/activities/peer-review-the-nuts-and-bolts-2/" class="icon-link" target="_blank">read article <span class="f1r-icon icon-27_sub_marker read-article-arrow orange"></span></a>
            </div>
            <div class="sidebar-item-wrapper">
                <div class="sidebar-item-title">A Guide to Peer Review in Ecology and Evolution</div>
                <div class="sidebar-item-published">Published by the British Ecological Society</div>
                <a href="http://www.britishecologicalsociety.org/publications/journals/" target="_blank" class="icon-link">Read Article <span class="f1r-icon icon-27_sub_marker read-article-arrow orange"></span></a>
            </div>
            <div class="sidebar-item-wrapper">
                <div class="sidebar-item-title">Are We Training Pit Bulls To Review Our Manuscripts?</div>
                <div class="sidebar-item-published">By Virginia Walbot (Stanford University)</div>
                <a href="http://jbiol.com/content/8/3/24" target="_blank" class="icon-link">Read Article <span class="f1r-icon icon-27_sub_marker read-article-arrow orange"></span></a>
            </div>
        </div>
        <div class="sidebar-info-box">
            <span class="bold">F1000Research</span> is working with the following organizations to educate young researchers about peer review. If you would like to collaborate with us, please contact <a href="mailto:research@f1000.com" class="in-text-link small">research@f1000.com</a>.
            <div class="sidebar-row">
                <a href="http://www.senseaboutscience.org" target="_blank" style="color: #333;">
                    <span class="sense-about-science" title="Open the Sense About Science site."></span>
                </a>
                <a href="http://www.neuroscience.nyu.edu/" target="_blank">
                    <span class="nyu-neuroscience for-referees-sidebar" title="Open the NYU Neuroscience page."></span>
                </a>
            </div>
        </div>
    
</div>
    </div>
  </div>
</div>

<script src="/js/app/research.helpers.js"></script>
<script src="/js/shared_scripts/referee_guidelines.js"></script>
<script>
  $(document).ready(function () {
    R.helpers.initialize({
      "backToTop": true,
      "stickyNavigation": true
    });
    RefereeGuidelines.init();
  });
</script>              </div>
        </div>

        
            
            <div class="o-page__footer sticky-email-wrapper">
                
                

                


<footer class="c-footer t-inverted">

    <div class="o-wrapper">
        <div class="o-layout">


                        
            <div class="o-layout__item u-mb--3">
                <div class="c-branding c-branding--research">
                    <img src="/img/research/F1000Research_white.svg" alt="F1000Research">
                </div>
            </div>


                        
            <div class="o-layout__item u-1/3@md u-mb--3">

                <span class="c-hr c-hr--thick c-hr--low u-mb--2"></span>

                <p class="t-h3 u-mt--0 u-mb--0">An innovative open access publishing platform offering rapid publication and open peer review, whilst supporting data deposition and sharing.</p>

            </div>


                        
            <div class="o-layout__item u-2/3@md">

                <span class="c-hr c-hr--thick c-hr--low u-mb--2"></span>

                <div class="o-layout">
                    <nav class="c-footer__nav">

                            <div class="o-layout__item u-3/5@sm u-mb--3">


                                <div class="o-columns o-columns--2">

                                                                            <a href="/browse/articles" class="t-body c-footer__nav-item "      >Browse</a>
                                                                            <a href="/gateways" class="t-body c-footer__nav-item "      >Gateways</a>
                                                                            <a href="/collections" class="t-body c-footer__nav-item "      >Collections</a>
                                                                            <a href="/about" class="t-body c-footer__nav-item "      >How it Works</a>
                                                                            <a href="/contact" class="t-body c-footer__nav-item "      >Contact</a>
                                                                            <a href="/developers" class="t-body c-footer__nav-item u-hide u-show@navbar"      >For Developers</a>
                                                                            <a href="/about/legal/privacypolicy/cookies" class="t-body c-footer__nav-item "      >Cookie Notice</a>
                                                                            <a href="/about/legal/privacypolicy" class="t-body c-footer__nav-item "      >Privacy Notice</a>
                                                                            <a href="/published/rss" class="t-body c-footer__nav-item "   title="RSS feed of published articles"     >RSS</a>
                                    
                                </div>

                            </div>

                            <div class="o-layout__item u-2/5@sm u-center u-right@sm u-mb--3">

                                <div class="u-hide u-show@lg">
                                    <div class="_mdl-layout">
                                        <a class="mdl-button mdl-js-button mdl-button--inverted mdl-button--no-shadow mdl-js-ripple-effect mdl-button--outline" href="/for-authors/publish-your-research"   data-test-id="footer_submit_research"  >Submit Your Research</a>
                                    </div>
                                </div>

                            </div>

                    </nav>
                </div>

            </div>

            <div class="o-layout__item u-mb--2">
                <div class="c-footer__share">
                        <div class="c-footer__share">
        <span class="c-footer__share-icon" title="Open Access">
            <span class="f1r-icon icon-100_open_access license-icon"></span>
        </span>

        <a class="c-footer__share-icon" href="//creativecommons.org/licenses" target="_blank" title="Creative Commons License CC-BY">
            <span class="f1r-icon icon-116_cc license-icon license-icon-cc"></span>
            <span class="f1r-icon icon-117_ccby license-icon license-icon-cc"></span>
        </a>

        <a class="c-footer__share-icon" href="//creativecommons.org/about/cc0" target="_blank" title="Creative Commons License CC0">
            <span class="f1r-icon icon-118_cco license-icon"></span>
        </a>

    </div>
                </div>
            </div>


                        
            <div class="o-layout__item u-1/3@md u-mb--3">

                <span class="c-hr c-hr--low u-mb--3"></span>

                <p class="c-footer__social u-mt--0 u-mb--0 u-white--low-med">Follow us
                    <a href="https://www.facebook.com/F1000" target="_blank" class="c-footer__social-icon f1r-icon icon-55_footer_facebook"></a>
                    <a href="https://twitter.com/#!/F1000Research" target="_blank" class="c-footer__social-icon f1r-icon icon-56_footer_twitter"></a>
                    <a href="http://www.youtube.com/user/F1000research" target="_blank" class="c-footer__social-icon f1r-icon icon-57_footer_youtube"></a></p>

            </div>


                        
            <div class="o-layout__item u-2/3@md u-right@md">

                <span class="c-hr c-hr--low u-mb--3"></span>

                <p class="t-caption u-white--low-med">&copy; 2012-2025 F1000 Research Ltd. ISSN 2046-1402 | <a href="/about/legal" class="copyrightLegal">Legal</a> | Partner of <a target="_blank" href="https://www.research4life.org/">Research4Life</a>  &bull; <a target="_blank" href="http://crossref.org/">CrossRef</a> &bull; <a target="_blank" href="http://orcid.org/">ORCID</a> &bull; <a target="_blank" href="http://www.fairsharing.org">FAIRSharing</a></p>

            </div>
        </div>
    </div>

</footer>            </div>
        
    </div>

    <script>
                    R.templateTests.simpleTemplate = R.template('<p class="$variable.one">$text</p><p class="${variable.two}">$text</p><p class="$!variable.three">$text</p><p class="$!{variable.four}">$text</p><p class="${selector}.five">$text</p>');
            R.templateTests.runTests();
        
        var F1000platform = new F1000.Platform({
            name: "f1000research",
            displayName: "F1000Research",
            hostName: "f1000research.com",
            id: "1",
            editorialEmail: "research@f1000.com",
            infoEmail: "info@f1000.com",
            usePmcStats: true
        });

                    $(function(){R.ui.dropdowns('.dropdown-for-authors, .dropdown-for-about, .dropdown-for-myresearch');});
            // $(function(){R.ui.dropdowns('.dropdown-for-referees');});

            $(document).ready(function () {
                if ($(".cookie-warning").is(":visible")) {
                    $(".sticky").css("margin-bottom", "35px");
                    $(".devices").addClass("devices-and-cookie-warning");
                }
                $(".cookie-warning .close-button").click(function (e) {
                    $(".devices").removeClass("devices-and-cookie-warning");
                    $(".sticky").css("margin-bottom", "0");
                });

                $("#tweeter-feed .tweet-message").each(function (i, message) {
                    var self = $(message);
                    self.html(linkify(self.html()));
                });

                $(".partner").on("mouseenter mouseleave", function() {
                    $(this).find(".gray-scale, .colour").toggleClass("is-hidden");
                });
            });
        
    </script>

            
<div class="sign-in-popup">
	<!-- <a href="#" class="sign-in shadow">Sign in <span class="sign-in-image-active"></span></a> -->
	<a href="#" class="sign-in ${locale}">Sign In <span class="arrow-closed sign-in-arrow-padding arrow-opened"></span></a>
	<div class="sign-in-form">

            <form action="https://f1000research.com/j_spring_oauth_security_check" id="googleOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                            <input id="google-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-google" name="system" type="hidden" value="GOOGLE"/>
    </form>
            <form action="https://f1000research.com/j_spring_oauth_security_check" id="ECAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                            <input id="ec-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-ec" name="system" type="hidden" value="EC"/>
    </form>
            <form action="https://f1000research.com/j_spring_oauth_security_check" id="facebookOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                            <input id="facebook-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-fb" name="system" type="hidden" value="FACEBOOK"/>
    </form>
            <form action="https://f1000research.com/j_spring_oauth_security_check" id="orcidOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                            <input id="orcid-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-orcid" name="system" type="hidden" value="ORCID"/>
    </form>
		<form id="sign-in-form" class="login-container" action="https://f1000research.com/login" method="post" name="f">
           <div id="sign-in-form-gfb-popup"></div>

                                                            <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                            			<input type="text" name="username" id="signin-email-box" class="sign-in-input" placeholder="Email address" autocomplete="email">
			<input type="password" name="password" id="signin-password-box" class="sign-in-input" placeholder="Password" autocomplete="current-password">
			<div class="sign-in-remember">
                <div class="checkbox-wrapper">
    				<input type="checkbox" id="remember-me" name="remember_me" class="checkbox is-hidden">
                </div>
                <span class="checkbox-label">Remember me</span>
			</div>
			<a href="#" class="sign-in-link" id="forgot-password-link">Forgotten your password?</a>
			<div class="sign-in-button-container margin-top margin-left-20 margin-bottom">
				<button type="submit" id="sign-in-button" class="sign-in-buttons general-white-orange-button">Sign In</button>
				<button type="button" id="sign-in-cancel" class="sign-in-buttons sign-in-cancel-button margin-left">Cancel</button>
				<div class="clearfix"></div>
			</div>
			<div class="sign-in-error">Email or password not correct. Please try again</div>
			<div class="sign-in-loading">Please wait...</div>
		</form>
		<div class="forgot-password-container">
			
<script type="text/javascript">
	$(function(){
		// Note: All the setup needs to run against a name attribute and *not* the id due the clonish
		// nature of facebox...
		$("a[id=googleSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("GOOGLE");
            $("form[id=oAuthForm]").submit();
        });
        $("a[id=facebookSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("FACEBOOK");
            $("form[id=oAuthForm]").submit();
        });
        $("a[id=orcidSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("ORCID");
            $("form[id=oAuthForm]").submit();
        });
	});
</script>

<span class="text first">
	If you've forgotten your password, please enter your email address below and we'll send you instructions on how to reset your password.
    <p>The email address should be the one you originally registered with F1000.</p>
</span>
<input name="email" class="sign-in-input" id="email-forgot-password" type="text" placeholder="Email address">
<div class="forgot-password-email-error">
	Email address not valid, please try again
</div>
<div class="forgot-password-google-email-error">
    <p>You registered with F1000 via Google, so we cannot reset your password.</p>
	<p>To sign in, please click <a href="#" id="googleSignInButton">here</a>.</p>
    <p>If you still need help with your Google account password, please click <a href="https://www.google.com/accounts/recovery">here</a>.</p>
</div>
<div class="forgot-password-facebook-email-error">
    <p>You registered with F1000 via Facebook, so we cannot reset your password.</p>
    <p>To sign in, please click <a href="#" id="facebookSignInButton">here</a>.</p>
	<p>If you still need help with your Facebook account password, please click <a href="https://www.facebook.com/recover/initiate">here</a>.</p>
</div>
<div class="clearfix"></div>
<div class="forgot-password-captcha-error">
	Code not correct, please try again
</div>
<div class="clearfix"></div>
<div class="sign-in-button-container margin-left-20 margin-bottom">
	<button type="button" id="sign-in-reset-password" class="sign-in-buttons general-white-orange-button">Reset password</button>
	<button type="button" id="forgot-password-cancel" class="sign-in-buttons sign-in-cancel-button margin-left">Cancel</button>
	<div class="clearfix"></div>
</div>
<span class="text last">
	<a href="mailto:">Email us</a> for further assistance.
</span>
<form action="https://f1000research.com/j_spring_oauth_security_check" id="oAuthForm" method="post" target="_top">
                        <input class="target-field" type="hidden" name="target" value="/for-referees/peer-reviewing-tips/examples"/>
                <input id="oAuthSystem" name="system" type="hidden"/>
</form>
			<div class="forgot-password-server-error">Server error, please try again.</div>
			<div class="sign-in-success">
                <p>If your email address is registered with us, we will email you instructions to reset your password.</p>
                <p>If you think you should have received this email but it has not arrived, please check your spam filters and/or contact <a href="mailto:"></a> for further assistance.</p>
            </div>
			<div class="sign-in-loading">Please wait...</div>
		</div>

		<div class="sign-in-form-register-section">
			<div class="sign-in-button-container margin-left-20 margin-bottom">
				<a href="/register" title="Register"><button type="button" id="sign-in-register-button" class="sign-in-buttons general-white-orange-button">Register</button></a>
				<div class="clearfix"></div>
			</div>
		</div>

	</div>
</div>

<script type="text/javascript">
$(document).ready(function () {

    signIn.createSignInAsRow($("#sign-in-form-gfb-popup"));

    $(".target-field").each(function () {
        var uris = $(this).val().split("/");
        if (uris.pop() === "login") {
        	$(this).val(uris.toString().replace(",","/"));
        }
    });
});
</script>
                <script type="text/javascript" src="/js/app/research.analytics.js"></script>

        <!-- Start of HubSpot Embed Code -->
        <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/4475190.js"></script>
        <!-- End of HubSpot Embed Code -->
    
            <script src="https://my.hellobar.com/4e0495c6f18cbd68731a1dc1978195a144e767ba.js" type="text/javascript" charset="utf-8" async="async"></script>
    </body>

</html>